{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import requests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [],
   "source": [
    "CONTENT_DETAIL_URL = 'https://api.biorxiv.org/details/biorxiv'\n",
    "#https://api.biorxiv.org/details/[server]/[interval]/[cursor]/[format] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "DATE_START = '2011-01-01'#YYYY-MM-DD \n",
    "DATE_END = '2019-12-31'\n",
    "CURSOR = '0'\n",
    "#cursor is to start point, ie cursor 12 abstract 0 will be equal to cursor 0 abstract 12"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [],
   "source": [
    "def build_url():\n",
    "    url = CONTENT_DETAIL_URL +'/'+ DATE_START +'/'+ DATE_END +'/'+ CURSOR\n",
    "    return url"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [],
   "source": [
    "url = build_url()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://api.biorxiv.org/details/biorxiv/2011-01-01/2019-12-31/0'"
      ]
     },
     "execution_count": 84,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [],
   "source": [
    "data = requests.get(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [],
   "source": [
    "j= data.json()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_doi(j):\n",
    "    \"\"\"Extract doi from the collections\"\"\"\n",
    "    coll_length = len(j['collection'])\n",
    "    for i in coll_length:\n",
    "        doi = j['collection'][i]['doi']\n",
    "    retu"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [],
   "source": [
    "example_abst = j['collection'][0]['abstract']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'The quest for the ecological niches of Saccharomyces cerevisiae ranged from wineries to oaks and more recently to the gut of Crabro Wasps. Here we propose the role of the human gut in shaping S. cerevisiae evolution, presenting the genetic structure of a previously unknown population of yeasts, associated with Crohns disease, providing evidence for clonal expansion within humans gut. To understand the role of immune function in the human-yeast interaction we classified strains according to their immunomodulatory properties, discovering a set of genetically homogeneous isolates, capable of inducing anti-inflammatory signals via regulatory T cells proliferation, and on the contrary, a positive association between strain mosaicism and ability to elicit inflammatory, IL-17 driven, immune responses. The approach integrating genomics with immune phenotyping showed selection on genes involved in sporulation and cell wall remodeling as central for the evolution of S. cerevisiae Crohns strains from passengers to commensals to potential pathogens.'"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "example_abst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'messages': [{'status': 'ok',\n",
       "   'interval': '2011-01-01:2019-12-31',\n",
       "   'cursor': 0,\n",
       "   'count': 100,\n",
       "   'count_new_papers': 68801,\n",
       "   'total': 94990}],\n",
       " 'collection': [{'doi': '10.1101/001891',\n",
       "   'title': 'Population genomics of Saccharomyces cerevisiae human isolates: passengers, colonizers, invaders.',\n",
       "   'authors': 'Carlotta De Filippo;Monica Di Paola;Irene Stefanini;Lisa Rizzetto;Luisa Berná;Matteo Ramazzotti;Leonardo Dapporto;Damariz Rivero;Ivo G Gut;Marta Gut;Mónica Bayés;Jean-Luc Legras;Roberto Viola;Cristina Massi-Benedetti;Antonella De Luca;Luigina Romani;Paolo Lionetti;Duccio Cavalieri;',\n",
       "   'author_corresponding': 'Duccio Cavalieri',\n",
       "   'author_corresponding_institution': 'Fondazione E. Mach (FEM)',\n",
       "   'date': '2014-01-17',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'The quest for the ecological niches of Saccharomyces cerevisiae ranged from wineries to oaks and more recently to the gut of Crabro Wasps. Here we propose the role of the human gut in shaping S. cerevisiae evolution, presenting the genetic structure of a previously unknown population of yeasts, associated with Crohns disease, providing evidence for clonal expansion within humans gut. To understand the role of immune function in the human-yeast interaction we classified strains according to their immunomodulatory properties, discovering a set of genetically homogeneous isolates, capable of inducing anti-inflammatory signals via regulatory T cells proliferation, and on the contrary, a positive association between strain mosaicism and ability to elicit inflammatory, IL-17 driven, immune responses. The approach integrating genomics with immune phenotyping showed selection on genes involved in sporulation and cell wall remodeling as central for the evolution of S. cerevisiae Crohns strains from passengers to commensals to potential pathogens.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001867',\n",
       "   'title': 'Estimating seed bank accumulation and dynamics in three obligate-seeder Proteaceae species',\n",
       "   'authors': 'Meaghan E. Jenkins;David Morrison;Tony D. Auld;',\n",
       "   'author_corresponding': 'David Morrison',\n",
       "   'author_corresponding_institution': 'Swedish University of Agricultural Sciences',\n",
       "   'date': '2014-01-17',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'The seed bank dynamics of the three co-occurring obligate-seeder (i.e. fire-sensitive) Proteaceae species, Banksia ericifolia, Banksia marginata and Petrophile pulchella, were examined at sites of varying time since the most recent fire (i.e. plant age) in the Sydney region. Significant variation among species was found in the number of cones produced, the position of the cones within the canopy, the percentage of barren cones produced (Banksia species only), the number of follicles/bracts produced per cone, and the number of seeds lost/released due to spontaneous fruit rupture. Thus, three different regeneration strategies were observed, highlighting the variation in reproductive strategies of co-occurring Proteaceae species. Ultimately, B. marginata potentially accumulated a seed bank of [~]3000 seeds per plant after 20 years, with [~]1500 seeds per plant for P. pulchella and [~]500 for B. ericifolia. Based on these data, B. marginata and B. ericifolia require a minimum fire-free period of 8-10 years, with 7-8 years for P. pulchella, to allow for an adequate seed bank to accumulate and thus ensure local persistence of these species in fire-prone habitats.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001875',\n",
       "   'title': 'How and where to look for tRNAs in Metazoan mitochondrial genomes, and what you might find when you get there',\n",
       "   'authors': 'David Morrison;',\n",
       "   'author_corresponding': 'David Morrison',\n",
       "   'author_corresponding_institution': 'Swedish University of Agricultural Sciences',\n",
       "   'date': '2014-01-17',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Molecular Biology ',\n",
       "   'abstract': \"The ability to locate and annotate mitochondrial genes is an important practical issue, given the rapidly increasing number of mitogenomes appearing in the public databases. Unfortunately, tRNA genes in Metazoan mitochondria have proved to be problematic because they often vary in number (genes missing or duplicated) and also in the secondary structure of the transcribed tRNAs (T or D arms missing). I have performed a series of comparative analyses of the tRNA genes of a broad range of Metazoan mitogenomes in order to address this issue. I conclude that no single computer program is necessarily capable of finding all of the tRNA genes in any given mitogenome, and that use of both the ARWEN and DOGMA programs is sometimes necessary because they produce complementary false negatives. There are apparently a very large number of erroneous annotations in the databased mitogenome sequences, including missed genes, wrongly annotated locations, false complements, and inconsistent criteria for assigning the 5' and 3' boundaries; and I have listed many of these. The extent of overlap between genes is often greatly exaggerated due to inconsistent annotations, although notable overlaps involving tRNAs are apparently real. Finally, three novel hypotheses were examined and found to have support from the comparative analyses: (1) some organisms have mitogenomic locations that simultaneously code for multiple tRNAs; (2) some organisms have mitogenomic locations that simultaneously code for tRNAs and proteins (but not rRNAs); and (3) one group of nematodes has several genes that code for tRNAs lacking both the D and T arms.\",\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001875',\n",
       "   'title': 'How and where to look for tRNAs in Metazoan mitochondrial genomes, and what you might find when you get there',\n",
       "   'authors': 'David Morrison;',\n",
       "   'author_corresponding': 'David Morrison',\n",
       "   'author_corresponding_institution': 'Swedish University of Agricultural Sciences',\n",
       "   'date': '2014-01-22',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Molecular Biology ',\n",
       "   'abstract': \"The ability to locate and annotate mitochondrial genes is an important practical issue, given the rapidly increasing number of mitogenomes appearing in the public databases. Unfortunately, tRNA genes in Metazoan mitochondria have proved to be problematic because they often vary in number (genes missing or duplicated) and also in the secondary structure of the transcribed tRNAs (T or D arms missing). I have performed a series of comparative analyses of the tRNA genes of a broad range of Metazoan mitogenomes in order to address this issue. I conclude that no single computer program is necessarily capable of finding all of the tRNA genes in any given mitogenome, and that use of both the ARWEN and DOGMA programs is sometimes necessary because they produce complementary false negatives. There are apparently a very large number of erroneous annotations in the databased mitogenome sequences, including missed genes, wrongly annotated locations, false complements, and inconsistent criteria for assigning the 5' and 3' boundaries; and I have listed many of these. The extent of overlap between genes is often greatly exaggerated due to inconsistent annotations, although notable overlaps involving tRNAs are apparently real. Finally, three novel hypotheses were examined and found to have support from the comparative analyses: (1) some organisms have mitogenomic locations that simultaneously code for multiple tRNAs; (2) some organisms have mitogenomic locations that simultaneously code for tRNAs and proteins (but not rRNAs); and (3) one group of nematodes has several genes that code for tRNAs lacking both the D and T arms.\",\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001883',\n",
       "   'title': 'Tracking global changes induced in the CD4 T cell receptor repertoire by immunization with a complex antigen using short stretches of CDR3 protein sequence.',\n",
       "   'authors': 'Niclas Thomas;Katharine Best;Mattia Cinelli;Shlomit Reich-Zeliger;Hila Gal;Eric Shifrut;Asaf Madi;Nir Friedman;John Shawe-Taylor;Benny Chain;',\n",
       "   'author_corresponding': 'Benny Chain',\n",
       "   'author_corresponding_institution': 'UCL',\n",
       "   'date': '2014-01-17',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Immunology ',\n",
       "   'abstract': 'The clonal theory of adaptive immunity proposes that immunological responses are encoded by increases in the frequency of lymphocytes carrying antigen-specific receptors. In this study, we measure the frequency of different TcRs in CD4+ T cell populations of mice immunized with a complex antigen, killed Mycobacterium tuberculosis, using high throughput parallel sequencing of the TcR beta chain. In order to track the changes induced by immunisation within this very heterogeneous repertoire, the sequence data were classified by counting the frequency of different clusters of short (3 or 4) continuous stretches of amino acids within the CDR3 repertoire of different mice. Both unsupervised (hierarchical clustering) and supervised (support vector machine) analysis of these different distributions of sequence clusters differentiated between immunised and unimmunised mice with 100% efficiency. The CD4+ T cell receptor repertoires of mice 5 and 14 days post immunisation were clearly different from that of unimmunised mice, but were not distinguishable from each other. However, the repertoires of mice 60 days post immunisation were distinct both from unimmunised mice, and the day 5/14 animals. Our results reinforce the remarkable diversity of the T cell receptor repertoire, resulting in many diverse private TcRs contributing to the T cell response even in genetically identical mice responding to the same antigen. Finally, specific motifs defined by short sequences of amino acids within the CDR3 region may have a major effect on TcR specificity. The results of this study provide new insights into the properties of the CD4+ adaptive T cell response.',\n",
       "   'published': '10.1093/bioinformatics/btu523',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001909',\n",
       "   'title': 'The shrinking human protein coding complement: are there fewer than 20,000 genes?',\n",
       "   'authors': 'Iakes Ezkurdia;David Juan;Jose Manuel Rodriguez;Adam Frankish;Mark Deikhans;Jennifer L Harrow;Jesus Vazquez;Alfonso Valencia;Michael Tress;',\n",
       "   'author_corresponding': 'Michael Tress',\n",
       "   'author_corresponding_institution': 'Spanish National Cancer Research Centre',\n",
       "   'date': '2014-01-17',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Determining the full complement of protein-coding genes is a key goal of genome annotation. The most powerful approach for confirming protein coding potential is the detection of cellular protein expression through peptide mass spectrometry experiments. Here we map the peptides detected in 7 large-scale proteomics studies to almost 60% of the protein coding genes in the GENCODE annotation the human genome. We find that conservation across vertebrate species and the age of the gene family are key indicators of whether a peptide will be detected in proteomics experiments. We find peptides for most highly conserved genes and for practically all genes that evolved before bilateria. At the same time there is almost no evidence of protein expression for genes that have appeared since primates, or for genes that do not have any protein-like features or cross-species conservation. We identify 19 non-protein-like features such as weak conservation, no protein features or ambiguous annotations in major databases that are indicators of low peptide detection rates. We use these features to describe a set of 2,001 genes that are potentially non-coding, and show that many of these genes behave more like non-coding genes than protein-coding genes. We detect peptides for just 3% of these genes. We suggest that many of these 2,001 genes do not code for proteins under normal circumstances and that they should not be included in the human protein coding gene catalogue. These potential non-coding genes will be revised as part of the ongoing human genome annotation effort.',\n",
       "   'published': '10.1093/hmg/ddu309',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001818',\n",
       "   'title': 'Emergence of structural and dynamical properties of ecological mutualistic networks',\n",
       "   'authors': 'Samir Suweis;Filippo Simini;Jayanth Banavar;Amos Maritan;',\n",
       "   'author_corresponding': 'Samir Suweis',\n",
       "   'author_corresponding_institution': 'Universiyt of Padova',\n",
       "   'date': '2014-01-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Mutualistic networks are formed when the interactions between two classes of species are mutually beneficial. They are important examples of cooperation shaped by evolution. Mutualism between animals and plants plays a key role in the organization of ecological communities1-3. Such networks in ecology have generically evolved a nested architecture4,5 independent of species composition and latitude6,7 - specialists interact with proper subsets of the nodes with whom generalists interact1. Despite sustained efforts5,8,9,10 to explain observed network structure on the basis of community-level stability or persistence, such correlative studies have reached minimal consensus11,12,13. Here we demonstrate that nested interaction networks could emerge as a consequence of an optimization principle aimed at maximizing the species abundance in mutualistic communities. Using analytical and numerical approaches, we show that because of the mutualistic interactions, an increase in abundance of a given species results in a corresponding increase in the total number of individuals in the community, as also the nestedness of the interaction matrix. Indeed, the species abundances and the nestedness of the interaction matrix are correlated by an amount that depends on the strength of the mutualistic interactions. Nestedness and the observed spontaneous emergence of generalist and specialist species occur for several dynamical implementations of the variational principle under stationary conditions. Optimized networks, while remaining stable, tend to be less resilient than their counterparts with randomly assigned interactions. In particular, we analytically show that the abundance of the rarest species is directly linked to the resilience of the community. Our work provides a unifying framework for studying the emergent structural and dynamical properties of ecological mutualistic networks2,5,10,14.',\n",
       "   'published': '10.1038/nature12438',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001826',\n",
       "   'title': 'Expertly validated models suggest responses to climate change are related to species traits: a phylogenetically-controlled analysis of the Order Lagomorpha',\n",
       "   'authors': 'Katie Leach;Ruth Kelly;Alison Cameron;W.Ian Montgomery;Neil Reid;',\n",
       "   'author_corresponding': 'Katie Leach',\n",
       "   'author_corresponding_institution': \"Queen's University Belfast\",\n",
       "   'date': '2014-01-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Climate change during the last five decades has impacted significantly on natural ecosystems and the rate of current climate change is of great concern among conservation biologists. Species Distribution Models (SDMs) have been used widely to project changes in species bioclimatic envelopes under future climate scenarios. Here, we aimed to advance this technique by assessing future changes in the bioclimatic envelopes of an entire mammalian Order, the Lagomorpha, using a novel framework for model validation based jointly on subjective expert evaluation and objective model evaluation statistics. SDMs were built using climatic, topographical and habitat variables for all 87 species under past and current climate scenarios. Expert evaluation and Kappa values were used to validate past and current distribution models and only those deemed  modellable through our framework were projected under future climate scenarios (58 species). We then used phylogenetically-controlled regressions to test whether species traits were correlated with predicted responses to climate change. Climate change will impact more than two-thirds of the Lagomorpha, with leporids (rabbits, hares and jackrabbits) likely to undertake poleward shifts with little overall change in range extent, whilst pikas are likely to show extreme shifts to higher altitudes associated with marked range declines, including the likely extinction of Kozlovs Pika (Ochotona koslowi). Smaller-bodied species were more likely to exhibit range contractions and elevational increases, but showing little poleward movement, and fecund species were more likely to shift latitudinally and elevationally. Our results suggest that species traits may be important indicators of future climate change and we believe multi-species approaches, as demonstrated here, are likely to lead to more effective mitigation measures and conservation management.',\n",
       "   'published': '10.1371/journal.pone.0122267',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001826',\n",
       "   'title': 'Expertly validated models suggest responses to climate change are related to species traits: a phylogenetically-controlled analysis of the Order Lagomorpha',\n",
       "   'authors': 'Katie Leach;Ruth Kelly;Alison Cameron;W.Ian Montgomery;Neil Reid;',\n",
       "   'author_corresponding': 'Katie Leach',\n",
       "   'author_corresponding_institution': \"Queen's University Belfast\",\n",
       "   'date': '2014-10-01',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Climate change during the last five decades has impacted significantly on natural ecosystems and the rate of current climate change is of great concern among conservation biologists. Species Distribution Models (SDMs) have been used widely to project changes in species bioclimatic envelopes under future climate scenarios. Here, we aimed to advance this technique by assessing future changes in the bioclimatic envelopes of an entire mammalian Order, the Lagomorpha, using a novel framework for model validation based jointly on subjective expert evaluation and objective model evaluation statistics. SDMs were built using climatic, topographical and habitat variables for all 87 species under past and current climate scenarios. Expert evaluation and Kappa values were used to validate past and current distribution models and only those deemed  modellable through our framework were projected under future climate scenarios (58 species). We then used phylogenetically-controlled regressions to test whether species traits were correlated with predicted responses to climate change. Climate change will impact more than two-thirds of the Lagomorpha, with leporids (rabbits, hares and jackrabbits) likely to undertake poleward shifts with little overall change in range extent, whilst pikas are likely to show extreme shifts to higher altitudes associated with marked range declines, including the likely extinction of Kozlovs Pika (Ochotona koslowi). Smaller-bodied species were more likely to exhibit range contractions and elevational increases, but showing little poleward movement, and fecund species were more likely to shift latitudinally and elevationally. Our results suggest that species traits may be important indicators of future climate change and we believe multi-species approaches, as demonstrated here, are likely to lead to more effective mitigation measures and conservation management.',\n",
       "   'published': '10.1371/journal.pone.0122267',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001842',\n",
       "   'title': 'The\\xa0emergence\\xa0of\\xa0the\\xa0rescue\\xa0effect\\xa0from\\xa0explicit\\xa0within- and\\xa0between-patch\\xa0dynamics\\xa0in\\xa0a metapopulation',\n",
       "   'authors': 'Anders Eriksson;Federico Elías-Wolff;Bernhard Mehlig;Andrea Manica;',\n",
       "   'author_corresponding': 'Anders Eriksson',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2014-01-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Immigration can rescue local populations from extinction, helping to stabilise a metapopulation. Local population dynamics is important for determining the strength of this rescue effect, but the mechanistic link between local demographic parameters and the rescue effect at the metapopulation level has received very little attention by modellers. We develop an analytical framework that allows us to describe the emergence of the rescue effect from interacting local stochastic dynamics. We show this framework to be applicable to a wide range of spatial scales, providing a powerful and convenient alternative to individual-based models for making predictions concerning the fate of metapopulations. We show that the rescue effect plays an important role in minimising the increase in local extinction probability associated with high demographic stochasticity, but its role is more limited in the case of high local environmental stochasticity of recruitment or survival. While most models postulate the rescue effect, our framework provides an explicit mechanistic link between local dynamics and the emergence of the rescue effect, and more generally the stability of the whole metapopulation.',\n",
       "   'published': '10.1098/rspb.2013.3127',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001842',\n",
       "   'title': 'The\\xa0emergence\\xa0of\\xa0the\\xa0rescue\\xa0effect\\xa0from\\xa0explicit\\xa0within- and\\xa0between-patch\\xa0dynamics\\xa0in\\xa0a metapopulation',\n",
       "   'authors': 'Anders Eriksson;Federico Elías-Wolff;Bernhard Mehlig;Andrea Manica;',\n",
       "   'author_corresponding': 'Anders Eriksson',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2014-03-05',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Immigration can rescue local populations from extinction, helping to stabilise a metapopulation. Local population dynamics is important for determining the strength of this rescue effect, but the mechanistic link between local demographic parameters and the rescue effect at the metapopulation level has received very little attention by modellers. We develop an analytical framework that allows us to describe the emergence of the rescue effect from interacting local stochastic dynamics. We show this framework to be applicable to a wide range of spatial scales, providing a powerful and convenient alternative to individual-based models for making predictions concerning the fate of metapopulations. We show that the rescue effect plays an important role in minimising the increase in local extinction probability associated with high demographic stochasticity, but its role is more limited in the case of high local environmental stochasticity of recruitment or survival. While most models postulate the rescue effect, our framework provides an explicit mechanistic link between local dynamics and the emergence of the rescue effect, and more generally the stability of the whole metapopulation.',\n",
       "   'published': '10.1098/rspb.2013.3127',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001800',\n",
       "   'title': 'The Toxoplasma Acto-MyoA Motor Complex Is Important but Not Essential for Gliding Motility and Host Cell Invasion',\n",
       "   'authors': 'Saskia Egarter;Nicole Andenmatten;Allison J Jackson;Jamie A Whitelaw;Gurmann Pall;Jennifer A Black;David JP Ferguson;Isabelle Tardieux;Alex Mogilner;Markus Meissner;',\n",
       "   'author_corresponding': 'Markus Meissner',\n",
       "   'author_corresponding_institution': 'University of Glasgow',\n",
       "   'date': '2014-01-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'Apicomplexan parasites are thought to actively invade the host cell by gliding motility. This movement is powered by the parasite own actomyosin system and depends on the regulated polymerisation and depolymerisation of actin to generate the force for gliding and host cell penetration. Recent studies demonstrated that Toxoplasma gondii can invade the host cell in the absence of several core components of the invasion machinery, such as the motor protein myosin A (MyoA), the microneme proteins MIC2 and AMA1 and actin, indicating the presence of alternative invasion mechanisms. Here the roles of MyoA, MLC1, GAP45 and Act1, core components of the gliding machinery, are re-dissected in detail. Although important roles of these components for gliding motility and host cell invasion are verified, mutant parasites remain invasive and do not show a block of gliding motility, suggesting that other mechanisms must be in place to enable the parasite to move and invade the host cell. A novel, hypothetical model for parasite gliding motility and invasion is presented based on osmotic forces generated in the cytosol of the parasite that are converted into motility.',\n",
       "   'published': '10.1371/journal.pone.0091819',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001800',\n",
       "   'title': 'The Toxoplasma Acto-MyoA Motor Complex Is Important but Not Essential for Gliding Motility and Host Cell Invasion',\n",
       "   'authors': 'Saskia Egarter;Nicole Andenmatten;Allison J Jackson;Jamie A Whitelaw;Gurmann Pall;Jennifer A Black;David JP Ferguson;Isabelle Tardieux;Alex Mogilner;Markus Meissner;',\n",
       "   'author_corresponding': 'Markus Meissner',\n",
       "   'author_corresponding_institution': 'University of Glasgow',\n",
       "   'date': '2014-02-12',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'Apicomplexan parasites are thought to actively invade the host cell by gliding motility. This movement is powered by the parasite own actomyosin system and depends on the regulated polymerisation and depolymerisation of actin to generate the force for gliding and host cell penetration. Recent studies demonstrated that Toxoplasma gondii can invade the host cell in the absence of several core components of the invasion machinery, such as the motor protein myosin A (MyoA), the microneme proteins MIC2 and AMA1 and actin, indicating the presence of alternative invasion mechanisms. Here the roles of MyoA, MLC1, GAP45 and Act1, core components of the gliding machinery, are re-dissected in detail. Although important roles of these components for gliding motility and host cell invasion are verified, mutant parasites remain invasive and do not show a block of gliding motility, suggesting that other mechanisms must be in place to enable the parasite to move and invade the host cell. A novel, hypothetical model for parasite gliding motility and invasion is presented based on osmotic forces generated in the cytosol of the parasite that are converted into motility.',\n",
       "   'published': '10.1371/journal.pone.0091819',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001800',\n",
       "   'title': 'The Toxoplasma Acto-MyoA Motor Complex Is Important but Not Essential for Gliding Motility and Host Cell Invasion',\n",
       "   'authors': 'Saskia Egarter;Nicole Andenmatten;Allison J Jackson;Jamie A Whitelaw;Gurmann Pall;Jennifer A Black;David JP Ferguson;Isabelle Tardieux;Alex Mogilner;Markus Meissner;',\n",
       "   'author_corresponding': 'Markus Meissner',\n",
       "   'author_corresponding_institution': 'University of Glasgow',\n",
       "   'date': '2014-03-18',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'Apicomplexan parasites are thought to actively invade the host cell by gliding motility. This movement is powered by the parasite own actomyosin system and depends on the regulated polymerisation and depolymerisation of actin to generate the force for gliding and host cell penetration. Recent studies demonstrated that Toxoplasma gondii can invade the host cell in the absence of several core components of the invasion machinery, such as the motor protein myosin A (MyoA), the microneme proteins MIC2 and AMA1 and actin, indicating the presence of alternative invasion mechanisms. Here the roles of MyoA, MLC1, GAP45 and Act1, core components of the gliding machinery, are re-dissected in detail. Although important roles of these components for gliding motility and host cell invasion are verified, mutant parasites remain invasive and do not show a block of gliding motility, suggesting that other mechanisms must be in place to enable the parasite to move and invade the host cell. A novel, hypothetical model for parasite gliding motility and invasion is presented based on osmotic forces generated in the cytosol of the parasite that are converted into motility.',\n",
       "   'published': '10.1371/journal.pone.0091819',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001792',\n",
       "   'title': 'Human paternal and maternal demographic histories: insights from high-resolution Y chromosome and mtDNA sequences',\n",
       "   'authors': 'Sebastian Lippold;Hongyang Xu;Albert Ko;Mingkun Li;Gabriel Renaud;Anne Butthof;Roland Schroeder;Mark Stoneking;',\n",
       "   'author_corresponding': 'Mark Stoneking',\n",
       "   'author_corresponding_institution': 'MPI-EVA',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Genetics ',\n",
       "   'abstract': 'To investigate in detail the paternal and maternal demographic histories of humans, we obtained [~]500 kb of non-recombining Y chromosome (NRY) sequences and complete mtDNA genome sequences from 623 males from 51 populations in the CEPH Human Genome Diversity Panel (HGDP). Our results: confirm the controversial assertion that genetic differences between human populations on a global scale are bigger for the NRY than for mtDNA; suggest very small ancestral effective population sizes (<100) for the out-of-Africa migration as well as for many human populations; and indicate that the ratio of female effective population size to male effective population size (Nf/Nm) has been greater than one throughout the history of modern humans, and has recently increased due to faster growth in Nf. However, we also find substantial differences in patterns of mtDNA vs. NRY variation in different regional groups; thus, global patterns of variation are not necessarily representative of specific geographic regions.',\n",
       "   'published': '10.1186/2041-2223-5-13',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001784',\n",
       "   'title': 'Global Epistasis Makes Adaptation Predictable Despite Sequence-Level Stochasticity',\n",
       "   'authors': 'Sergey Kryazhimskiy;Daniel Paul Rice;Elizabeth Jerison;Michael M Desai;',\n",
       "   'author_corresponding': 'Michael M Desai',\n",
       "   'author_corresponding_institution': 'Harvard University',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Epistasis can make adaptation highly unpredictable, rendering evolutionary trajectories contingent on the chance effects of initial mutations. We used experimental evolution in Saccharomyces cerevisiae to quantify this effect, finding dramatic differences in adaptability between 64 closely related genotypes. Despite these differences, sequencing of 105 evolved clones showed no significant effect of initial genotype on future sequence-level evolution. Instead, reconstruction experiments revealed a consistent pattern of diminishing returns epistasis. Our results suggest that many beneficial mutations affecting a variety of biological processes are globally coupled: they interact strongly, but only through their combined effect on fitness. Sequence-level adaptation is thus highly stochastic. Nevertheless, fitness evolution is strikingly predictable because differences in adaptability are determined only by global fitness-mediated epistasis, not by the identity of individual mutations.',\n",
       "   'published': '10.1126/science.1250939',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001784',\n",
       "   'title': 'Global Epistasis Makes Adaptation Predictable Despite Sequence-Level Stochasticity',\n",
       "   'authors': 'Sergey Kryazhimskiy;Daniel Paul Rice;Elizabeth Jerison;Michael M Desai;',\n",
       "   'author_corresponding': 'Michael M Desai',\n",
       "   'author_corresponding_institution': 'Harvard University',\n",
       "   'date': '2014-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Epistasis can make adaptation highly unpredictable, rendering evolutionary trajectories contingent on the chance effects of initial mutations. We used experimental evolution in Saccharomyces cerevisiae to quantify this effect, finding dramatic differences in adaptability between 64 closely related genotypes. Despite these differences, sequencing of 105 evolved clones showed no significant effect of initial genotype on future sequence-level evolution. Instead, reconstruction experiments revealed a consistent pattern of diminishing returns epistasis. Our results suggest that many beneficial mutations affecting a variety of biological processes are globally coupled: they interact strongly, but only through their combined effect on fitness. Sequence-level adaptation is thus highly stochastic. Nevertheless, fitness evolution is strikingly predictable because differences in adaptability are determined only by global fitness-mediated epistasis, not by the identity of individual mutations.',\n",
       "   'published': '10.1126/science.1250939',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001776',\n",
       "   'title': 'Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens',\n",
       "   'authors': 'Haoyang Cai;Nitin Kumar;Homayoun C Bagheri;Christian von Mering;Mark Robinson;Michael Baudis;',\n",
       "   'author_corresponding': 'Michael Baudis',\n",
       "   'author_corresponding_institution': 'University of Zurich',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'BackgroundChromothripsis is a recently discovered phenomenon of genomic rearrangement, possibly arising during a single genome-shattering event. This could provide an alternative paradigm in cancer development, replacing the gradual accumulation of genomic changes with a \\\\\"one-off\\\\\" catastrophic event. However, the term has been used with varying operational definitions, with the minimal consensus being a large number of locally clustered copy number aberrations. The mechanisms underlying these chromothripsis-like patterns (CTLP) and their specific impact on tumorigenesis are still poorly understood.\\\\n\\\\nResultsHere, we identified CTLP in 918 cancer samples, from a dataset of more than 22,000 oncogenomic arrays covering 132 cancer types. Fragmentation hotspots were found to be located on chromosome 8, 11, 12 and 17. Among the various cancer types, soft-tissue tumors exhibited particularly high CTLP frequencies. Genomic context analysis revealed that CTLP rearrangements frequently occurred in genomes that additionally harbored multiple copy number aberrations (CNAs). An investigation into the affected chromosomal regions showed a large proportion of arm-level pulverization and telomere related events, which would be compatible to a number of underlying mechanisms. We also report evidence that these genomic events may be correlated with patient age, stage and survival rate.\\\\n\\\\nConclusionsThrough a large-scale analysis of oncogenomic array data sets, this study characterized features associated with genomic aberrations patterns, compatible to the spectrum of \\\\\"chromothripsis\\\\\"-definitions as previously used. While quantifying clustered genomic copy number aberrations in cancer samples, our data indicates an underlying biological heterogeneity behind these chromothripsis-like patterns, beyond a well defined \\\\\"chromthripsis\\\\\" phenomenon.',\n",
       "   'published': '10.1186/1471-2164-15-82',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001776',\n",
       "   'title': 'Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens',\n",
       "   'authors': 'Haoyang Cai;Nitin Kumar;Homayoun C Bagheri;Christian von Mering;Mark Robinson;Michael Baudis;',\n",
       "   'author_corresponding': 'Michael Baudis',\n",
       "   'author_corresponding_institution': 'University of Zurich',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'BackgroundChromothripsis is a recently discovered phenomenon of genomic rearrangement, possibly arising during a single genome-shattering event. This could provide an alternative paradigm in cancer development, replacing the gradual accumulation of genomic changes with a \\\\\"one-off\\\\\" catastrophic event. However, the term has been used with varying operational definitions, with the minimal consensus being a large number of locally clustered copy number aberrations. The mechanisms underlying these chromothripsis-like patterns (CTLP) and their specific impact on tumorigenesis are still poorly understood.\\\\n\\\\nResultsHere, we identified CTLP in 918 cancer samples, from a dataset of more than 22,000 oncogenomic arrays covering 132 cancer types. Fragmentation hotspots were found to be located on chromosome 8, 11, 12 and 17. Among the various cancer types, soft-tissue tumors exhibited particularly high CTLP frequencies. Genomic context analysis revealed that CTLP rearrangements frequently occurred in genomes that additionally harbored multiple copy number aberrations (CNAs). An investigation into the affected chromosomal regions showed a large proportion of arm-level pulverization and telomere related events, which would be compatible to a number of underlying mechanisms. We also report evidence that these genomic events may be correlated with patient age, stage and survival rate.\\\\n\\\\nConclusionsThrough a large-scale analysis of oncogenomic array data sets, this study characterized features associated with genomic aberrations patterns, compatible to the spectrum of \\\\\"chromothripsis\\\\\"-definitions as previously used. While quantifying clustered genomic copy number aberrations in cancer samples, our data indicates an underlying biological heterogeneity behind these chromothripsis-like patterns, beyond a well defined \\\\\"chromthripsis\\\\\" phenomenon.',\n",
       "   'published': '10.1186/1471-2164-15-82',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001768',\n",
       "   'title': 'Quantification of nuclear transport in single cells',\n",
       "   'authors': 'Lucía Durrieu;Rikard Johansson;Alan Bush;David L.I. Janzén;Martin Gollvik;Gunnar Cedersund;Alejandro Colman-Lerner;',\n",
       "   'author_corresponding': 'Alejandro Colman-Lerner',\n",
       "   'author_corresponding_institution': 'IFIByNE, DFBMC, FCEN, UBA, Buenos Aires, Argentine',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'Regulation of nuclear transport is a key cellular function involved in many central processes, such as gene expression regulation and signal transduction. Rates of protein movement between cellular compartments can be measured by FRAP. However, no standard and reliable methods to calculate transport rates exist. Here we introduce a method to extract import and export rates, suitable for noisy single cell data. This method consists of microscope procedures, routines for data processing, an ODE model to fit to the data, and algorithms for parameter optimization and error estimation.\\\\n\\\\nUsing this method, we successfully measured import and export rates in individual yeast. For YFP, average transport rates were 0.15 sec-1. We estimated confidence intervals for these parameters through likelihood profile analysis. We found large cell-to-cell variation (CV = 0.79) in these rates, suggesting a hitherto unknown source of cellular heterogeneity. Given the passive nature of YFP diffusion, we attribute this variation to large differences among cells in the number or quality of nuclear pores.\\\\n\\\\nOwing to its broad applicability and sensitivity, this method will allow deeper mechanistic insight into nuclear transport processes and into the largely unstudied cell-to-cell variation in kinetic rates.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-01-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-01-14',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-01-16',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-04-22',\n",
       "   'version': '4',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-07-11',\n",
       "   'version': '5',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001750',\n",
       "   'title': 'Shifts in stability and control effectiveness during evolution of Paraves support aerial maneuvering hypotheses for flight origins',\n",
       "   'authors': 'Dennis Evangelista;Sharlene Cam;Tony Huynh;Austin Kwong;Homayun Mehrabani;Kyle Tse;Robert Dudley;',\n",
       "   'author_corresponding': 'Dennis Evangelista',\n",
       "   'author_corresponding_institution': 'University of North Carolina at Chapel Hill',\n",
       "   'date': '2014-10-03',\n",
       "   'version': '6',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'The capacity for aerial maneuvering was likely a major influence on the evolution of flying animals. Here we evaluate consequences of paravian morphology for aerial performance by quantifying static stability and control effectiveness of physical models for numerous taxa sampled from within the lineage leading to birds (Paraves). Results of aerodynamic testing are mapped phylogenetically to examine how maneuvering characteristics correspond to tail shortening, forewing elaboration, and other morphological features. In the evolution of Paraves we observe shifts from static stability to inherently unstable aerial planforms; control effectiveness also migrated from tails to the forewings. These shifts suggest that some degree of aerodynamic control and and capacity for maneuvering preceded the evolution of strong power stroke. The timing of shifts also suggests features normally considered in light of development of a power stroke may play important roles in control.',\n",
       "   'published': '10.7717/peerj.632',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001719',\n",
       "   'title': 'Modeling the functional relationship network at the splice isoform level through heterogeneous data integration',\n",
       "   'authors': 'Hongdong Li;Rajasree Menon;Ridvan Eksi;Aysam Guerler;Yang Zhang;Gilbert S. Omenn;Yuanfang Guan;',\n",
       "   'author_corresponding': 'Yuanfang Guan',\n",
       "   'author_corresponding_institution': 'University of Michigan, Ann Arbor',\n",
       "   'date': '2014-01-09',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Functional relationship networks, which reveal the collaborative roles between genes, have significantly accelerated our understanding of gene functions and phenotypic relevance. However, establishing such networks for alternatively spliced isoforms remains a difficult, unaddressed problem due to the lack of systematic functional annotations at the isoform level, which renders most supervised learning methods difficult to be applied to isoforms. Here we describe a novel multiple instance learning-based probabilistic approach that integrates large-scale, heterogeneous genomic datasets, including RNA-seq, exon array, protein docking and pseudo-amino acid composition, for modeling a global functional relationship network at the isoform level in the mouse. Using this approach, we formulate a gene pair as a set of isoform pairs of potentially different properties. Through simulation and cross-validation studies, we showed the superior accuracy of our algorithm in revealing the isoform-level functional relationships. The local networks reveal functional diversity of the isoforms of the same gene, as demonstrated by both large-scale analyses and experimental and literature evidence for the disparate functions revealed for the isoforms of Ptbp1 and Anxa6 by our network. Our work can assist the understanding of the diversity of functions achieved by alternative splicing of a limited set of genes in mammalian genomes, and may shift the current gene-centered network prediction paradigm to the isoform level.\\\\n\\\\nAuthor summaryProteins carry out their functions through interacting with each other. Such interactions can be achieved through direct physical interactions, genetic interactions, or co-regulation. To summarize these interactions, researches have established functional relationship networks, in which each gene is represented as a node and the connections between the nodes represent how likely two genes work in the same biological process. Currently, these networks are established at the gene level only, while each gene, in mammalian systems, can be alternatively spliced into multiple isoforms that may have drastically different interaction partners. This information can be mined through integrating data that provide isoform-level information, such as RNA-seq and protein docking scores predicted from amino acid sequences. In this study, we developed a novel algorithm to integrate such data for predicting isoform-level functional relationship networks, which allows us to investigate the collaborative roles between genes at a high resolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001719',\n",
       "   'title': 'Modeling the functional relationship network at the splice isoform level through heterogeneous data integration',\n",
       "   'authors': 'Hongdong Li;Rajasree Menon;Ridvan Eksi;Aysam Guerler;Yang Zhang;Gilbert S. Omenn;Yuanfang Guan;',\n",
       "   'author_corresponding': 'Yuanfang Guan',\n",
       "   'author_corresponding_institution': 'University of Michigan, Ann Arbor',\n",
       "   'date': '2014-03-07',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Functional relationship networks, which reveal the collaborative roles between genes, have significantly accelerated our understanding of gene functions and phenotypic relevance. However, establishing such networks for alternatively spliced isoforms remains a difficult, unaddressed problem due to the lack of systematic functional annotations at the isoform level, which renders most supervised learning methods difficult to be applied to isoforms. Here we describe a novel multiple instance learning-based probabilistic approach that integrates large-scale, heterogeneous genomic datasets, including RNA-seq, exon array, protein docking and pseudo-amino acid composition, for modeling a global functional relationship network at the isoform level in the mouse. Using this approach, we formulate a gene pair as a set of isoform pairs of potentially different properties. Through simulation and cross-validation studies, we showed the superior accuracy of our algorithm in revealing the isoform-level functional relationships. The local networks reveal functional diversity of the isoforms of the same gene, as demonstrated by both large-scale analyses and experimental and literature evidence for the disparate functions revealed for the isoforms of Ptbp1 and Anxa6 by our network. Our work can assist the understanding of the diversity of functions achieved by alternative splicing of a limited set of genes in mammalian genomes, and may shift the current gene-centered network prediction paradigm to the isoform level.\\\\n\\\\nAuthor summaryProteins carry out their functions through interacting with each other. Such interactions can be achieved through direct physical interactions, genetic interactions, or co-regulation. To summarize these interactions, researches have established functional relationship networks, in which each gene is represented as a node and the connections between the nodes represent how likely two genes work in the same biological process. Currently, these networks are established at the gene level only, while each gene, in mammalian systems, can be alternatively spliced into multiple isoforms that may have drastically different interaction partners. This information can be mined through integrating data that provide isoform-level information, such as RNA-seq and protein docking scores predicted from amino acid sequences. In this study, we developed a novel algorithm to integrate such data for predicting isoform-level functional relationship networks, which allows us to investigate the collaborative roles between genes at a high resolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001735',\n",
       "   'title': 'Ecological and Evolutionary Oscillations in Host-Parasite Population Dynamics, and The Red Queen',\n",
       "   'authors': 'Jomar Fajardo Rabajante;',\n",
       "   'author_corresponding': 'Jomar Fajardo Rabajante',\n",
       "   'author_corresponding_institution': 'University of the Philippines',\n",
       "   'date': '2014-01-10',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'In a host-parasite system, the constitutive interaction among the species, regulated by the growth rates and functional response, may induce populations to approach equilibrium or sometimes to exhibit simple cycles or peculiar oscillations, such as chaos. A large carrying capacity coupled with appropriate parasitism effectiveness frequently drives long-term apparent oscillatory dynamics in population size. We name these oscillations due to the structure of the constitutive interaction among species as ecological.\\\\n\\\\nOn the other hand, there are also exceptional cases when the evolving quantitative traits of the hosts and parasites induce oscillating population size, which we call as evolutionary. This oscillatory behavior is dependent on the speed of evolutionary adaptation and degree of evolutionary trade-off. A moderate level of negative trade-off is essential for the existence of oscillations. Evolutionary oscillations due to the host-parasite coevolution (known as the Red Queen) can be observed beyond the ecological oscillations, especially when there are more than two competing species involved.\\\\n\\\\nOne Sentence SummaryWe investigate several cases yielding to oscillating host-parasite populations, and we found that the Red Queen hypothesis can explain some of the exceptional cases.\\\\n\\\\nGraphical Abstract:\\\\n\\\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=\\\\\"FIGDIR/small/001735_ufig1.gif\\\\\" ALT=\\\\\"Figure 1\\\\\">\\\\nView larger version (28K):\\\\norg.highwire.dtl.DTLVardef@1280478org.highwire.dtl.DTLVardef@3d6e28org.highwire.dtl.DTLVardef@10b2718org.highwire.dtl.DTLVardef@133a78d_HPS_FORMAT_FIGEXP  M_FIG C_FIG',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001735',\n",
       "   'title': 'Ecological and Evolutionary Oscillations in Host-Parasite Population Dynamics, and The Red Queen',\n",
       "   'authors': 'Jomar Fajardo Rabajante;',\n",
       "   'author_corresponding': 'Jomar Fajardo Rabajante',\n",
       "   'author_corresponding_institution': 'University of the Philippines',\n",
       "   'date': '2014-01-25',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'In a host-parasite system, the constitutive interaction among the species, regulated by the growth rates and functional response, may induce populations to approach equilibrium or sometimes to exhibit simple cycles or peculiar oscillations, such as chaos. A large carrying capacity coupled with appropriate parasitism effectiveness frequently drives long-term apparent oscillatory dynamics in population size. We name these oscillations due to the structure of the constitutive interaction among species as ecological.\\\\n\\\\nOn the other hand, there are also exceptional cases when the evolving quantitative traits of the hosts and parasites induce oscillating population size, which we call as evolutionary. This oscillatory behavior is dependent on the speed of evolutionary adaptation and degree of evolutionary trade-off. A moderate level of negative trade-off is essential for the existence of oscillations. Evolutionary oscillations due to the host-parasite coevolution (known as the Red Queen) can be observed beyond the ecological oscillations, especially when there are more than two competing species involved.\\\\n\\\\nOne Sentence SummaryWe investigate several cases yielding to oscillating host-parasite populations, and we found that the Red Queen hypothesis can explain some of the exceptional cases.\\\\n\\\\nGraphical Abstract:\\\\n\\\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=\\\\\"FIGDIR/small/001735_ufig1.gif\\\\\" ALT=\\\\\"Figure 1\\\\\">\\\\nView larger version (28K):\\\\norg.highwire.dtl.DTLVardef@1280478org.highwire.dtl.DTLVardef@3d6e28org.highwire.dtl.DTLVardef@10b2718org.highwire.dtl.DTLVardef@133a78d_HPS_FORMAT_FIGEXP  M_FIG C_FIG',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001743',\n",
       "   'title': 'OPPOSING MICROTUBULE MOTORS CONTROL MOTILITY, MORPHOLOGY, AND CARGO SEGREGATION DURING ER-TO-GOLGI TRANSPORT.',\n",
       "   'authors': 'Anna K Brown;Sylvie D Hunt;David J Stephens;',\n",
       "   'author_corresponding': 'David J Stephens',\n",
       "   'author_corresponding_institution': 'University of Bristol',\n",
       "   'date': '2014-01-10',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'We recently demonstrated that dynein and kinesin motors drive multiple aspects of endosomal function in mammalian cells. These functions include driving motility, maintaining morphology (notably through providing longitudinal tension to support vesicle fission), and driving cargo sorting. Microtubule motors drive bidirectional motility during traffic between the endoplasmic reticulum (ER) and Golgi. Here, we have examined the role of microtubule motors in transport carrier motility, morphology, and domain organization during ER-to-Golgi transport. We show that consistent with our findings for endosomal dynamics, microtubule motor function during ER-to-Golgi transport of secretory is required for motility, morphology of, and cargo sorting within vesicular tubular carriers en route to the Golgi. Our data are consistent with previous findings that defined roles for dynein-1 and kinesin-1 (KIF5B) and kinesin-2 in this trafficking step. Our high resolution tracking data identify some intriguing aspects. Depletion of kinesin-1 reduces the number of motile structures seen which is in line with other findings relating to the role of kinesin-1 in ER export. However, those transport carriers that were produced had a much greater run length suggesting that this motor can act as a brake on anterograde motility. Kinesin-2 depletion did not significantly reduce the number of motile transport carriers but did cause a similar increase in run length. These data suggest that kinesins act as negative regulators of ER-to-Golgi transport. Depletion of dynein not only reduced the number of motile carriers formed but also caused tubulation of carriers similar to that seen for SNX-coated early endosomes. Our data indicated that the previously observed anterograde-retrograde polarity of transport carriers in transit to the Golgi from the ER is maintained by microtubule motor function.',\n",
       "   'published': '10.1242/bio.20147633',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001727',\n",
       "   'title': 'SIANN: Strain Identification by Alignment to Near Neighbors',\n",
       "   'authors': 'Samuel Minot;Stephen D Turner;Krista L Ternus;Dana R Kadavy;',\n",
       "   'author_corresponding': 'Samuel Minot',\n",
       "   'author_corresponding_institution': 'Signature Science, LLC',\n",
       "   'date': '2014-01-10',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001701',\n",
       "   'title': 'An unmet actin requirement explains the mitotic inhibition of clathrin-mediated endocytosis',\n",
       "   'authors': 'Satdip Kaur;Andrew B Fielding;Gisela Gassner;Nicholas J Carter;Stephen J Royle;',\n",
       "   'author_corresponding': 'Stephen J Royle',\n",
       "   'author_corresponding_institution': 'University of Warwick',\n",
       "   'date': '2014-01-09',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'Clathrin-mediated endocytosis (CME) is the major internalisation route for many different receptor types in mammalian cells. CME is shut down during early mitosis, but the mechanism of this inhibition is unclear. Here we show that the mitotic shutdown is due to an unmet requirement for actin in CME. In mitotic cells, membrane tension is increased and this invokes a requirement for the actin cytoskeleton to assist the CME machinery to overcome the increased load. However, the actin cytoskeleton is engaged in the formation of a rigid cortex in mitotic cells and is therefore unavailable for deployment. We demonstrate that CME can be \\\\\"restarted\\\\\" in mitotic cells despite high membrane tension, by allowing actin to engage in endocytosis. Mitotic phosphorylation of endocytic proteins is maintained in mitotic cells with restored CME, indicating that direct phosphorylation of the CME machinery does not account for shutdown.',\n",
       "   'published': '10.7554/eLife.00829',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001701',\n",
       "   'title': 'An unmet actin requirement explains the mitotic inhibition of clathrin-mediated endocytosis',\n",
       "   'authors': 'Satdip Kaur;Andrew B Fielding;Gisela Gassner;Nicholas J Carter;Stephen J Royle;',\n",
       "   'author_corresponding': 'Stephen J Royle',\n",
       "   'author_corresponding_institution': 'University of Warwick',\n",
       "   'date': '2014-02-18',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Cell Biology ',\n",
       "   'abstract': 'Clathrin-mediated endocytosis (CME) is the major internalisation route for many different receptor types in mammalian cells. CME is shut down during early mitosis, but the mechanism of this inhibition is unclear. Here we show that the mitotic shutdown is due to an unmet requirement for actin in CME. In mitotic cells, membrane tension is increased and this invokes a requirement for the actin cytoskeleton to assist the CME machinery to overcome the increased load. However, the actin cytoskeleton is engaged in the formation of a rigid cortex in mitotic cells and is therefore unavailable for deployment. We demonstrate that CME can be \\\\\"restarted\\\\\" in mitotic cells despite high membrane tension, by allowing actin to engage in endocytosis. Mitotic phosphorylation of endocytic proteins is maintained in mitotic cells with restored CME, indicating that direct phosphorylation of the CME machinery does not account for shutdown.',\n",
       "   'published': '10.7554/eLife.00829',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001693',\n",
       "   'title': 'A statistical mechanics model for the collective epigenetic histone modification dynamics',\n",
       "   'authors': 'Hang Zhang;XIAO-JUN TIAN;Abhishek Mukhopadhyay;Kenneth S Kim;Jianhua Xing;',\n",
       "   'author_corresponding': 'Jianhua Xing',\n",
       "   'author_corresponding_institution': 'Virginia Tech',\n",
       "   'date': '2014-01-08',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'Epigenetic histone modifications play an important role in the maintenance of different cell phenotypes. The exact molecular mechanism for inheritance of the modification patterns over cell generations remains elusive. We construct a Potts-type model based on experimentally observed nearest-neighbor enzyme lateral interactions and nucleosome covalent modification state biased enzyme recruitment. The model can lead to effective nonlocal interactions among nucleosomes suggested in previous theoretical studies, and epigenetic memory is robustly inheritable against stochastic cellular processes.\\\\n\\\\nPACS numbers: 82.39.Rt, 87.17.Aa, 87.16.Yc, 87.16.A',\n",
       "   'published': '10.1103/PhysRevLett.112.068101',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001693',\n",
       "   'title': 'A statistical mechanics model for the collective epigenetic histone modification dynamics',\n",
       "   'authors': 'Hang Zhang;XIAO-JUN TIAN;Abhishek Mukhopadhyay;Kenneth S Kim;Jianhua Xing;',\n",
       "   'author_corresponding': 'Jianhua Xing',\n",
       "   'author_corresponding_institution': 'Virginia Tech',\n",
       "   'date': '2014-01-08',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Biophysics ',\n",
       "   'abstract': 'Epigenetic histone modifications play an important role in the maintenance of different cell phenotypes. The exact molecular mechanism for inheritance of the modification patterns over cell generations remains elusive. We construct a Potts-type model based on experimentally observed nearest-neighbor enzyme lateral interactions and nucleosome covalent modification state biased enzyme recruitment. The model can lead to effective nonlocal interactions among nucleosomes suggested in previous theoretical studies, and epigenetic memory is robustly inheritable against stochastic cellular processes.\\\\n\\\\nPACS numbers: 82.39.Rt, 87.17.Aa, 87.16.Yc, 87.16.A',\n",
       "   'published': '10.1103/PhysRevLett.112.068101',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000109',\n",
       "   'title': 'Speciation and introgression between Mimulus nasutus and Mimulus guttatus',\n",
       "   'authors': 'Yaniv Brandvain;Amanda M Kenney;Lex Fagel;Graham Coop;Andrea L Sweigart;',\n",
       "   'author_corresponding': 'Yaniv Brandvain',\n",
       "   'author_corresponding_institution': 'Department of Evolution and Ecology & Center for Population Biology, University of California -Davis',\n",
       "   'date': '2013-11-07',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Mimulus guttatus and M. nasutus are an evolutionary and ecological model sister species pair differentiated by ecology, mating system, and partial reproductive isolation. Despite extensive research on this system, the history of divergence and differentiation in this sister pair is unclear. We present and analyze a novel population genomic data set which shows that M. nasutus &quot;budded&quot; off of a central Californian M. guttatus population within the last 200 to 500 thousand years. In this time, the M. nasutus genome has accrued numerous genomic signatures of the transition to predominant selfing. Despite clear biological differentiation, we document ongoing, bidirectional introgression. We observe a negative relationship between the recombination rate and divergence between M. nasutus and sympatric M. guttatus samples, suggesting that selection acts against M. nasutus ancestry in M. guttatus.',\n",
       "   'published': '10.1371/journal.pgen.1004410',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000075',\n",
       "   'title': 'A Scalable Formulation for Engineering Combination Therapies for Evolutionary Dynamics of Disease',\n",
       "   'authors': 'Vanessa Jonsson;Anders Rantzer;Richard M Murray;',\n",
       "   'author_corresponding': 'Vanessa Jonsson',\n",
       "   'author_corresponding_institution': 'Caltech',\n",
       "   'date': '2013-11-07',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'It has been shown that optimal controller synthesis for positive systems can be formulated as a linear program. Leveraging these results, we propose a scalable iterative algorithm for the systematic design of sparse, small gain feedback strategies that stabilize the evolutionary dynamics of a generic disease model. We achieve the desired feedback structure by augmenting the optimization problems with {ell}1 and {ell}2 regularization terms, and illustrate our method on an example inspired by an experimental study aimed at finding appropriate HIV neutralizing antibody therapy combinations in the presence of escape mutants.',\n",
       "   'published': '10.1109/ACC.2014.6859452',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000075',\n",
       "   'title': 'A Scalable Formulation for Engineering Combination Therapies for Evolutionary Dynamics of Disease',\n",
       "   'authors': 'Vanessa Jonsson;Anders Rantzer;Richard M Murray;',\n",
       "   'author_corresponding': 'Vanessa Jonsson',\n",
       "   'author_corresponding_institution': 'Caltech',\n",
       "   'date': '2014-03-30',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'It has been shown that optimal controller synthesis for positive systems can be formulated as a linear program. Leveraging these results, we propose a scalable iterative algorithm for the systematic design of sparse, small gain feedback strategies that stabilize the evolutionary dynamics of a generic disease model. We achieve the desired feedback structure by augmenting the optimization problems with {ell}1 and {ell}2 regularization terms, and illustrate our method on an example inspired by an experimental study aimed at finding appropriate HIV neutralizing antibody therapy combinations in the presence of escape mutants.',\n",
       "   'published': '10.1109/ACC.2014.6859452',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000240',\n",
       "   'title': 'Genome-wide targets of selection: female response to experimental removal of sexual selection in Drosophila melanogaster',\n",
       "   'authors': 'Paolo Innocenti;Ilona Flis;Edward H Morrow;',\n",
       "   'author_corresponding': 'Edward H Morrow',\n",
       "   'author_corresponding_institution': 'University of Sussex',\n",
       "   'date': '2013-11-12',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Despite the common assumption that promiscuity should in general be favored in males, but not in females, to date there is no consensus on the general impact of multiple mating on female fitness. Notably, very little is known about the genetic and physiological features underlying the female response to sexual selection pressures. By combining an experimental evolution approach with genomic techniques, we investigated the effects of single and multiple matings on female fecundity and gene expression. We experimentally manipulated the mating system in replicate populations of Drosophila melanogaster by removing sexual selection, with the aim of testing differences in short term post-mating effects of females evolved under different mating strategies. We show that monogamous females suffer decreased fecundity, a decrease that was partially recovered by experimentally reversing the selection pressure back to the ancestral promiscuous state. The post-mating gene expression profiles of monogamous females differ significantly from promiscuous females, involving 9% of the genes tested. These transcripts are active in several tissues, mainly ovaries, neural tissues and midgut, and are involved in metabolic processes, reproduction and signaling pathways. Our results demonstrate how the female post-mating response can evolve under different mating systems, and provide novel insights into the genes targeted by sexual selection in females, by identifying a list of candidate genes responsible for the decrease in female fecundity in the absence of promiscuity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000208',\n",
       "   'title': 'Population genomics of parallel hybrid zones in the mimetic butterflies, H. melpomene and H. erato',\n",
       "   'authors': 'Nicola Nadeau;Mayte Ruiz;Patricio Salazar;Brian Counterman;Jose Alejandro Medina;Humberto Ortiz-Zuazaga;Anna Morrison;W. Owen McMillan;Chri Jiggins;Riccardo Papa;',\n",
       "   'author_corresponding': 'Chri Jiggins',\n",
       "   'author_corresponding_institution': 'Cambridge',\n",
       "   'date': '2013-11-12',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Hybrid zones can be valuable tools for studying evolution and identifying genomic regions responsible for adaptive divergence and underlying phenotypic variation. Hybrid zones between subspecies of Heliconius butterflies can be very narrow and are maintained by strong selection acting on colour pattern. The co-mimetic species H. erato and H. melpomene have parallel hybrid zones where both species undergo a change from one colour pattern form to another. We use restriction associated DNA sequencing to obtain several thousand genome wide sequence markers and use these to analyse patterns of population divergence across two pairs of parallel hybrid zones in Peru and Ecuador. We compare two approaches for analysis of this type of data; alignment to a reference genome and de novo assembly, and find that alignment gives the best results for species both closely (H. melpomene) and distantly (H. erato, ~15% divergent) related to the reference sequence. Our results confirm that the colour pattern controlling loci account for the majority of divergent regions across the genome, but we also detect other divergent regions apparently unlinked to colour pattern differences. We also use association mapping to identify previously unmapped colour pattern loci, in particular the Ro locus. Finally, we identify within our sample a new cryptic population of H. timareta in Ecuador, which occurs at relatively low altitude and is mimetic with H. melpomene malleti.',\n",
       "   'published': '10.1101/gr.169292.113',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000398',\n",
       "   'title': 'The Origin of Human-infecting Avian Influenza A H6N1 Virus',\n",
       "   'authors': 'Liangsheng Zhang;Zhenguo Zhang;',\n",
       "   'author_corresponding': 'Zhenguo Zhang',\n",
       "   'author_corresponding_institution': 'Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802, USA',\n",
       "   'date': '2013-11-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'In this study, we retraced the origin of the reported avian influenza A H6N1 virus infecting a 20-year-old woman in Taiwan. As we know, this is the first reported case of human infection by the H6N1 virus, because this subtype virus usually circulates in birds and poultry. Therefore it is crucial to know how this virus attained the ability to infect humans. Using phylogenetic analysis, we found that this virus was derived from reassortments of multiple lineages of H6N1 viruses and H5N2 viruses. The results deepen our understanding of how the new human-infecting virus originated and based on these we discussed possible explanations for the H6N1 infection of humans. Our results, together with recent studies of H7N9 viruses which result in severe disorders, suggest that reassortments among avian-type viruses are quite often, which may sometimes result in fatal infections in humans. Thus a close watch on the circulation of avian influenza viruses is pretty necessary.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000406',\n",
       "   'title': 'Universality and predictability in molecular quantitative genetics',\n",
       "   'authors': 'Armita Nourmohammad;Torsten Held;Michael Lassig;',\n",
       "   'author_corresponding': 'Michael Lassig',\n",
       "   'author_corresponding_institution': 'University of Cologne',\n",
       "   'date': '2013-11-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Molecular traits, such as gene expression levels or protein binding affinities, are increasingly accessible to quantitative measurement by modern high-throughput techniques. Such traits measure molecular functions and, from an evolutionary point of view, are important as targets of natural selection. We review recent developments in evolutionary theory and experiments that are expected to become building blocks of a quantitative genetics of molecular traits. We focus on universal evolutionary characteristics: these are largely independent of a traits genetic basis, which is often at least partially unknown. We show that universal measurements can be used to infer selection on a quantitative trait, which determines its evolutionary mode of conservation or adaptation. Furthermore, universality is closely linked to predictability of trait evolution across lineages. We argue that universal trait statistics extends over a range of cellular scales and opens new avenues of quantitative evolutionary systems biology.',\n",
       "   'published': '10.1016/j.gde.2013.11.001',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000406',\n",
       "   'title': 'Universality and predictability in molecular quantitative genetics',\n",
       "   'authors': 'Armita Nourmohammad;Torsten Held;Michael Lassig;',\n",
       "   'author_corresponding': 'Michael Lassig',\n",
       "   'author_corresponding_institution': 'University of Cologne',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Molecular traits, such as gene expression levels or protein binding affinities, are increasingly accessible to quantitative measurement by modern high-throughput techniques. Such traits measure molecular functions and, from an evolutionary point of view, are important as targets of natural selection. We review recent developments in evolutionary theory and experiments that are expected to become building blocks of a quantitative genetics of molecular traits. We focus on universal evolutionary characteristics: these are largely independent of a traits genetic basis, which is often at least partially unknown. We show that universal measurements can be used to infer selection on a quantitative trait, which determines its evolutionary mode of conservation or adaptation. Furthermore, universality is closely linked to predictability of trait evolution across lineages. We argue that universal trait statistics extends over a range of cellular scales and opens new avenues of quantitative evolutionary systems biology.',\n",
       "   'published': '10.1016/j.gde.2013.11.001',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000521',\n",
       "   'title': 'Pathways to social evolution: reciprocity, relatedness, and synergy',\n",
       "   'authors': 'Jeremy Van Cleve;Erol Akcay;',\n",
       "   'author_corresponding': 'Jeremy Van Cleve',\n",
       "   'author_corresponding_institution': 'National Evolutionary Synthesis Center',\n",
       "   'date': '2013-11-16',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Many organisms live in populations structured by space and by class, exhibit plastic responses to their social partners, and are subject to non-additive ecological and fitness effects. Social evolution theory has long recognized that all of these factors can lead to different selection pressures but has only recently attempted to synthesize how these factors interact. Using models for both discrete and continuous phenotypes, we show that analyzing these factors in a consistent framework reveals that they interact with one another in ways previously overlooked. Specifically, behavioral responses (reciprocity), genetic relatedness, and synergy interact in non-trivial ways that cannot be easily captured by simple summary indices of assortment. We demonstrate the importance of these interactions by showing how they have been neglected in previous synthetic models of social behavior both within and between species. These interactions also affect the level of behavioral responses that can evolve in the long run; proximate biological mechanisms are evolutionarily stable when they generate enough responsiveness relative to the level of responsiveness that exactly balances the ecological costs and benefits. Given the richness of social behavior across taxa, these interactions should be a boon for empirical research as they are likely crucial for describing the complex relationship linking ecology, demography, and social behavior.',\n",
       "   'published': '10.1111/evo.12438',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000521',\n",
       "   'title': 'Pathways to social evolution: reciprocity, relatedness, and synergy',\n",
       "   'authors': 'Jeremy Van Cleve;Erol Akcay;',\n",
       "   'author_corresponding': 'Jeremy Van Cleve',\n",
       "   'author_corresponding_institution': 'National Evolutionary Synthesis Center',\n",
       "   'date': '2014-04-17',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Many organisms live in populations structured by space and by class, exhibit plastic responses to their social partners, and are subject to non-additive ecological and fitness effects. Social evolution theory has long recognized that all of these factors can lead to different selection pressures but has only recently attempted to synthesize how these factors interact. Using models for both discrete and continuous phenotypes, we show that analyzing these factors in a consistent framework reveals that they interact with one another in ways previously overlooked. Specifically, behavioral responses (reciprocity), genetic relatedness, and synergy interact in non-trivial ways that cannot be easily captured by simple summary indices of assortment. We demonstrate the importance of these interactions by showing how they have been neglected in previous synthetic models of social behavior both within and between species. These interactions also affect the level of behavioral responses that can evolve in the long run; proximate biological mechanisms are evolutionarily stable when they generate enough responsiveness relative to the level of responsiveness that exactly balances the ecological costs and benefits. Given the richness of social behavior across taxa, these interactions should be a boon for empirical research as they are likely crucial for describing the complex relationship linking ecology, demography, and social behavior.',\n",
       "   'published': '10.1111/evo.12438',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000588',\n",
       "   'title': 'On the concept of biological function, junk DNA and the gospels of ENCODE and Graur et al.',\n",
       "   'authors': 'Claudiu I Bandea;',\n",
       "   'author_corresponding': 'Claudiu I Bandea',\n",
       "   'author_corresponding_institution': 'Centers for Disease Control and Prevention',\n",
       "   'date': '2013-11-18',\n",
       "   'version': '1',\n",
       "   'type': 'Contradictory Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'In a recent article entitled \\\\\"On the immortality of television sets: \\\\\"function\\\\\" in the human genome according to the evolution-free gospel of ENCODE\\\\\", Graur et al. dismantle ENCODEs evidence and conclusion that 80% of the human genome is functional. However, the article by Graur et al. contains assumptions and statements that are questionable. Primarily, the authors limit their evaluation of DNAs biological functions to informational roles, sidestepping putative non-informational functions. Here, I bring forward an old hypothesis on the evolution of genome size and on the role of so called  junk DNA (jDNA), which might explain C-value enigma. According to this hypothesis, the jDNA functions as a defense mechanism against insertion mutagenesis by endogenous and exogenous inserting elements such as retroviruses, thereby protecting informational DNA sequences from inactivation or alteration of their expression. Notably, this model couples the mechanisms and the selective forces responsible for the origin of jDNA with its putative protective biological function, which represents a classic example of  fighting fire with fire. One of the key tenets of this theory is that in humans and many other species, jDNAs serves as a protective mechanism against insertional oncogenic transformation. As an adaptive defense mechanism, the amount of protective DNA varies from one species to another based on the rate of its origin, insertional mutagenesis activity, and evolutionary constraints on genome size.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000661',\n",
       "   'title': 'Natural Allelic Variations of Xenobiotic Enzymes Pleiotropically Affect Sexual Dimorphism in Oryzias latipes',\n",
       "   'authors': 'Takafumi Katsumura;Shoji Oda;Shigeki Nakagome;Tsunehiko Hanihara;Hiroshi Kataoka;Hiroshi Mitani;Shoji Kawamura;Hiroki Oota;',\n",
       "   'author_corresponding': 'Hiroki Oota',\n",
       "   'author_corresponding_institution': 'Kitasato University School of Medicine',\n",
       "   'date': '2013-11-19',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Summary Summary Highlights Results and Discussion Accession Numbers Reference Sexual dimorphisms, which are phenotypic differences between males and females, are driven by sexual selection [1, 2]. Interestingly, sexually selected traits show geographic variations within species despite strong directional selective pressures [3, 4]. However, genetic factors that regulate varied sexual differences remain unknown. In this study, we show that polymorphisms in cytochrome P450 (CYP) 1B1, which encodes a xenobiotic-metabolising enzyme, are associated with local differences of sexual dimorphisms in the anal fin morphology of medaka fish (Oryzias latipes). High and low activity CYP1B1 alleles increased and decreased differences in anal fin sizes ...',\n",
       "   'published': '10.1098/rspb.2014.2259',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000661',\n",
       "   'title': 'Natural Allelic Variations of Xenobiotic Enzymes Pleiotropically Affect Sexual Dimorphism in Oryzias latipes',\n",
       "   'authors': 'Takafumi Katsumura;Shoji Oda;Shigeki Nakagome;Tsunehiko Hanihara;Hiroshi Kataoka;Hiroshi Mitani;Shoji Kawamura;Hiroki Oota;',\n",
       "   'author_corresponding': 'Hiroki Oota',\n",
       "   'author_corresponding_institution': 'Kitasato University School of Medicine',\n",
       "   'date': '2013-11-19',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Summary Summary Highlights Results and Discussion Accession Numbers Reference Sexual dimorphisms, which are phenotypic differences between males and females, are driven by sexual selection [1, 2]. Interestingly, sexually selected traits show geographic variations within species despite strong directional selective pressures [3, 4]. However, genetic factors that regulate varied sexual differences remain unknown. In this study, we show that polymorphisms in cytochrome P450 (CYP) 1B1, which encodes a xenobiotic-metabolising enzyme, are associated with local differences of sexual dimorphisms in the anal fin morphology of medaka fish (Oryzias latipes). High and low activity CYP1B1 alleles increased and decreased differences in anal fin sizes ...',\n",
       "   'published': '10.1098/rspb.2014.2259',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000661',\n",
       "   'title': 'Natural Allelic Variations of Xenobiotic Enzymes Pleiotropically Affect Sexual Dimorphism in Oryzias latipes',\n",
       "   'authors': 'Takafumi Katsumura;Shoji Oda;Shigeki Nakagome;Tsunehiko Hanihara;Hiroshi Kataoka;Hiroshi Mitani;Shoji Kawamura;Hiroki Oota;',\n",
       "   'author_corresponding': 'Hiroki Oota',\n",
       "   'author_corresponding_institution': 'Kitasato University School of Medicine',\n",
       "   'date': '2013-11-25',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Summary Summary Highlights Results and Discussion Accession Numbers Reference Sexual dimorphisms, which are phenotypic differences between males and females, are driven by sexual selection [1, 2]. Interestingly, sexually selected traits show geographic variations within species despite strong directional selective pressures [3, 4]. However, genetic factors that regulate varied sexual differences remain unknown. In this study, we show that polymorphisms in cytochrome P450 (CYP) 1B1, which encodes a xenobiotic-metabolising enzyme, are associated with local differences of sexual dimorphisms in the anal fin morphology of medaka fish (Oryzias latipes). High and low activity CYP1B1 alleles increased and decreased differences in anal fin sizes ...',\n",
       "   'published': '10.1098/rspb.2014.2259',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001016',\n",
       "   'title': 'Predictability of adaptive evolution under the successive fixation assumption',\n",
       "   'authors': 'Sandeep Venkataram;Diamantis Sellis;Dmitri A Petrov;',\n",
       "   'author_corresponding': 'Dmitri A Petrov',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2013-12-02',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Predicting the course of evolution is critical for solving current biomedical challenges such as cancer and the evolution of drug resistant pathogens. One approach to studying evolutionary predictability is to observe repeated, independent evolutionary trajectories of similar organisms under similar selection pressures in order to empirically characterize this adaptive fitness landscape. As this approach is infeasible for many natural systems, a number of recent studies have attempted to gain insight into the adaptive fitness landscape by testing the plausibility of different orders of appearance for a specific set of adaptive mutations in a single adaptive trajectory. While this approach is technically feasible for systems with very few available adaptive mutations, the usefulness of this approach for predicting evolution in situations with highly polygenic adaptation is unknown. It is also unclear whether the presence of stable adaptive polymorphisms can influence the predictability of evolution as measured by these methods. In this work, we simulate adaptive evolution under Fishers geometric model to study evolutionary predictability. Remarkably, we find that the predictability estimated by these methods are anti-correlated, and that the presence of stable adaptive polymorphisms can both qualitatively and quantitatively change the predictability of evolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001016',\n",
       "   'title': 'Predictability of adaptive evolution under the successive fixation assumption',\n",
       "   'authors': 'Sandeep Venkataram;Diamantis Sellis;Dmitri A Petrov;',\n",
       "   'author_corresponding': 'Dmitri A Petrov',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2013-12-04',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Predicting the course of evolution is critical for solving current biomedical challenges such as cancer and the evolution of drug resistant pathogens. One approach to studying evolutionary predictability is to observe repeated, independent evolutionary trajectories of similar organisms under similar selection pressures in order to empirically characterize this adaptive fitness landscape. As this approach is infeasible for many natural systems, a number of recent studies have attempted to gain insight into the adaptive fitness landscape by testing the plausibility of different orders of appearance for a specific set of adaptive mutations in a single adaptive trajectory. While this approach is technically feasible for systems with very few available adaptive mutations, the usefulness of this approach for predicting evolution in situations with highly polygenic adaptation is unknown. It is also unclear whether the presence of stable adaptive polymorphisms can influence the predictability of evolution as measured by these methods. In this work, we simulate adaptive evolution under Fishers geometric model to study evolutionary predictability. Remarkably, we find that the predictability estimated by these methods are anti-correlated, and that the presence of stable adaptive polymorphisms can both qualitatively and quantitatively change the predictability of evolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001016',\n",
       "   'title': 'Predictability of adaptive evolution under the successive fixation assumption',\n",
       "   'authors': 'Sandeep Venkataram;Diamantis Sellis;Dmitri A Petrov;',\n",
       "   'author_corresponding': 'Dmitri A Petrov',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2014-08-21',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Predicting the course of evolution is critical for solving current biomedical challenges such as cancer and the evolution of drug resistant pathogens. One approach to studying evolutionary predictability is to observe repeated, independent evolutionary trajectories of similar organisms under similar selection pressures in order to empirically characterize this adaptive fitness landscape. As this approach is infeasible for many natural systems, a number of recent studies have attempted to gain insight into the adaptive fitness landscape by testing the plausibility of different orders of appearance for a specific set of adaptive mutations in a single adaptive trajectory. While this approach is technically feasible for systems with very few available adaptive mutations, the usefulness of this approach for predicting evolution in situations with highly polygenic adaptation is unknown. It is also unclear whether the presence of stable adaptive polymorphisms can influence the predictability of evolution as measured by these methods. In this work, we simulate adaptive evolution under Fishers geometric model to study evolutionary predictability. Remarkably, we find that the predictability estimated by these methods are anti-correlated, and that the presence of stable adaptive polymorphisms can both qualitatively and quantitatively change the predictability of evolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001016',\n",
       "   'title': 'Predictability of adaptive evolution under the successive fixation assumption',\n",
       "   'authors': 'Sandeep Venkataram;Diamantis Sellis;Dmitri A Petrov;',\n",
       "   'author_corresponding': 'Dmitri A Petrov',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2015-07-28',\n",
       "   'version': '4',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Evolutionary Biology ',\n",
       "   'abstract': 'Predicting the course of evolution is critical for solving current biomedical challenges such as cancer and the evolution of drug resistant pathogens. One approach to studying evolutionary predictability is to observe repeated, independent evolutionary trajectories of similar organisms under similar selection pressures in order to empirically characterize this adaptive fitness landscape. As this approach is infeasible for many natural systems, a number of recent studies have attempted to gain insight into the adaptive fitness landscape by testing the plausibility of different orders of appearance for a specific set of adaptive mutations in a single adaptive trajectory. While this approach is technically feasible for systems with very few available adaptive mutations, the usefulness of this approach for predicting evolution in situations with highly polygenic adaptation is unknown. It is also unclear whether the presence of stable adaptive polymorphisms can influence the predictability of evolution as measured by these methods. In this work, we simulate adaptive evolution under Fishers geometric model to study evolutionary predictability. Remarkably, we find that the predictability estimated by these methods are anti-correlated, and that the presence of stable adaptive polymorphisms can both qualitatively and quantitatively change the predictability of evolution.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000091',\n",
       "   'title': 'Designing Robustness to Temperature in a Feedforward Loop Circuit',\n",
       "   'authors': 'Shaunak Sen;Jongmin Kim;Richard M. Murray;',\n",
       "   'author_corresponding': 'Shaunak Sen',\n",
       "   'author_corresponding_institution': 'Indian Institute of Technology Delhi',\n",
       "   'date': '2013-11-07',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'Incoherent feedforward loops represent important biomolecular circuit elements capable of a rich set of dynamic behavior including adaptation and pulsed responses. Temperature can modulate some of these properties through its effect on the underlying reaction rate parameters. It is generally unclear how to design such a circuit where the properties are robust to variations in temperature. Here, we address this issue using a combination of tools from control and dynamical systems theory as well as preliminary experimental measurements towards such a design. We formalize temperature as an uncertainty acting on system dynamics, exploring both structured and unstructured uncertainty representations. Next, we analyze a standard incoherent feedforward loop circuit, noting mechanisms that intrinsically confer temperature robustness to some of its properties. Further, we explore different negative feedback configurations that can enhance the robustness to temperature. Finally, we find that the response of an incoherent feedforward loop circuit in cells can change with temperature. These results present groundwork for the design of a temperature-robust incoherent feedforward loop circuit.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000430',\n",
       "   'title': 'Negative autoregulation matches production and demand in synthetic transcriptional networks',\n",
       "   'authors': 'Elisa Franco;Giulia Giordano;Per-Ola Forsberg;Richard M Murray;',\n",
       "   'author_corresponding': 'Elisa Franco',\n",
       "   'author_corresponding_institution': 'University of California at Riverside',\n",
       "   'date': '2013-11-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'We propose a negative feedback architecture that regulates activity of artificial genes, or \\\\\"genelets\\\\\", to meet their output downstream demand, achieving robustness with respect to uncertain open-loop output production rates. In particular, we consider the case where the outputs of two genelets interact to form a single assembled product. We show with analysis and experiments that negative autoregulation matches the production and demand of the outputs: the magnitude of the regulatory signal is proportional to the \\\\\"error\\\\\" between the circuit output concentration and its actual demand. This two-device system is experimentally implemented using in vitro transcriptional networks, where reactions are systematically designed by optimizing nucleic acid sequences with publicly available software packages. We build a predictive ordinary differential equation (ODE) model that captures the dynamics of the system, and can be used to numerically assess the scalability of this architecture to larger sets of interconnected genes. Finally, with numerical simulations we contrast our negative autoregulation scheme with a cross-activation architecture, which is less scalable and results in slower response times.',\n",
       "   'published': '10.1021/sb400157z',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000430',\n",
       "   'title': 'Negative autoregulation matches production and demand in synthetic transcriptional networks',\n",
       "   'authors': 'Elisa Franco;Giulia Giordano;Per-Ola Forsberg;Richard M Murray;',\n",
       "   'author_corresponding': 'Elisa Franco',\n",
       "   'author_corresponding_institution': 'University of California at Riverside',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'We propose a negative feedback architecture that regulates activity of artificial genes, or \\\\\"genelets\\\\\", to meet their output downstream demand, achieving robustness with respect to uncertain open-loop output production rates. In particular, we consider the case where the outputs of two genelets interact to form a single assembled product. We show with analysis and experiments that negative autoregulation matches the production and demand of the outputs: the magnitude of the regulatory signal is proportional to the \\\\\"error\\\\\" between the circuit output concentration and its actual demand. This two-device system is experimentally implemented using in vitro transcriptional networks, where reactions are systematically designed by optimizing nucleic acid sequences with publicly available software packages. We build a predictive ordinary differential equation (ODE) model that captures the dynamics of the system, and can be used to numerically assess the scalability of this architecture to larger sets of interconnected genes. Finally, with numerical simulations we contrast our negative autoregulation scheme with a cross-activation architecture, which is less scalable and results in slower response times.',\n",
       "   'published': '10.1021/sb400157z',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001008',\n",
       "   'title': 'Efficient Search, Mapping, and Optimization of Multi-protein Genetic Systems in Diverse Bacteria',\n",
       "   'authors': 'Iman Farasat;Manish Kushwaha;Jason Collens;Michael Easterbrook;Matthew Guido;Howard M Salis;',\n",
       "   'author_corresponding': 'Howard M Salis',\n",
       "   'author_corresponding_institution': 'Penn State University',\n",
       "   'date': '2013-12-02',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'Engineering multi-protein genetic systems to maximize their performance remains a combinatorial challenge, particularly when measurement throughput is limited. We have developed a computational design and modeling approach to build predictive models and identify optimal expression levels, while circumventing combinatorial explosion. Maximally informative genetic system variants are first designed by the RBS Library Calculator, an algorithm that optimizes the smallest ribosome binding site library to efficiently search the expression space across a >10,000-fold range with tailored search resolutions, sequence constraints, and well-predicted translation rates. We validated the algorithms predictions using a 644 sequence data-set, within single and multi-protein genetic systems, modifying plasmids and genomes, and in Escherichia coli and Bacillus subtilis. We then combined the search algorithm with kinetic modeling to map the mechanistic relationship between sequence, expression, and overall activity for a 3-enzyme biosynthesis pathway, requiring only 73 measurements to forward design highly productive pathway variants. The combination of sequence desi gn and systems modeling accelerates the optimization of many-protein systems, and allow previous measurements to quantitatively inform future designs.',\n",
       "   'published': '10.15252/msb.20134955',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001008',\n",
       "   'title': 'Efficient Search, Mapping, and Optimization of Multi-protein Genetic Systems in Diverse Bacteria',\n",
       "   'authors': 'Iman Farasat;Manish Kushwaha;Jason Collens;Michael Easterbrook;Matthew Guido;Howard M Salis;',\n",
       "   'author_corresponding': 'Howard M Salis',\n",
       "   'author_corresponding_institution': 'Penn State University',\n",
       "   'date': '2014-03-03',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'Engineering multi-protein genetic systems to maximize their performance remains a combinatorial challenge, particularly when measurement throughput is limited. We have developed a computational design and modeling approach to build predictive models and identify optimal expression levels, while circumventing combinatorial explosion. Maximally informative genetic system variants are first designed by the RBS Library Calculator, an algorithm that optimizes the smallest ribosome binding site library to efficiently search the expression space across a >10,000-fold range with tailored search resolutions, sequence constraints, and well-predicted translation rates. We validated the algorithms predictions using a 644 sequence data-set, within single and multi-protein genetic systems, modifying plasmids and genomes, and in Escherichia coli and Bacillus subtilis. We then combined the search algorithm with kinetic modeling to map the mechanistic relationship between sequence, expression, and overall activity for a 3-enzyme biosynthesis pathway, requiring only 73 measurements to forward design highly productive pathway variants. The combination of sequence desi gn and systems modeling accelerates the optimization of many-protein systems, and allow previous measurements to quantitatively inform future designs.',\n",
       "   'published': '10.15252/msb.20134955',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001008',\n",
       "   'title': 'Efficient Search, Mapping, and Optimization of Multi-protein Genetic Systems in Diverse Bacteria',\n",
       "   'authors': 'Iman Farasat;Manish Kushwaha;Jason Collens;Michael Easterbrook;Matthew Guido;Howard M Salis;',\n",
       "   'author_corresponding': 'Howard M Salis',\n",
       "   'author_corresponding_institution': 'Penn State University',\n",
       "   'date': '2014-08-05',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'Engineering multi-protein genetic systems to maximize their performance remains a combinatorial challenge, particularly when measurement throughput is limited. We have developed a computational design and modeling approach to build predictive models and identify optimal expression levels, while circumventing combinatorial explosion. Maximally informative genetic system variants are first designed by the RBS Library Calculator, an algorithm that optimizes the smallest ribosome binding site library to efficiently search the expression space across a >10,000-fold range with tailored search resolutions, sequence constraints, and well-predicted translation rates. We validated the algorithms predictions using a 644 sequence data-set, within single and multi-protein genetic systems, modifying plasmids and genomes, and in Escherichia coli and Bacillus subtilis. We then combined the search algorithm with kinetic modeling to map the mechanistic relationship between sequence, expression, and overall activity for a 3-enzyme biosynthesis pathway, requiring only 73 measurements to forward design highly productive pathway variants. The combination of sequence desi gn and systems modeling accelerates the optimization of many-protein systems, and allow previous measurements to quantitatively inform future designs.',\n",
       "   'published': '10.15252/msb.20134955',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000885',\n",
       "   'title': 'Resource usage and gene circuit performance characterization in a cell-free ?breadboard?',\n",
       "   'authors': 'Dan Siegal-Gaskins;Zoltan A. Tuza;Jongmin Kim;Vincent Noireaux;Richard M. Murray;',\n",
       "   'author_corresponding': 'Dan Siegal-Gaskins',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2013-11-25',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'The many successes of synthetic biology have come in a manner largely different from those in other engineering disciplines; in particular, without well-characterized and simplified prototyping environments to play a role analogous to wind-tunnels in aerodynamics and breadboards in electrical engineering. However, as the complexity of synthetic circuits increases, the benefits--in cost savings and design cycle time--of a more traditional engineering approach can be significant. We have recently developed an in vitro  breadboard prototyping platform based on E. coli cell extract that allows biocircuits to operate in an environment considerably simpler than but functionally similar to in vivo. The simplicity of this system makes it a promising tool for rapid biocircuit design and testing, as well as for probing fundamental aspects of gene circuit operation normally masked by cellular complexity. In this work we characterize the cell-free breadboard using real-time and simultaneous measurements of transcriptional and translational activities of a small set of reporter genes and a transcriptional activation cascade. We determine the effects of promoter strength, gene concentration, and nucleoside triphosphate concentration on biocircuit properties, and we isolate the specific contributions of essential biomolecular resources--core RNA polymerase and ribosomes--to overall performance. Importantly, we show how limits on resources, particularly those involved in translation, are manifested as reduced expression in the presence of orthogonal genes that serve as additional loads on the system.',\n",
       "   'published': '10.1021/sb400203p',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000885',\n",
       "   'title': 'Resource usage and gene circuit performance characterization in a cell-free ?breadboard?',\n",
       "   'authors': 'Dan Siegal-Gaskins;Zoltan A. Tuza;Jongmin Kim;Vincent Noireaux;Richard M. Murray;',\n",
       "   'author_corresponding': 'Dan Siegal-Gaskins',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2013-11-26',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'The many successes of synthetic biology have come in a manner largely different from those in other engineering disciplines; in particular, without well-characterized and simplified prototyping environments to play a role analogous to wind-tunnels in aerodynamics and breadboards in electrical engineering. However, as the complexity of synthetic circuits increases, the benefits--in cost savings and design cycle time--of a more traditional engineering approach can be significant. We have recently developed an in vitro  breadboard prototyping platform based on E. coli cell extract that allows biocircuits to operate in an environment considerably simpler than but functionally similar to in vivo. The simplicity of this system makes it a promising tool for rapid biocircuit design and testing, as well as for probing fundamental aspects of gene circuit operation normally masked by cellular complexity. In this work we characterize the cell-free breadboard using real-time and simultaneous measurements of transcriptional and translational activities of a small set of reporter genes and a transcriptional activation cascade. We determine the effects of promoter strength, gene concentration, and nucleoside triphosphate concentration on biocircuit properties, and we isolate the specific contributions of essential biomolecular resources--core RNA polymerase and ribosomes--to overall performance. Importantly, we show how limits on resources, particularly those involved in translation, are manifested as reduced expression in the presence of orthogonal genes that serve as additional loads on the system.',\n",
       "   'published': '10.1021/sb400203p',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000885',\n",
       "   'title': 'Resource usage and gene circuit performance characterization in a cell-free ?breadboard?',\n",
       "   'authors': 'Dan Siegal-Gaskins;Zoltan A. Tuza;Jongmin Kim;Vincent Noireaux;Richard M. Murray;',\n",
       "   'author_corresponding': 'Dan Siegal-Gaskins',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2013-12-10',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'The many successes of synthetic biology have come in a manner largely different from those in other engineering disciplines; in particular, without well-characterized and simplified prototyping environments to play a role analogous to wind-tunnels in aerodynamics and breadboards in electrical engineering. However, as the complexity of synthetic circuits increases, the benefits--in cost savings and design cycle time--of a more traditional engineering approach can be significant. We have recently developed an in vitro  breadboard prototyping platform based on E. coli cell extract that allows biocircuits to operate in an environment considerably simpler than but functionally similar to in vivo. The simplicity of this system makes it a promising tool for rapid biocircuit design and testing, as well as for probing fundamental aspects of gene circuit operation normally masked by cellular complexity. In this work we characterize the cell-free breadboard using real-time and simultaneous measurements of transcriptional and translational activities of a small set of reporter genes and a transcriptional activation cascade. We determine the effects of promoter strength, gene concentration, and nucleoside triphosphate concentration on biocircuit properties, and we isolate the specific contributions of essential biomolecular resources--core RNA polymerase and ribosomes--to overall performance. Importantly, we show how limits on resources, particularly those involved in translation, are manifested as reduced expression in the presence of orthogonal genes that serve as additional loads on the system.',\n",
       "   'published': '10.1021/sb400203p',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000885',\n",
       "   'title': 'Resource usage and gene circuit performance characterization in a cell-free ?breadboard?',\n",
       "   'authors': 'Dan Siegal-Gaskins;Zoltan A. Tuza;Jongmin Kim;Vincent Noireaux;Richard M. Murray;',\n",
       "   'author_corresponding': 'Dan Siegal-Gaskins',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2014-03-09',\n",
       "   'version': '4',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'The many successes of synthetic biology have come in a manner largely different from those in other engineering disciplines; in particular, without well-characterized and simplified prototyping environments to play a role analogous to wind-tunnels in aerodynamics and breadboards in electrical engineering. However, as the complexity of synthetic circuits increases, the benefits--in cost savings and design cycle time--of a more traditional engineering approach can be significant. We have recently developed an in vitro  breadboard prototyping platform based on E. coli cell extract that allows biocircuits to operate in an environment considerably simpler than but functionally similar to in vivo. The simplicity of this system makes it a promising tool for rapid biocircuit design and testing, as well as for probing fundamental aspects of gene circuit operation normally masked by cellular complexity. In this work we characterize the cell-free breadboard using real-time and simultaneous measurements of transcriptional and translational activities of a small set of reporter genes and a transcriptional activation cascade. We determine the effects of promoter strength, gene concentration, and nucleoside triphosphate concentration on biocircuit properties, and we isolate the specific contributions of essential biomolecular resources--core RNA polymerase and ribosomes--to overall performance. Importantly, we show how limits on resources, particularly those involved in translation, are manifested as reduced expression in the presence of orthogonal genes that serve as additional loads on the system.',\n",
       "   'published': '10.1021/sb400203p',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000448',\n",
       "   'title': 'Design and implementation of a synthetic biomolecular concentration tracker',\n",
       "   'authors': 'Victoria Hsiao;Emmanuel LC de los Santos;Weston R Whitaker;John E Dueber;Richard M Murray;',\n",
       "   'author_corresponding': 'Victoria Hsiao',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'As a field, synthetic biology strives to engineer increasingly complex artificial systems in living cells. Active feedback in closed loop systems offers a dynamic and adaptive way to ensure constant relative activity independent of intrinsic and extrinsic noise. In this work, we design, model, and implement a biomolecular concentration tracker, in which an output protein tracks the concentration of an input protein. Synthetic modular protein scaffold domains are used to colocalize a two-component system, and a single negative feedback loop modulates the production of the output protein. Using a combination of model and experimental work, we show that the circuit achieves real-time protein concentration tracking in Escherichia coli and that steady state outputs can be tuned.',\n",
       "   'published': '10.1021/sb500024b',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000448',\n",
       "   'title': 'Design and implementation of a synthetic biomolecular concentration tracker',\n",
       "   'authors': 'Victoria Hsiao;Emmanuel LC de los Santos;Weston R Whitaker;John E Dueber;Richard M Murray;',\n",
       "   'author_corresponding': 'Victoria Hsiao',\n",
       "   'author_corresponding_institution': 'California Institute of Technology',\n",
       "   'date': '2013-12-10',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Synthetic Biology ',\n",
       "   'abstract': 'As a field, synthetic biology strives to engineer increasingly complex artificial systems in living cells. Active feedback in closed loop systems offers a dynamic and adaptive way to ensure constant relative activity independent of intrinsic and extrinsic noise. In this work, we design, model, and implement a biomolecular concentration tracker, in which an output protein tracks the concentration of an input protein. Synthetic modular protein scaffold domains are used to colocalize a two-component system, and a single negative feedback loop modulates the production of the output protein. Using a combination of model and experimental work, we show that the circuit achieves real-time protein concentration tracking in Escherichia coli and that steady state outputs can be tuned.',\n",
       "   'published': '10.1021/sb500024b',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000455',\n",
       "   'title': 'A data repository and analysis framework for spontaneous neural activity recordings in developing retina',\n",
       "   'authors': 'Stephen Eglen;Michael Weeks;Mark Jessop;Jennifer Simonotto;Tom Jackson;Evelyne Sernagor;',\n",
       "   'author_corresponding': 'Stephen Eglen',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Neuroscience ',\n",
       "   'abstract': 'BackgroundDuring early development, neural circuits fire spontaneously, generating activity episodes with complex spatiotemporal patterns. Recordings of spontaneous activity have been made in many parts of the nervous system over the last 25 years, reporting developmental changes in activity patterns and the effects of various genetic perturbations.\\\\n\\\\nResultsWe present a curated repository of multielectrode array recordings of spontaneous activity in developing mouse and ferret retina. The data have been annotated with minimal metadata and converted into HDF5. This paper describes the structure of the data, along with examples of reproducible research using these data files. We also demonstrate how these data can be analysed in the CARMEN workflow system. This article is written as a literate programming document; all programs and data described here are freely available.\\\\n\\\\nConclusions1. We hope this repository will lead to novel analysis of spontaneous activity recorded in different laboratories. 2. We encourage published data to be added to the repository. 3. This repository serves as an example of how multielectrode array recordings can be stored for long-term reuse.',\n",
       "   'published': '10.1186/2047-217X-3-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000455',\n",
       "   'title': 'A data repository and analysis framework for spontaneous neural activity recordings in developing retina',\n",
       "   'authors': 'Stephen Eglen;Michael Weeks;Mark Jessop;Jennifer Simonotto;Tom Jackson;Evelyne Sernagor;',\n",
       "   'author_corresponding': 'Stephen Eglen',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2013-11-27',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Neuroscience ',\n",
       "   'abstract': 'BackgroundDuring early development, neural circuits fire spontaneously, generating activity episodes with complex spatiotemporal patterns. Recordings of spontaneous activity have been made in many parts of the nervous system over the last 25 years, reporting developmental changes in activity patterns and the effects of various genetic perturbations.\\\\n\\\\nResultsWe present a curated repository of multielectrode array recordings of spontaneous activity in developing mouse and ferret retina. The data have been annotated with minimal metadata and converted into HDF5. This paper describes the structure of the data, along with examples of reproducible research using these data files. We also demonstrate how these data can be analysed in the CARMEN workflow system. This article is written as a literate programming document; all programs and data described here are freely available.\\\\n\\\\nConclusions1. We hope this repository will lead to novel analysis of spontaneous activity recorded in different laboratories. 2. We encourage published data to be added to the repository. 3. This repository serves as an example of how multielectrode array recordings can be stored for long-term reuse.',\n",
       "   'published': '10.1186/2047-217X-3-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000455',\n",
       "   'title': 'A data repository and analysis framework for spontaneous neural activity recordings in developing retina',\n",
       "   'authors': 'Stephen Eglen;Michael Weeks;Mark Jessop;Jennifer Simonotto;Tom Jackson;Evelyne Sernagor;',\n",
       "   'author_corresponding': 'Stephen Eglen',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2014-02-18',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Neuroscience ',\n",
       "   'abstract': 'BackgroundDuring early development, neural circuits fire spontaneously, generating activity episodes with complex spatiotemporal patterns. Recordings of spontaneous activity have been made in many parts of the nervous system over the last 25 years, reporting developmental changes in activity patterns and the effects of various genetic perturbations.\\\\n\\\\nResultsWe present a curated repository of multielectrode array recordings of spontaneous activity in developing mouse and ferret retina. The data have been annotated with minimal metadata and converted into HDF5. This paper describes the structure of the data, along with examples of reproducible research using these data files. We also demonstrate how these data can be analysed in the CARMEN workflow system. This article is written as a literate programming document; all programs and data described here are freely available.\\\\n\\\\nConclusions1. We hope this repository will lead to novel analysis of spontaneous activity recorded in different laboratories. 2. We encourage published data to be added to the repository. 3. This repository serves as an example of how multielectrode array recordings can be stored for long-term reuse.',\n",
       "   'published': '10.1186/2047-217X-3-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000067',\n",
       "   'title': 'Genetics of single-cell protein abundance variation in large yeast populations',\n",
       "   'authors': 'Frank Albert;Sebastian Treusch;Arthur H Shockley;Joshua S Bloom;Leonid Kruglyak;',\n",
       "   'author_corresponding': 'Leonid Kruglyak',\n",
       "   'author_corresponding_institution': 'UCLA',\n",
       "   'date': '2013-11-07',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Many DNA sequence variants influence phenotypes by altering gene expression. Our understanding of these variants is limited by sample sizes of current studies and by measurements of mRNA rather than protein abundance. We developed a powerful method for identifying genetic loci that influence protein expression in very large populations of the yeast Saccharomyes cerevisiae. The method measures single-cell protein abundance through the use of green-fluorescent-protein tags. We applied this method to 160 genes and detected many more loci per gene than previous studies. We also observed closer correspondence between loci that influence protein abundance and loci that influence mRNA abundance of a given gene. Most loci cluster at hotspot locations that influence multiple proteins--in some cases, more than half of those examined. The variants that underlie these hotspots have profound effects on the gene regulatory network and provide insights into genetic variation in cell physiology between yeast strains.',\n",
       "   'published': '10.1038/nature12904',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000265',\n",
       "   'title': 'A genome wide dosage suppressor network reveals genetic robustness and a novel mechanism for Huntington&amp;#146;s disease',\n",
       "   'authors': 'Biranchi Patra;Yoshiko Kon;Gitanjali Yadav;Anthony Sevold;Jesse P Frumkin;Ravishankar R Vallabhajosyula;Arend Hintze;Bjørn Østman;Jory Schossau;Ashish Bhan;Bruz Marzolf;Jenna K Tamashiro;Amardeep Kaur;Nitin S Baliga;Elizabeth J Grayhack;Christoph Adami;David J Galas;Alpan Raval;Eric M Phizicky;Animesh Ray;',\n",
       "   'author_corresponding': 'Animesh Ray',\n",
       "   'author_corresponding_institution': 'Keck Graduate Institute',\n",
       "   'date': '2013-11-12',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Mutational robustness is the extent to which an organism has evolved to withstand the effects of deleterious mutations. We explored the extent of mutational robustness in the budding yeast by genome wide dosage suppressor analysis of 53 conditional lethal mutations in cell division cycle and RNA synthesis related genes, revealing 660 suppressor interactions of which 642 are novel. This collection has several distinctive features, including high co-occurrence of mutant-suppressor pairs within protein modules, highly correlated functions between the pairs, and higher diversity of functions among the co-suppressors than previously observed. Dosage suppression of essential genes encoding RNA polymerase subunits and chromosome cohesion complex suggest a surprising degree of functional plasticity of macromolecular complexes and the existence of degenerate pathways for circumventing potentially lethal mutations. The utility of dosage-suppressor networks is illustrated by the discovery of a novel connection between chromosome cohesion-condensation pathways involving homologous recombination, and Huntingtons disease.',\n",
       "   'published': '10.1093/nar/gkw1148',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000216',\n",
       "   'title': 'A Complete Public Domain Family Genomics Dataset',\n",
       "   'authors': 'Manuel Corpas;Mike Cariaso;Alain Coletta;David Weiss;Andrew P Harrison;Federico Moran;Huanming Yang;',\n",
       "   'author_corresponding': 'Manuel Corpas',\n",
       "   'author_corresponding_institution': 'Independent',\n",
       "   'date': '2013-11-12',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'BackgroundThe availability of open access genomic data is essential for the personal genomics field. Public genomic data allow comparative analyses, testing of new tools and genotype-phenotype association studies. Personal genomics data of unrelated individuals are available in the public domain, notably the Personal Genome Project; however, to date genomics family data and metadata are severely lacking, mainly due to cost, privacy concerns or restricted access to Next Generation Sequencing (NGS) technology. Family data have a lot to offer as they allow the study of heritability, something which is impossible to do just by using unrelated individuals.\\\\n\\\\nFindingsA whole family from Southern Spain decided to genotype, sequence and analyse their personal genomes making them publicly available under a Creative Commons 0 license (CC0; commonly denominated as public domain). These data include a) five 23andMe SNP chip genotype bed files, b) four raw exomes with their assorted bam files and VCF files, c) a metagenomic raw sequencing data file and d) derived data of likely phenotypes using SNPedia-derived tools.\\\\n\\\\nConclusionsTo our knowledge this is the first CC0 released set of genomic, phenotypic and metagenomic data for a whole family. This dataset is also unique in that it was obtained through direct-to-consumer genetic tests. Hence any ordinary citizen with enough budget and samples should be able to reproduce this experiment. We envisage this dataset to be a useful resource for a variety of applications in the personal genomics field as a) negative control data for trait association discovery, b) testing data for development of new software and c) sample data for heritability studies. We encourage prospective users to share with us derived results so that they can be added to our existing collection.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000315',\n",
       "   'title': 'On the Reproducibility of TCGA Ovarian Cancer MicroRNA Profiles',\n",
       "   'authors': 'Ying-Wooi Wan;Claire Mach;Genevera I. Allen;Matthew Anderson;Zhandong Liu;',\n",
       "   'author_corresponding': 'Zhandong Liu',\n",
       "   'author_corresponding_institution': 'Baylor College of Medicine',\n",
       "   'date': '2013-11-13',\n",
       "   'version': '1',\n",
       "   'type': 'Contradictory Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Dysregulated microRNA (miRNA) expression is a well-established feature of human cancer. However, the role of specific miRNAs in determining cancer outcomes remains unclear. Using Level 3 expression data from the Cancer Genome Atlas (TCGA), we identified 61 miRNAs that are associated with overall survival in 469 ovarian cancers profiled by microarray (p<0.01). We also identified 12 miRNAs that are associated with survival when miRNAs were profiled in the same specimens using Next Generation Sequencing (miRNA-Seq) (p<0.01). Surprisingly, only 1 miRNA transcript is associated with ovarian cancer survival in both datasets. Our analyses indicate that this discrepancy is due to the fact that miRNA levels reported by the two platforms correlate poorly, even after correcting for potential issues inherent to signal detection algorithms. Further investigation is warranted.',\n",
       "   'published': '10.1371/journal.pone.0087782',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000752',\n",
       "   'title': 'Joint analysis of functional genomic data and genome-wide association studies of 18 human traits',\n",
       "   'authors': 'Joseph Pickrell;',\n",
       "   'author_corresponding': 'Joseph Pickrell',\n",
       "   'author_corresponding_institution': 'New York Genome Center',\n",
       "   'date': '2013-11-19',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Annotations of gene structures and regulatory elements can inform genome-wide association studies (GWAS). However, choosing the relevant annotations for interpreting an association study of a given trait remains challenging. We describe a statistical model that uses association statistics computed across the genome to identify classes of genomic element that are enriched or depleted for loci that influence a trait. The model naturally incorporates multiple types of annotations. We applied the model to GWAS of 18 human traits, including red blood cell traits, platelet traits, glucose levels, lipid levels, height, BMI, and Crohns disease. For each trait, we evaluated the relevance of 450 different genomic annotations, including protein-coding genes, enhancers, and DNase-I hypersensitive sites in over a hundred tissues and cell lines. We show that the fraction of phenotype-associated SNPs that influence protein sequence ranges from around 2% (for platelet volume) up to around 20% (for LDL cholesterol); that repressed chromatin is significantly depleted for SNPs associated with several traits; and that cell type-specific DNase-I hypersensitive sites are enriched for SNPs associated with several traits (for example, the spleen in platelet volume). Finally, by re-weighting each GWAS using information from functional genomics, we increase the number of loci with high-confidence associations by around 5%.',\n",
       "   'published': '10.1016/j.ajhg.2014.03.004',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000752',\n",
       "   'title': 'Joint analysis of functional genomic data and genome-wide association studies of 18 human traits',\n",
       "   'authors': 'Joseph Pickrell;',\n",
       "   'author_corresponding': 'Joseph Pickrell',\n",
       "   'author_corresponding_institution': 'New York Genome Center',\n",
       "   'date': '2013-11-22',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Annotations of gene structures and regulatory elements can inform genome-wide association studies (GWAS). However, choosing the relevant annotations for interpreting an association study of a given trait remains challenging. We describe a statistical model that uses association statistics computed across the genome to identify classes of genomic element that are enriched or depleted for loci that influence a trait. The model naturally incorporates multiple types of annotations. We applied the model to GWAS of 18 human traits, including red blood cell traits, platelet traits, glucose levels, lipid levels, height, BMI, and Crohns disease. For each trait, we evaluated the relevance of 450 different genomic annotations, including protein-coding genes, enhancers, and DNase-I hypersensitive sites in over a hundred tissues and cell lines. We show that the fraction of phenotype-associated SNPs that influence protein sequence ranges from around 2% (for platelet volume) up to around 20% (for LDL cholesterol); that repressed chromatin is significantly depleted for SNPs associated with several traits; and that cell type-specific DNase-I hypersensitive sites are enriched for SNPs associated with several traits (for example, the spleen in platelet volume). Finally, by re-weighting each GWAS using information from functional genomics, we increase the number of loci with high-confidence associations by around 5%.',\n",
       "   'published': '10.1016/j.ajhg.2014.03.004',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000752',\n",
       "   'title': 'Joint analysis of functional genomic data and genome-wide association studies of 18 human traits',\n",
       "   'authors': 'Joseph Pickrell;',\n",
       "   'author_corresponding': 'Joseph Pickrell',\n",
       "   'author_corresponding_institution': 'New York Genome Center',\n",
       "   'date': '2014-01-22',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Annotations of gene structures and regulatory elements can inform genome-wide association studies (GWAS). However, choosing the relevant annotations for interpreting an association study of a given trait remains challenging. We describe a statistical model that uses association statistics computed across the genome to identify classes of genomic element that are enriched or depleted for loci that influence a trait. The model naturally incorporates multiple types of annotations. We applied the model to GWAS of 18 human traits, including red blood cell traits, platelet traits, glucose levels, lipid levels, height, BMI, and Crohns disease. For each trait, we evaluated the relevance of 450 different genomic annotations, including protein-coding genes, enhancers, and DNase-I hypersensitive sites in over a hundred tissues and cell lines. We show that the fraction of phenotype-associated SNPs that influence protein sequence ranges from around 2% (for platelet volume) up to around 20% (for LDL cholesterol); that repressed chromatin is significantly depleted for SNPs associated with several traits; and that cell type-specific DNase-I hypersensitive sites are enriched for SNPs associated with several traits (for example, the spleen in platelet volume). Finally, by re-weighting each GWAS using information from functional genomics, we increase the number of loci with high-confidence associations by around 5%.',\n",
       "   'published': '10.1016/j.ajhg.2014.03.004',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000752',\n",
       "   'title': 'Joint analysis of functional genomic data and genome-wide association studies of 18 human traits',\n",
       "   'authors': 'Joseph Pickrell;',\n",
       "   'author_corresponding': 'Joseph Pickrell',\n",
       "   'author_corresponding_institution': 'New York Genome Center',\n",
       "   'date': '2014-02-25',\n",
       "   'version': '4',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Genomics ',\n",
       "   'abstract': 'Annotations of gene structures and regulatory elements can inform genome-wide association studies (GWAS). However, choosing the relevant annotations for interpreting an association study of a given trait remains challenging. We describe a statistical model that uses association statistics computed across the genome to identify classes of genomic element that are enriched or depleted for loci that influence a trait. The model naturally incorporates multiple types of annotations. We applied the model to GWAS of 18 human traits, including red blood cell traits, platelet traits, glucose levels, lipid levels, height, BMI, and Crohns disease. For each trait, we evaluated the relevance of 450 different genomic annotations, including protein-coding genes, enhancers, and DNase-I hypersensitive sites in over a hundred tissues and cell lines. We show that the fraction of phenotype-associated SNPs that influence protein sequence ranges from around 2% (for platelet volume) up to around 20% (for LDL cholesterol); that repressed chromatin is significantly depleted for SNPs associated with several traits; and that cell type-specific DNase-I hypersensitive sites are enriched for SNPs associated with several traits (for example, the spleen in platelet volume). Finally, by re-weighting each GWAS using information from functional genomics, we increase the number of loci with high-confidence associations by around 5%.',\n",
       "   'published': '10.1016/j.ajhg.2014.03.004',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000364',\n",
       "   'title': 'Sampling principles for biodiversity study',\n",
       "   'authors': 'Xubin Pan;',\n",
       "   'author_corresponding': 'Xubin Pan',\n",
       "   'author_corresponding_institution': 'Chinese Academy of Inspection and Quarantine',\n",
       "   'date': '2013-11-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Ecology ',\n",
       "   'abstract': 'Sampling is a fundamental tool in ecology and critical for biodiversity measurement. However, basic principles of biodiversity sampling have been overlooked for many years. In this paper, I proposed and explored five principles of sampling for a specific area and biodiversity study. The first principle of sampling, species increasing with area, is that the number of species increases with the area. The second principle of sampling, individuals increasing with area, is that the number of individuals increases with the area. The third principle of sampling, sum of species number, is that the sum of species number in one area and species number in another area is no less than the total species number in the two areas. The fourth principle of sampling, individual complement, is that the sum of the mathematical expectation of individual number of one or several species in the area a and that of the same one or several species in the area A-a is the total individual number N of the same one or several species in the total area A. The fifth principle of sampling, species-area theory, is that the sum of the mathematical expectation of number of species in the area a and that of number of species lost if area A-a is cleared is the total species number M in the total area A.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000158',\n",
       "   'title': 'Functional Annotation Signatures of Disease Susceptibility Loci Improve SNP Association Analysis',\n",
       "   'authors': 'Edwin S Iversen;Gary Lipton;Merlise A. Clyde;Alvaro N. A. Monteiro;',\n",
       "   'author_corresponding': 'Edwin S Iversen',\n",
       "   'author_corresponding_institution': 'Duke University',\n",
       "   'date': '2013-11-11',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'We describe the development and application of a Bayesian statistical model for the prior probability of phenotype-genotype association that incorporates data from past association studies and publicly available functional annotation data regarding the susceptibility variants under study. The model takes the form of a binary regression of association status on a set of annotation variables whose coefficients were estimated through an analysis of associated SNPs housed in the GWAS Catalog (GC). The set of functional predictors we examined includes measures that have been demonstrated to correlate with the association status of SNPs in the GC and some whose utility in this regard is speculative: summaries of the UCSC Human Genome Browser ENCODE super-track data, dbSNP function class, sequence conservation summaries, proximity to genomic variants included in the Database of Genomic Variants (DGV) and known regulatory elements included in the Open Regulatory Annotation database (ORegAnno), PolyPhen-2 probabilities and RegulomeDB categories. Because we expected that only a fraction of the annotation variables would contribute to predicting association, we employed a penalized likelihood method to reduce the impact of non-informative predictors and evaluated the models ability to predict GC SNPs not used to construct the model. We show that the functional data alone are predictive of a SNPs presence in the GC. Further, using data from a genome-wide study of ovarian cancer, we demonstrate that their use as prior data when testing for association is practical at the genome-wide scale and improves power to detect associations.',\n",
       "   'published': '10.1186/1471-2164-15-398',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000489',\n",
       "   'title': 'Gappy TotalReCaller for RNASeq Base-Calling and Mapping',\n",
       "   'authors': 'Bud (Bhubaneswar) Mishra;',\n",
       "   'author_corresponding': 'Bud (Bhubaneswar) Mishra',\n",
       "   'author_corresponding_institution': 'New York University',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Understanding complex mammalian biology depends crucially on our ability to define a precise map of all the transcripts encoded in a genome, and to measure their relative abundances. A promising assay depends on RNASeq approaches, which builds on next generation sequencing pipelines capable of interrogating cDNAs extracted from a cell. The underlying pipeline starts with base-calling, collect the sequence reads and interpret the raw-read in terms of transcripts that are grouped with respect to different splice-variant isoforms of a messenger RNA. We address a very basic problem involved in all of these pipeline, namely accurate Bayesian base-calling, which could combine the analog intensity data with suitable underlying priors on base-composition in the transcripts. In the context of sequencing genomic DNA, a powerful approach for base-calling has been developed in the TotalReCaller pipeline. For these purposes, it uses a suitable reference whole-genome sequence in a compressed self-indexed format to derive its priors. However, TotalReCaller faces many new challenges in the transcriptomic domain, especially since we still lack a fully annotated library of all possible transcripts, and hence a sufficiently good prior. There are many possible solutions, similar to the ones developed for TotalReCaller, in applications addressing de novo sequencing and assembly, where partial contigs or string-graphs could be used to boot-strap the Bayesian priors on base-composition. A similar approach would be applicable here too, partial assembly of transcripts can be used to characterize the splicing junctions or organize them in incompatibility graphs and then used as priors for TotalReCaller. The key algorithmic techniques for this purpose have been addressed in a forthcoming paper on Stringomics. Here, we address a related but fundamental problem, by assuming that we only have a reference genome, with certain intervals marked as candidate regions for ORF (Open Reading Frames), but not necessarily complete annotations regarding the 5 or 3 termini of a gene or its exon-intron structure. The algorithms we describe find the most accurate base-calls of a cDNA with the best possible segmentation, all mapped to the genome appropriately.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000497',\n",
       "   'title': 'Unexpected links reflect the noise in networks',\n",
       "   'authors': 'Anatoly Yambartsev;Michael Perlin;Yevgeniy Kovchegov;Natalia Shulzhenko;Karina Mine;Andrey Morgun;',\n",
       "   'author_corresponding': 'Andrey Morgun',\n",
       "   'author_corresponding_institution': 'Oregon State University',\n",
       "   'date': '2013-11-15',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Gene regulatory networks are commonly used for modeling biological processes and revealing underlying molecular mechanisms. The reconstruction of gene regulatory networks from observational data is a challenging task, especially, considering the large number of involved players (e.g. genes) and much fewer biological replicates available for analysis. Herein, we proposed a new statistical method of estimating the number of erroneous edges that strongly enhances the commonly used inference approaches. This method is based on special relationship between correlation and causality, and allows to identify and to remove approximately half of erroneous edges. Using the mathematical model of Bayesian networks and positive correlation inequalities we established a mathematical foundation for our method. Analyzing real biological datasets, we found a strong correlation between the results of our method and the commonly used false discovery rate (FDR) technique. Furthermore, the simulation analysis demonstrates that in large networks, our new method provides a more precise estimation of the proportion of erroneous links than FDR.',\n",
       "   'published': '10.1186/s13062-016-0155-0',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000851',\n",
       "   'title': 'Comment on “TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions” by Kim et al.',\n",
       "   'authors': 'Alexander Dobin;Thomas R Gingeras;',\n",
       "   'author_corresponding': 'Alexander Dobin',\n",
       "   'author_corresponding_institution': 'Cold Spring Harbor Laboratory',\n",
       "   'date': '2013-11-22',\n",
       "   'version': '1',\n",
       "   'type': 'Contradictory Results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'In the recent paper [1] (thereafter referred to as \\\\\"TopHat2paper\\\\\") the accuracy of TopHat2 was compared to other RNA-seq aligners. In this comment we re-examine most important analyses from the TopHat2paper and identify several deficiencies that significantly diminished performance of some of the aligners, including incorrect choice of mapping parameters, unfair comparison metrics, and unrealistic simulated data. Using STAR [2] as an exemplar, we demonstrate that correcting these deficiencies makes its accuracy equal or better than that of TopHat2. Furthermore, this exercise highlighted some serious issues with the TopHat2 algorithms, such as poor recall of alignments with a moderate (>3) number of mismatches, low sensitivity and high false discovery rate for splice junction detection, loss of precision for the realignment algorithm, and large number of false chimeric alignments. ...',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000919',\n",
       "   'title': 'Inferring tree causal models of cancer progression with probability raising',\n",
       "   'authors': 'Loes Olde Loohuis;Giulio Caravagna;Alex Graudenzi;Daniele Ramazzotti;Giancarlo Mauri;Marco Antoniotti;Bud Mishra;',\n",
       "   'author_corresponding': 'Bud Mishra',\n",
       "   'author_corresponding_institution': 'Courant Institute of Mathematical Sciences',\n",
       "   'date': '2013-11-25',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Existing techniques to reconstruct tree models of progression for accumulative processes, such as cancer, seek to estimate causation by combining correlation and a frequentist notion of temporal priority. In this paper, we define a novel theoretical framework called CAPRESE (CAncer PRogression Extraction with Single Edges) to reconstruct such models based on the notion of probabilistic causation defined by Suppes. We consider a general reconstruction setting complicated by the presence of noise in the data due to biological variation, as well as experimental or measurement errors. To improve tolerance to noise we define and use a shrinkage-like estimator. We prove the correctness of our algorithm by showing asymptotic convergence to the correct tree under mild constraints on the level of noise. Moreover, on synthetic data, we show that our approach outperforms the state-of-the-art, that it is efficient even with a relatively small number of samples and that its performance quickly converges to its asymptote as the number of samples increases. For real cancer datasets obtained with different technologies, we highlight biologically significant differences in the progressions inferred with respect to other competing techniques and we also show how to validate conjectured biological relations with progression models.',\n",
       "   'published': '10.1371/journal.pone.0108358',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000919',\n",
       "   'title': 'Inferring tree causal models of cancer progression with probability raising',\n",
       "   'authors': 'Loes Olde Loohuis;Giulio Caravagna;Alex Graudenzi;Daniele Ramazzotti;Giancarlo Mauri;Marco Antoniotti;Bud Mishra;',\n",
       "   'author_corresponding': 'Bud Mishra',\n",
       "   'author_corresponding_institution': 'Courant Institute of Mathematical Sciences',\n",
       "   'date': '2014-08-19',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Existing techniques to reconstruct tree models of progression for accumulative processes, such as cancer, seek to estimate causation by combining correlation and a frequentist notion of temporal priority. In this paper, we define a novel theoretical framework called CAPRESE (CAncer PRogression Extraction with Single Edges) to reconstruct such models based on the notion of probabilistic causation defined by Suppes. We consider a general reconstruction setting complicated by the presence of noise in the data due to biological variation, as well as experimental or measurement errors. To improve tolerance to noise we define and use a shrinkage-like estimator. We prove the correctness of our algorithm by showing asymptotic convergence to the correct tree under mild constraints on the level of noise. Moreover, on synthetic data, we show that our approach outperforms the state-of-the-art, that it is efficient even with a relatively small number of samples and that its performance quickly converges to its asymptote as the number of samples increases. For real cancer datasets obtained with different technologies, we highlight biologically significant differences in the progressions inferred with respect to other competing techniques and we also show how to validate conjectured biological relations with progression models.',\n",
       "   'published': '10.1371/journal.pone.0108358',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000919',\n",
       "   'title': 'Inferring tree causal models of cancer progression with probability raising',\n",
       "   'authors': 'Loes Olde Loohuis;Giulio Caravagna;Alex Graudenzi;Daniele Ramazzotti;Giancarlo Mauri;Marco Antoniotti;Bud Mishra;',\n",
       "   'author_corresponding': 'Bud Mishra',\n",
       "   'author_corresponding_institution': 'Courant Institute of Mathematical Sciences',\n",
       "   'date': '2014-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Existing techniques to reconstruct tree models of progression for accumulative processes, such as cancer, seek to estimate causation by combining correlation and a frequentist notion of temporal priority. In this paper, we define a novel theoretical framework called CAPRESE (CAncer PRogression Extraction with Single Edges) to reconstruct such models based on the notion of probabilistic causation defined by Suppes. We consider a general reconstruction setting complicated by the presence of noise in the data due to biological variation, as well as experimental or measurement errors. To improve tolerance to noise we define and use a shrinkage-like estimator. We prove the correctness of our algorithm by showing asymptotic convergence to the correct tree under mild constraints on the level of noise. Moreover, on synthetic data, we show that our approach outperforms the state-of-the-art, that it is efficient even with a relatively small number of samples and that its performance quickly converges to its asymptote as the number of samples increases. For real cancer datasets obtained with different technologies, we highlight biologically significant differences in the progressions inferred with respect to other competing techniques and we also show how to validate conjectured biological relations with progression models.',\n",
       "   'published': '10.1371/journal.pone.0108358',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000919',\n",
       "   'title': 'Inferring tree causal models of cancer progression with probability raising',\n",
       "   'authors': 'Loes Olde Loohuis;Giulio Caravagna;Alex Graudenzi;Daniele Ramazzotti;Giancarlo Mauri;Marco Antoniotti;Bud Mishra;',\n",
       "   'author_corresponding': 'Bud Mishra',\n",
       "   'author_corresponding_institution': 'Courant Institute of Mathematical Sciences',\n",
       "   'date': '2014-10-10',\n",
       "   'version': '4',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Existing techniques to reconstruct tree models of progression for accumulative processes, such as cancer, seek to estimate causation by combining correlation and a frequentist notion of temporal priority. In this paper, we define a novel theoretical framework called CAPRESE (CAncer PRogression Extraction with Single Edges) to reconstruct such models based on the notion of probabilistic causation defined by Suppes. We consider a general reconstruction setting complicated by the presence of noise in the data due to biological variation, as well as experimental or measurement errors. To improve tolerance to noise we define and use a shrinkage-like estimator. We prove the correctness of our algorithm by showing asymptotic convergence to the correct tree under mild constraints on the level of noise. Moreover, on synthetic data, we show that our approach outperforms the state-of-the-art, that it is efficient even with a relatively small number of samples and that its performance quickly converges to its asymptote as the number of samples increases. For real cancer datasets obtained with different technologies, we highlight biologically significant differences in the progressions inferred with respect to other competing techniques and we also show how to validate conjectured biological relations with progression models.',\n",
       "   'published': '10.1371/journal.pone.0108358',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001024',\n",
       "   'title': 'Exploring DNA structures in real-time polymerase kinetics using Pacific Biosciences sequencer data',\n",
       "   'authors': 'Sterling Sawaya;James Boocock;Mik Black;Neil Gemmell;',\n",
       "   'author_corresponding': 'Sterling Sawaya',\n",
       "   'author_corresponding_institution': 'University of Otago',\n",
       "   'date': '2013-12-02',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Bioinformatics ',\n",
       "   'abstract': 'Pausing of DNA polymerase can indicate the presence of a DNA structure that differs from the canonical double-helix. Here we detail a method to investigate how polymerase pausing in the Pacific Biosciences sequencer reads can be related to DNA structure. The Pacific Biosciences sequencer uses optics to view a polymerase and its interaction with a single DNA molecule in real-time, offering a unique way to detect potential alternative DNA structures. We have developed a new way to examine polymerase kinetics and relate it to the DNA sequence by using a wavelet transform of read information from the sequencer. We use this method to examine how polymerase kinetics are related to nucleotide base composition. We then examine tandem repeat sequences known for their ability to form different DNA structures: (CGG)n and (CG)n repeats which can, respectively, form G-quadruplex DNA and Z-DNA. We find pausing around the (CGG)n repeat that may indicate the presence of G-quadruplexes in some of the sequencer reads. The (CG)n repeat does not appear to cause polymerase pausing, but its kinetics signature nevertheless suggests the possibility that alternative nucleotide conformations may sometimes be present. We discuss the implications of using our method to discover DNA sequences capable of forming alternative structures. The analyses presented here can be reproduced on any Pacific Biosciences kinetics data for any DNA pattern of interest using an R package that we have made publicly available.\\\\n\\\\nAuthor SummaryDNA can be found in various forms that differ from the double-helix first discovered by Watson and Crick in 1953. These alternative DNA structures depend on the DNA sequence, and researchers continue to explore which sequences have the potential to form alternative structures. Here we advance the use of Pacific Biosciences sequencer data to explore potential alternative DNA structures. The Pacific Bio-sciences sequencer provides an unprecedented way to examine the interaction between DNA polymerase and DNA by following a single polymerase in real time as it copies a DNA molecule. The pausing of DNA polymerase is a common method for exploring the DNA sequences that have the potential to form alternative DNA structures, and Pacific Biosciences data has previously been used to measure polymerase pausing at a slipped strand structure. DNA polymerase is known to pause at some of these alternative structures, such as the structure known as the G-quadruplex, a DNA structure that has potentially importing regulatory significance. We examine polymerase kinetics around a G-quadruplex, and find evidence of polymerase pausing in the Pacific Biosciences kinetics. We provide a method, with publicly available code, so that others can examine these polymerase kinetics for any sequence of interest.',\n",
       "   'published': '10.1186/s12859-014-0449-0',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000349',\n",
       "   'title': 'Filling up the tree: considering the self-organization of avian roosting behavior',\n",
       "   'authors': 'Bradly J Alicea;',\n",
       "   'author_corresponding': 'Bradly J Alicea',\n",
       "   'author_corresponding_institution': 'Michigan State University',\n",
       "   'date': '2013-11-13',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Zoology ',\n",
       "   'abstract': 'In this paper, models for understanding bird roosting will be considered for purposes of developing better Artificial Life models of complex behavior. Roosting involves multiple flocks of birds picking a single tree limb to rest on for the night, and can be considered an iterative, time-dependent process that unfolds over a 45-minute interval roughly corresponding to twilight. Two models will be used to better understand the main components of this behavior. The constrained dynamics model, which represents continuous random absorption on a one-dimensional lattice, will be used to characterize the dynamics of crowding in the tree structure over time. A second approach involves the relationships between complex networks and roosting behaviors, in particular the evolution of structured networks via rules of incorporation and interaction. Finally, the percolation model will be proposed as a way to bridge behaviors explained by the constrained dynamics and complex network models.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000349',\n",
       "   'title': 'Filling up the tree: considering the self-organization of avian roosting behavior',\n",
       "   'authors': 'Bradly J Alicea;',\n",
       "   'author_corresponding': 'Bradly J Alicea',\n",
       "   'author_corresponding_institution': 'Michigan State University',\n",
       "   'date': '2013-12-01',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'Zoology ',\n",
       "   'abstract': 'In this paper, models for understanding bird roosting will be considered for purposes of developing better Artificial Life models of complex behavior. Roosting involves multiple flocks of birds picking a single tree limb to rest on for the night, and can be considered an iterative, time-dependent process that unfolds over a 45-minute interval roughly corresponding to twilight. Two models will be used to better understand the main components of this behavior. The constrained dynamics model, which represents continuous random absorption on a one-dimensional lattice, will be used to characterize the dynamics of crowding in the tree structure over time. A second approach involves the relationships between complex networks and roosting behaviors, in particular the evolution of structured networks via rules of incorporation and interaction. Finally, the percolation model will be proposed as a way to bridge behaviors explained by the constrained dynamics and complex network models.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001594',\n",
       "   'title': 'Morphometrics of a wild Asian elephant exhibiting disproportionate dwarfism',\n",
       "   'authors': 'Shermin de Silva;Udaya S Weerathunga;Tennekoon Pushpakumara;',\n",
       "   'author_corresponding': 'Shermin de Silva',\n",
       "   'author_corresponding_institution': 'Colorado State University, EFECT, Trunks & Leaves (Inc.)',\n",
       "   'date': '2013-12-24',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Zoology ',\n",
       "   'abstract': 'Dwarfism is a condition characterized by shorter stature, at times accompanied by differential skeletal growth pro-portions relative to the species-typical physical conformation. Causes vary and well-documented in humans as well as certain mammalian species in captive or laboratory conditions, but rarely observed in the wild. Here we report on a single case of apparent dwarfism in a free-ranging adult male Asian elephant (Elephas maximus) in Sri Lanka, comparing physical dimensions to those of other males in the same population, males in other populations, and records in previous literature. The subject was found to have a shoulder height of approximately 195cm, is shorter than the average height of typical mature males, with a body length of 218cm. This ratio of body length to height deviates from what is typically observed, which is approximately 1:1. In absolute height the subject was similar to the attributes of a captive elephant documented in 1955 in Sri Lanka, also said to be a dwarf, however the two specimens differed in the relative proportions of height vs. body length. The subject also exhibits a slight elongation of the skull. We discuss how this phenotype compares to cases of dwarfism in other non-human animals.',\n",
       "   'published': '10.1186/1756-0500-7-933',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000380',\n",
       "   'title': 'A model of flux regulation in the cholesterol biosynthesis pathway: Immune mediated graduated flux reduction versus statin-like led stepped flux reduction',\n",
       "   'authors': 'Steven Watterson;Maria-Luisa Guerriero;Mathieu Blanc;Alexander Mazein;Laurence Loewe;Kevin Robertson;Holly Gibbs;Guanghou Shui;Markus Wenk;Jane Hillston;Peter Ghazal;',\n",
       "   'author_corresponding': 'Steven Watterson',\n",
       "   'author_corresponding_institution': 'University of Ulster',\n",
       "   'date': '2013-11-14',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'Systems Biology ',\n",
       "   'abstract': 'Graphical Abstract\\\\n\\\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=37 SRC=\\\\\"FIGDIR/small/000380_ufig1.gif\\\\\" ALT=\\\\\"Figure 1\\\\\">\\\\nView larger version (14K):\\\\norg.highwire.dtl.DTLVardef@197f26org.highwire.dtl.DTLVardef@1eac3f3org.highwire.dtl.DTLVardef@1e698f1org.highwire.dtl.DTLVardef@42fb4b_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIWe model the cholesterol biosynthesis pathway and its regulation\\\\nC_LIO_LIThe innate immune response leads to a suppression of flux through the pathway\\\\nC_LIO_LIStatin inhibitors show a different mode of suppression to the immune response\\\\nC_LIO_LIStatin inhibitor suppression is less robust and less specific than immune suppression\\\\nC_LI\\\\n\\\\nAsbtractThe cholesterol biosynthesis pathway has recently been shown to play an important role in the innate immune response to viral infection with host protection occurring through a coordinate down regulation of the enzymes catalyzing each metabolic step. In contrast, statin based drugs, which form the principle pharmaceutical agents for decreasing the activity of this pathway, target a single enzyme. Here, we build an ordinary differential equation model of the cholesterol biosynthesis pathway in order to investigate how the two regulatory strategies impact upon the behaviour of the pathway. We employ a modest set of assumptions: that the pathway operates away from saturation, that each metabolite is involved in multiple cellular interactions and that mRNA levels reflect enzyme concentrations. Using data taken from primary bone marrow derived macrophage cells infected with murine cytomegalovirus infection or treated with IFN{gamma}, we show that, under these assumptions, coordinate down regulation of enzyme activity imparts a graduated reduction in flux along the pathway. In contrast, modelling a statin-like treatment that achieves the same degree of down-regulation in cholesterol production, we show that this delivers a step change in flux along the pathway. The graduated reduction mediated by physiological coordinate regulation of multiple enzymes supports a mechanism that allows a greater level of specificity, altering cholesterol levels with less impact upon interactions branching from the pathway, than pharmacological step reductions. We argue that coordinate regulation is likely to show a long-term evolutionary advantage over single enzyme regulation. Finally, the results from our models have implications for future pharmaceutical therapies intended to target cholesterol production with greater specificity and fewer off target effects, suggesting that this can be achieved by mimicking the coordinated down-regulation observed in immunological responses.',\n",
       "   'published': '10.1016/j.biochi.2012.05.024',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000562',\n",
       "   'title': 'A structural classification of candidate oscillators and multistationary systems',\n",
       "   'authors': 'Franco Blanchini;Elisa Franco;Giulia Giordano;',\n",
       "   'author_corresponding': 'Elisa Franco',\n",
       "   'author_corresponding_institution': 'University of California at Riverside',\n",
       "   'date': '2013-11-18',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Systems Biology ',\n",
       "   'abstract': \"Molecular systems are uncertain: the variability of reaction parameters and the presence of unknown interactions can weaken the predictive capacity of solid mathematical models. However, strong conclusions on the admissible dynamic behaviors of a model can often be achieved without detailed knowledge of its specific parameters. In particular, starting with Thomas' conjectures, loop-based criteria have been largely used to characterize oscillatory and multistationary dynamic outcomes in systems with a sign definite Jacobian.\\\\n\\\\nWe build on the rich literature focused on the identification of potential oscillatory and multistationary behaviors based on parameter-free criteria. We propose a classification for sign-definite non autocatalytic biological networks which summarize several existing results in the literature, adding new results when necessary. We define candidate oscillators and multistationary systems based on their admissible transitions to instability. We introduce four categories: strong/weak candidate oscillatory/multistationary systems, which correspond to networks in which all/some of the existing feedback loops are negative/positive. We provide necessary and sufficient conditions characterizing strong and weak candidate oscillators and multistationary systems based on the exclusive or simultaneous presence of positive and negative loops in their linearized dynamics. We also consider the case in which the overall system is the connection of several stable aggregate monotone components, providing conditions in terms of positive/negative loops in a suitable network with aggregate monotone systems as nodes.\\\\n\\\\nMost realistic examples of biological networks fall in the gray area of systems in which both positive and negative cycles are present: therefore, both oscillatory and bistable behavior are in principle possible. Native systems with a large number of components are often interconnections of monotone modules, where negative/positive loops among modules characterize oscillatory and bistable behaviors, in agreement with our results. Finally, we note that many canonical example circuits exhibiting oscillations or bistability fall in the categories of strong candidate oscillators/multistationary systems.\",\n",
       "   'published': '10.1007/s11538-014-0023-y',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000778',\n",
       "   'title': 'Quantifying the turnover of transcriptional subclasses of HIV-1-infected cells',\n",
       "   'authors': 'Christian L Althaus;Beda Joos;Alan S Perelson;Huldrych F Günthard;',\n",
       "   'author_corresponding': 'Christian L Althaus',\n",
       "   'author_corresponding_institution': 'University of Bern',\n",
       "   'date': '2013-11-20',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Systems Biology ',\n",
       "   'abstract': 'HIV-1-infected cells in peripheral blood can be grouped into different transcriptional subclasses. Quantifying the turnover of these cellular subclasses can provide important insights into the viral life cycle and the generation and maintenance of latently infected cells. We used previously published data from five patients chronically infected with HIV-1 that initiated combination antiretroviral therapy (cART). Patient-matched PCR for unspliced and multiply spliced viral RNAs combined with limiting dilution analysis provided measurements of transcriptional profiles at the single cell level. Furthermore, measurement of intracellular transcripts and extracellular virion-enclosed HIV-1 RNA allowed us to distinguish productive from non-productive cells. We developed a mathematical model describing the dynamics of plasma virus and the transcriptional subclasses of HIV-1-infected cells. Fitting the model to the data allowed us to better understand the phenotype of different transcriptional subclasses and their contribution to the overall turnover of HIV-1 before and during cART. The average number of virus-producing cells in peripheral blood is small during chronic infection (25.7 cells ml-1). We find that 14.0%, 0.3% and 21.2% of infected cells become defectively, latently and persistently infected cells, respectively. Assuming that the infection is homogenous throughout the body, we estimate an average in vivo viral burst size of 2.1 x 104 virions per cell. Our study provides novel quantitative insights into the turnover and development of different subclasses of HIV-1-infected cells. The model predicts that the pool of latently infected cells becomes rapidly established during the first months of acute infection and continues to increase slowly during the first years of chronic infection. Having a detailed understanding of this process will be useful for the evaluation of viral eradication strategies that aim to deplete the latent reservoir of HIV-1.\\\\n\\\\nAuthor SummaryGaining a quantitative understanding of the development and turnover of different HIV-1-infected subpopulations of cells is crucial to improve the outcome of patients on combination antiretroviral therapy (cART). The population of latently infected cells is of particular interest as they represent the major barrier to a cure of HIV-1 infection. We developed a mathematical model that describes the dynamics of different transcriptionally active subclasses of HIV-1-infected cells and the viral load in peripheral blood. The model was fitted to previously published data from five chronically HIV-1-infected patients starting cART. This allowed us to estimate critical parameters of the within-host dynamics of HIV-1, such as the the number of virions produced by a single infected cell. The model further allowed investigation of HIV-1 dynamics during the acute phase. Computer simulations predict that latently infected cells become rapidly established during the first months of acute infection and continue to increase slowly during the first years of chronic infection. This illustrates the opportunity for strategies that aim to eradicate the virus during early cART as the pool of HIV-1 infected cells is substantially smaller during acute infection than during chronic infection.',\n",
       "   'published': '10.1371/journal.pcbi.1003871',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000778',\n",
       "   'title': 'Quantifying the turnover of transcriptional subclasses of HIV-1-infected cells',\n",
       "   'authors': 'Christian L Althaus;Beda Joos;Alan S Perelson;Huldrych F Günthard;',\n",
       "   'author_corresponding': 'Christian L Althaus',\n",
       "   'author_corresponding_institution': 'University of Bern',\n",
       "   'date': '2014-01-18',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Systems Biology ',\n",
       "   'abstract': 'HIV-1-infected cells in peripheral blood can be grouped into different transcriptional subclasses. Quantifying the turnover of these cellular subclasses can provide important insights into the viral life cycle and the generation and maintenance of latently infected cells. We used previously published data from five patients chronically infected with HIV-1 that initiated combination antiretroviral therapy (cART). Patient-matched PCR for unspliced and multiply spliced viral RNAs combined with limiting dilution analysis provided measurements of transcriptional profiles at the single cell level. Furthermore, measurement of intracellular transcripts and extracellular virion-enclosed HIV-1 RNA allowed us to distinguish productive from non-productive cells. We developed a mathematical model describing the dynamics of plasma virus and the transcriptional subclasses of HIV-1-infected cells. Fitting the model to the data allowed us to better understand the phenotype of different transcriptional subclasses and their contribution to the overall turnover of HIV-1 before and during cART. The average number of virus-producing cells in peripheral blood is small during chronic infection (25.7 cells ml-1). We find that 14.0%, 0.3% and 21.2% of infected cells become defectively, latently and persistently infected cells, respectively. Assuming that the infection is homogenous throughout the body, we estimate an average in vivo viral burst size of 2.1 x 104 virions per cell. Our study provides novel quantitative insights into the turnover and development of different subclasses of HIV-1-infected cells. The model predicts that the pool of latently infected cells becomes rapidly established during the first months of acute infection and continues to increase slowly during the first years of chronic infection. Having a detailed understanding of this process will be useful for the evaluation of viral eradication strategies that aim to deplete the latent reservoir of HIV-1.\\\\n\\\\nAuthor SummaryGaining a quantitative understanding of the development and turnover of different HIV-1-infected subpopulations of cells is crucial to improve the outcome of patients on combination antiretroviral therapy (cART). The population of latently infected cells is of particular interest as they represent the major barrier to a cure of HIV-1 infection. We developed a mathematical model that describes the dynamics of different transcriptionally active subclasses of HIV-1-infected cells and the viral load in peripheral blood. The model was fitted to previously published data from five chronically HIV-1-infected patients starting cART. This allowed us to estimate critical parameters of the within-host dynamics of HIV-1, such as the the number of virions produced by a single infected cell. The model further allowed investigation of HIV-1 dynamics during the acute phase. Computer simulations predict that latently infected cells become rapidly established during the first months of acute infection and continue to increase slowly during the first years of chronic infection. This illustrates the opportunity for strategies that aim to eradicate the virus during early cART as the pool of HIV-1 infected cells is substantially smaller during acute infection than during chronic infection.',\n",
       "   'published': '10.1371/journal.pcbi.1003871',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000927',\n",
       "   'title': 'Investigating the relation between stochastic differentiation and homeostasis in intestinal crypts via multiscale modeling',\n",
       "   'authors': 'Alex Graudenzi;Giulio Caravagna;Giovanni De Matteis;Marco Antoniotti;',\n",
       "   'author_corresponding': 'Alex Graudenzi',\n",
       "   'author_corresponding_institution': 'Dept. of Informatics, Systems and Communication',\n",
       "   'date': '2013-11-25',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Systems Biology ',\n",
       "   'abstract': 'Colorectal tumors originate and develop within intestinal crypts. Even though some of the essential phenomena that characterize crypt structure and dynamics have been effectively described in the past, the relation between the differentiation process and the overall crypt homeostasis is still partially understood. We here investigate this relation and other important biological phenomena by introducing a novel multiscale model that combines a morphological description of the crypt with a gene regulation model: the emergent dynamical behavior of the underlying gene regulatory network drives cell growth and differentiation processes, linking the two distinct spatio-temporal levels. The model relies on a few a priori assumptions, yet accounting for several key processes related to crypt functioning, such as: dynamic gene activation patterns, stochastic differentiation, signaling pathways ruling cell adhesion properties, cell displacement, cell growth, mitosis, apoptosis and the presence of biological noise.\\\\n\\\\nWe show that this modeling approach captures the major dynamical phenomena that characterize the regular physiology of crypts, such as cell sorting, coordinate migration, dynamic turnover, stem cell niche maintenance and clonal expansion. All in all, the model suggests that the process of stochastic differentiation might be sufficient to drive the crypt to homeostasis, under certain crypt configurations. Besides, our approach allows to make precise quantitative inferences that, when possible, were matched to the current biological knowledge and it permits to investigate the role of gene-level perturbations, with reference to cancer development. We also remark the theoretical framework is general and may applied to different tissues, organs or organisms.',\n",
       "   'published': '10.1371/journal.pone.0097272',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000745',\n",
       "   'title': 'ROS accumulation in cotton ovule epidermal cells is necessary for fiber initiation',\n",
       "   'authors': 'mingxiong pang;Nickolas Sanford;Thea Wilkins;',\n",
       "   'author_corresponding': 'mingxiong pang',\n",
       "   'author_corresponding_institution': 'Texas Tech University',\n",
       "   'date': '2013-11-19',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Plant Biology ',\n",
       "   'abstract': 'Cotton (Gossypium hirsutum) fiber, an extremely elongated and thickened single cell of the seed epidermis, is the worlds most important natural and economical textile fiber. Unlike Arabidopsis leaf trichomes, fiber initials are randomly developed and frequently form in adjacent seed epidermal cells and follow no apparent pattern. Numerous publications suggested cotton fiber development shares a similar mechanism with Arabidopsis leaf trichome development. Here we show that H2O2 accumulation in cotton ovule epidermal cells by NBT staining ovules at different development stages between TM1 and N1n2, a lintless-fuzzless doubled mutant originated from TM1. In contrast, Arabidopsis and cotton leaf trichomes do not show H2O2 content. By adding DPI (H2O2 inhibitor) and SHAM (H2O2 activator) in vitro ovule cultures, we show fiber initiation directly involves with H2O2 accumulation. We propose that the directional accumulation of H2O2 in cotton ovule epidermal cell is the drive for fiber initiation, elongation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000745',\n",
       "   'title': 'ROS accumulation in cotton ovule epidermal cells is necessary for fiber initiation',\n",
       "   'authors': 'mingxiong pang;Nickolas Sanford;Thea Wilkins;',\n",
       "   'author_corresponding': 'mingxiong pang',\n",
       "   'author_corresponding_institution': 'Texas Tech University',\n",
       "   'date': '2013-11-20',\n",
       "   'version': '2',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Plant Biology ',\n",
       "   'abstract': 'Cotton (Gossypium hirsutum) fiber, an extremely elongated and thickened single cell of the seed epidermis, is the worlds most important natural and economical textile fiber. Unlike Arabidopsis leaf trichomes, fiber initials are randomly developed and frequently form in adjacent seed epidermal cells and follow no apparent pattern. Numerous publications suggested cotton fiber development shares a similar mechanism with Arabidopsis leaf trichome development. Here we show that H2O2 accumulation in cotton ovule epidermal cells by NBT staining ovules at different development stages between TM1 and N1n2, a lintless-fuzzless doubled mutant originated from TM1. In contrast, Arabidopsis and cotton leaf trichomes do not show H2O2 content. By adding DPI (H2O2 inhibitor) and SHAM (H2O2 activator) in vitro ovule cultures, we show fiber initiation directly involves with H2O2 accumulation. We propose that the directional accumulation of H2O2 in cotton ovule epidermal cell is the drive for fiber initiation, elongation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000638',\n",
       "   'title': 'Effects of metsulfuron-methyl on aquatic plant (Lemna gibba L.) and recovery from after prolonged exposure under rice cropping conditions',\n",
       "   'authors': 'Raja Rajeswari;Atmakuru Ramesh;',\n",
       "   'author_corresponding': 'Atmakuru Ramesh',\n",
       "   'author_corresponding_institution': 'International Institute of bio-technology and toxicology',\n",
       "   'date': '2013-11-18',\n",
       "   'version': '1',\n",
       "   'type': 'Contradictory Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Plant Biology ',\n",
       "   'abstract': 'The effects and potential recovery of aquatic plant Lemna gibba exposed to a sulfonyl urea herbicide metsulfuron-methyl (MSM) for 120 days under rice cropping condition was investigated. The frond number was decreased by day 15 at the concentration 11 {micro}g/L and 100% inhibition on growth rate of Lemna was observed. Continuous decrease of frond number by day 50 at below the detectable level of residues exhibited symptoms (chlorosis) of MSM toxicity. Toxicity was assessed on the basis of toxicity index (TI) value, growth rate, yield and pigment contents (chlorophyll, carotene, total carotenoid and xanthophyll) of treated samples compared with untreated control. The observed value of 0.698 {micro}g/g chlorophyll a, 0.263 {micro}g/g chlorophyll b, 0.147 {micro}g/g carotene, 1.620 {micro}g/g total carotenoid and 1.473 {micro}g/g xanthophyll contents in treated samples was statistically significantly different from control value of 4.366 {micro}g/g chlorophyll a, 3.132 {micro}g/g chlorophyll b, 0.796 {micro}g/g carotene, 17.755 {micro}g/g total carotenoid and 16.937 {micro}g/g xanthophyll contents by day 50 samples. After prolonged exposure, growth rate, yield and pigment content for the treated samples recovered to control levels on day 120. The obtained data indicate the application of aquatic plant Lemna gibba as sensitive biomarker of water quality as well as the significance of selected biological parameters in the reliable assessment of toxic potential of MSM under rice cropping condition.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/000638',\n",
       "   'title': 'Effects of metsulfuron-methyl on aquatic plant (Lemna gibba L.) and recovery from after prolonged exposure under rice cropping conditions',\n",
       "   'authors': 'Raja Rajeswari;Atmakuru Ramesh;',\n",
       "   'author_corresponding': 'Atmakuru Ramesh',\n",
       "   'author_corresponding_institution': 'International Institute of bio-technology and toxicology',\n",
       "   'date': '2013-11-22',\n",
       "   'version': '2',\n",
       "   'type': 'Contradictory Results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'Plant Biology ',\n",
       "   'abstract': 'The effects and potential recovery of aquatic plant Lemna gibba exposed to a sulfonyl urea herbicide metsulfuron-methyl (MSM) for 120 days under rice cropping condition was investigated. The frond number was decreased by day 15 at the concentration 11 {micro}g/L and 100% inhibition on growth rate of Lemna was observed. Continuous decrease of frond number by day 50 at below the detectable level of residues exhibited symptoms (chlorosis) of MSM toxicity. Toxicity was assessed on the basis of toxicity index (TI) value, growth rate, yield and pigment contents (chlorophyll, carotene, total carotenoid and xanthophyll) of treated samples compared with untreated control. The observed value of 0.698 {micro}g/g chlorophyll a, 0.263 {micro}g/g chlorophyll b, 0.147 {micro}g/g carotene, 1.620 {micro}g/g total carotenoid and 1.473 {micro}g/g xanthophyll contents in treated samples was statistically significantly different from control value of 4.366 {micro}g/g chlorophyll a, 3.132 {micro}g/g chlorophyll b, 0.796 {micro}g/g carotene, 17.755 {micro}g/g total carotenoid and 16.937 {micro}g/g xanthophyll contents by day 50 samples. After prolonged exposure, growth rate, yield and pigment content for the treated samples recovered to control levels on day 120. The obtained data indicate the application of aquatic plant Lemna gibba as sensitive biomarker of water quality as well as the significance of selected biological parameters in the reliable assessment of toxic potential of MSM under rice cropping condition.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/001248',\n",
       "   'title': 'A conserved oomycete CRN effector targets and modulates tomato TCP14-2 to enhance virulence',\n",
       "   'authors': 'Remco Stam;Graham Motion;Petra C. Boevink;Edgar Huitema;',\n",
       "   'author_corresponding': 'Edgar Huitema',\n",
       "   'author_corresponding_institution': 'University of Dundee',\n",
       "   'date': '2013-12-11',\n",
       "   'version': '1',\n",
       "   'type': 'New Results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'Plant Biology ',\n",
       "   'abstract': 'Phytophthora spp. secrete vast arrays of effector molecules upon infection. A main class of intracellular effectors are the CRNs. They are translocated into the host cell and specifically localise to the nucleus where they are thought to perturb many different cellular processes. Although CRN proteins have been implicated as effectors, direct evidence of CRN mediated perturbation of host processes has been lacking. Here we show that a conserved CRN effector from P. capsici directly binds to tomato transcription factor SlTCP14-2. Previous studies in Arabidopsis thaliana have revealed that transcription factor TCP14 may be key immune signalling protein, targeted by effectors from divergent species. We extend on our understanding of TCP targeting by pathogen effectors by showing that the P. capsici effector CRN12_997 binds to SlTCP14-2 in plants. SlTCP14-2 over-expression enhances immunity to P. capsici, a phenotypic outcome that can be abolished by co-expression of CRN12_997.\\\\n\\\\nWe show that in the presence of CRN12_997, SlTCP14-2 association with nuclear chromatin is diminished, resulting in altered SlTCP14 subnuclear localisation. These results suggest that CRN12_997 prevents SlTCP14 from positively regulating defence against P. capsici. Our work demonstrates a direct interaction between an oomycete CRN and a host target required for suppression of immunity. Collectively, our results hint at a virulence strategy that is conserved within the oomycetes and may allow engineering of resistance to a wide range of crop pathogens.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'}]}"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "j"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###### Downloaded en_ner_bc5cdr_md and en_ner_bionlp13cg_md from SciSpacy. Maybe I can use a combination of both but mainly bionlp.\n",
    "#From bc5cdr only take disease names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "ner_bio = spacy.load('en_ner_bionlp13cg_md')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "ner_bc = spacy.load('en_ner_bc5cdr_md')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "abst = ner_bio(example_abst)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "b_abst = ner_bc(example_abst)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "OAs CHEMICAL\n",
      "MTL deficits DISEASE\n"
     ]
    }
   ],
   "source": [
    "for ent in b_abst.ents:\n",
    "    print(ent.text, ent.label_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "def run_models(abstract):\n",
    "    \"\"\"\n",
    "    Runs two Spacy models to recognize NER\n",
    "    Returns bionlp and bc5cdr output\n",
    "    \"\"\"\n",
    "    bio_abst = ner_bio(abstract)\n",
    "    bc_abst = ner_bc(abstract)\n",
    "    \n",
    "    return bio_abst, bc_abst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "bio, bc = run_models(example_abst)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "def entity_dict(tok_abst):\n",
    "    tok_dict = {}\n",
    "    for ent in tok_abst.ents:\n",
    "        tok_dict[f'{ent.text}'] = ent.label_\n",
    "    return tok_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'OAs': 'SIMPLE_CHEMICAL',\n",
       " 'prefrontal cortex': 'CANCER',\n",
       " 'PFC': 'SIMPLE_CHEMICAL',\n",
       " 'YAs': 'CANCER',\n",
       " 'brain': 'ORGAN',\n",
       " 'brain network': 'MULTI-TISSUE_STRUCTURE',\n",
       " 'network': 'TISSUE',\n",
       " 'medial temporal lobe': 'TISSUE',\n",
       " 'MTL': 'CANCER'}"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "entity_dict(bio)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(OAs,\n",
       " prefrontal cortex,\n",
       " PFC,\n",
       " YAs,\n",
       " PFC,\n",
       " brain,\n",
       " PFC,\n",
       " OAs,\n",
       " YAs,\n",
       " brain network,\n",
       " YAs,\n",
       " OAs,\n",
       " network,\n",
       " PFC,\n",
       " brain network,\n",
       " OAs,\n",
       " PFC,\n",
       " PFC,\n",
       " OAs,\n",
       " PFC,\n",
       " OAs,\n",
       " medial temporal lobe,\n",
       " MTL,\n",
       " PFC,\n",
       " OAs,\n",
       " MTL)"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def filter_bio(dictionary):\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
