{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 283,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_pickle('../Pickles/Pic/Biophysics .pkl')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [],
   "source": [
    "df2 = pd.read_pickle('../Pickles/Biophysics .pkl')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.concat([df, df2], axis = 0, ignore_index = True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = df.fillna(0)\n",
    "df_temp = df.drop(['unique_id', 'doi', 'version'], axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "UU = np.zeros((len(df_temp), len(df_temp)))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [],
   "source": [
    "UU = pd.DataFrame(UU, index=df_temp.index, columns=df_temp.index)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "from scipy.spatial import distance\n",
    "from tqdm import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  0%|          | 0/4432 [00:00<?, ?it/s]/anaconda3/envs/scispacy/lib/python3.6/site-packages/scipy/spatial/distance.py:720: RuntimeWarning: invalid value encountered in double_scalars\n",
      "  dist = 1.0 - uv / np.sqrt(uu * vv)\n",
      "100%|██████████| 4432/4432 [00:08<00:00, 552.96it/s]\n"
     ]
    }
   ],
   "source": [
    "u = 0\n",
    "for v in tqdm(UU.columns):\n",
    "        # 2. step: calculate similarities\n",
    "    UU.loc[u, v] = 1-distance.correlation(df_temp.loc[u],\n",
    "                                                    df_temp.loc[v])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 284,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>prostate cancer cell lines</th>\n",
       "      <th>LNCaP</th>\n",
       "      <th>PC3</th>\n",
       "      <th>\\\"second order\\</th>\n",
       "      <th>cellular line PC3</th>\n",
       "      <th>tumors.\\n\\nHighlightsO_LICancer</th>\n",
       "      <th>system.\\nC_LIO_LIProstate cancer cell lines</th>\n",
       "      <th>Gompertz dynamics\\nC_LIO_LIProstate cancer cell lines</th>\n",
       "      <th>matrix</th>\n",
       "      <th>...</th>\n",
       "      <th>tea</th>\n",
       "      <th>E. coli</th>\n",
       "      <th>single cell</th>\n",
       "      <th>Tat</th>\n",
       "      <th>inner membrane</th>\n",
       "      <th>TatA</th>\n",
       "      <th>TatB</th>\n",
       "      <th>TatBC</th>\n",
       "      <th>doi</th>\n",
       "      <th>version</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5eaa034c9b21ead543e3fb49</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/011189</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5eaa034c9b21ead543e3fb82</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/012930</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5eaa034c9b21ead543e3fb83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/012955</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5eaa03509b21ead543e3fc08</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/014449</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5eaa03559b21ead543e3fc23</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/014704</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>5eaa04819b21ead543e41662</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078352</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>5eaa04819b21ead543e41685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078626</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>5eaa04819b21ead543e41688</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078543</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>5eaa04819b21ead543e4169c</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078998</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>5eaa04869b21ead543e416e0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>10.1101/079715</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>202 rows × 883 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        index  prostate cancer cell lines  LNCaP  PC3  \\\n",
       "0    5eaa034c9b21ead543e3fb49                         1.0    3.0  2.0   \n",
       "1    5eaa034c9b21ead543e3fb82                         NaN    NaN  NaN   \n",
       "2    5eaa034c9b21ead543e3fb83                         NaN    NaN  NaN   \n",
       "3    5eaa03509b21ead543e3fc08                         NaN    NaN  NaN   \n",
       "4    5eaa03559b21ead543e3fc23                         NaN    NaN  NaN   \n",
       "..                        ...                         ...    ...  ...   \n",
       "197  5eaa04819b21ead543e41662                         NaN    NaN  NaN   \n",
       "198  5eaa04819b21ead543e41685                         NaN    NaN  NaN   \n",
       "199  5eaa04819b21ead543e41688                         NaN    NaN  NaN   \n",
       "200  5eaa04819b21ead543e4169c                         NaN    NaN  NaN   \n",
       "201  5eaa04869b21ead543e416e0                         NaN    NaN  NaN   \n",
       "\n",
       "     \\\"second order\\  cellular line PC3  tumors.\\n\\nHighlightsO_LICancer  \\\n",
       "0                1.0                1.0                              1.0   \n",
       "1                NaN                NaN                              NaN   \n",
       "2                NaN                NaN                              NaN   \n",
       "3                NaN                NaN                              NaN   \n",
       "4                NaN                NaN                              NaN   \n",
       "..               ...                ...                              ...   \n",
       "197              NaN                NaN                              NaN   \n",
       "198              NaN                NaN                              NaN   \n",
       "199              NaN                NaN                              NaN   \n",
       "200              NaN                NaN                              NaN   \n",
       "201              NaN                NaN                              NaN   \n",
       "\n",
       "     system.\\nC_LIO_LIProstate cancer cell lines  \\\n",
       "0                                            1.0   \n",
       "1                                            NaN   \n",
       "2                                            NaN   \n",
       "3                                            NaN   \n",
       "4                                            NaN   \n",
       "..                                           ...   \n",
       "197                                          NaN   \n",
       "198                                          NaN   \n",
       "199                                          NaN   \n",
       "200                                          NaN   \n",
       "201                                          NaN   \n",
       "\n",
       "     Gompertz dynamics\\nC_LIO_LIProstate cancer cell lines  matrix  ...  tea  \\\n",
       "0                                                  1.0         NaN  ...  NaN   \n",
       "1                                                  NaN         1.0  ...  NaN   \n",
       "2                                                  NaN         NaN  ...  NaN   \n",
       "3                                                  NaN         NaN  ...  NaN   \n",
       "4                                                  NaN         NaN  ...  NaN   \n",
       "..                                                 ...         ...  ...  ...   \n",
       "197                                                NaN         NaN  ...  NaN   \n",
       "198                                                NaN         NaN  ...  NaN   \n",
       "199                                                NaN         NaN  ...  1.0   \n",
       "200                                                NaN         NaN  ...  NaN   \n",
       "201                                                NaN         NaN  ...  NaN   \n",
       "\n",
       "     E. coli  single cell  Tat  inner membrane  TatA  TatB  TatBC  \\\n",
       "0        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "1        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "2        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "3        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "4        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "..       ...          ...  ...             ...   ...   ...    ...   \n",
       "197      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "198      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "199      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "200      1.0          1.0  NaN             NaN   NaN   NaN    NaN   \n",
       "201      2.0          NaN  4.0             6.0   8.0   1.0    2.0   \n",
       "\n",
       "                doi  version  \n",
       "0    10.1101/011189        2  \n",
       "1    10.1101/012930        1  \n",
       "2    10.1101/012955        1  \n",
       "3    10.1101/014449        1  \n",
       "4    10.1101/014704        1  \n",
       "..              ...      ...  \n",
       "197  10.1101/078352        1  \n",
       "198  10.1101/078626        1  \n",
       "199  10.1101/078543        1  \n",
       "200  10.1101/078998        1  \n",
       "201  10.1101/079715        1  \n",
       "\n",
       "[202 rows x 883 columns]"
      ]
     },
     "execution_count": 284,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "neighbors = UU.loc[0].sort_values(ascending=False)[1:50]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1       1.000000\n",
       "2570    0.838424\n",
       "2444    0.838424\n",
       "908     0.833767\n",
       "2224    0.816608\n",
       "1791    0.816605\n",
       "2393    0.809409\n",
       "2294    0.809187\n",
       "2285    0.809187\n",
       "1354    0.808398\n",
       "1346    0.808398\n",
       "1952    0.802400\n",
       "1993    0.789722\n",
       "1973    0.789722\n",
       "1965    0.789722\n",
       "2796    0.789453\n",
       "418     0.782420\n",
       "1513    0.776121\n",
       "1823    0.776077\n",
       "1715    0.776077\n",
       "1168    0.769894\n",
       "2321    0.745317\n",
       "1069    0.741040\n",
       "2585    0.733291\n",
       "2027    0.732843\n",
       "1444    0.728257\n",
       "287     0.719060\n",
       "2646    0.717923\n",
       "2651    0.717923\n",
       "2756    0.705603\n",
       "97      0.701957\n",
       "2309    0.698364\n",
       "1554    0.691343\n",
       "332     0.691343\n",
       "701     0.691343\n",
       "2337    0.675034\n",
       "1269    0.669383\n",
       "2675    0.663874\n",
       "2322    0.662178\n",
       "2146    0.662067\n",
       "757     0.660018\n",
       "2107    0.658921\n",
       "2211    0.653251\n",
       "804     0.653251\n",
       "2662    0.652346\n",
       "2580    0.650384\n",
       "1620    0.650349\n",
       "3034    0.631090\n",
       "128     0.620638\n",
       "Name: 0, dtype: float64"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "neighbors"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "new_neighbor = []\n",
    "query_entities = list(df.iloc[0][df.iloc[0]!=0].index)\n",
    "for i in list(neighbors.index):\n",
    "    neighbor_entities = list(df.iloc[i][df.iloc[i]!=0].index)\n",
    "    intersecting_ent = list(set(query_entities).intersection(neighbor_entities))\n",
    "    if len(intersecting_ent) >= 6:\n",
    "        new_neighbor.append(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[1]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_neighbor"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_meta_refined = pd.DataFrame(columns = df_meta.columns)\n",
    "for i in new_neighbor:\n",
    "    recom_article_meta = pd.DataFrame(df_meta.iloc[i]).T\n",
    "    df_meta_refined = pd.concat([df_meta_refined, recom_article_meta], axis = 0)\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>version</th>\n",
       "      <th>doi</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>author_corresponding</th>\n",
       "      <th>author_corresponding_institution</th>\n",
       "      <th>date</th>\n",
       "      <th>type</th>\n",
       "      <th>category</th>\n",
       "      <th>published</th>\n",
       "      <th>server</th>\n",
       "      <th>unique_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>10.1101/080440</td>\n",
       "      <td>The large terminase DNA packaging motor grips ...</td>\n",
       "      <td>Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...</td>\n",
       "      <td>Brian A. Kelch</td>\n",
       "      <td>University of Massachusetts Medical School</td>\n",
       "      <td>2016-10-19</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1093/nar/gkw1356</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04949b21ead543e417cc</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  version             doi                                              title  \\\n",
       "1       2  10.1101/080440  The large terminase DNA packaging motor grips ...   \n",
       "\n",
       "                                             authors author_corresponding  \\\n",
       "1  Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...       Brian A. Kelch   \n",
       "\n",
       "             author_corresponding_institution        date         type  \\\n",
       "1  University of Massachusetts Medical School  2016-10-19  new results   \n",
       "\n",
       "       category            published   server                 unique_id  \n",
       "1  biochemistry  10.1093/nar/gkw1356  biorxiv  5eaa04949b21ead543e417cc  "
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_meta_refined"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_doi_publ(doi):\n",
    "    \"\"\"\n",
    "    Parameters\n",
    "    ----------\n",
    "        doi: str\n",
    "    Returns\n",
    "    -------\n",
    "        found: bool\n",
    "        item: dict\n",
    "            Response from crossref\n",
    "    \"\"\"\n",
    "    bare_url = \"http://api.crossref.org/\"\n",
    "    url = \"{}works/{}\"\n",
    "    url = url.format(bare_url, doi)\n",
    "    r = requests.get(url)\n",
    "    found = False if r.status_code != 200 else True\n",
    "    item = r.json()\n",
    "\n",
    "    return found, item"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_doi_refined = df_meta_refined[df_meta_refined['published']!= 'NA']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Nucleic Acids Res']"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Where is it published\n",
    "get_doi_publ(df_doi_refined['published'].iloc[0])[1]['message']['short-container-title']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "5"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#How many times is it published\n",
    "get_doi_publ(df_doi_refined['published'].iloc[0])[1]['message']['is-referenced-by-count']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_meta = pd.read_pickle('../Pickles/biochemistry_meta.pkl')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "terminase                           1\n",
       "TerL                                6\n",
       "ATPase                              2\n",
       "DNA                                11\n",
       "doi                    10.1101/080440\n",
       "unique_id    5eaa048f9b21ead543e417a7\n",
       "version                             1\n",
       "Name: 0, dtype: object"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.iloc[0][df.iloc[0]!=0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "terminase                           1\n",
       "TerL                                6\n",
       "ATPase                              2\n",
       "DNA                                11\n",
       "doi                    10.1101/080440\n",
       "unique_id    5eaa04949b21ead543e417cc\n",
       "version                             2\n",
       "Name: 1, dtype: object"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.iloc[1][df.iloc[1]!=0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "version                                                                             2\n",
       "doi                                                                    10.1101/080440\n",
       "title                               The large terminase DNA packaging motor grips ...\n",
       "authors                             Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...\n",
       "author_corresponding                                                   Brian A. Kelch\n",
       "author_corresponding_institution           University of Massachusetts Medical School\n",
       "date                                                                       2016-10-19\n",
       "type                                                                      new results\n",
       "category                                                                 biochemistry\n",
       "published                                                         10.1093/nar/gkw1356\n",
       "server                                                                        biorxiv\n",
       "unique_id                                                    5eaa04949b21ead543e417cc\n",
       "Name: 1, dtype: object"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_meta.iloc[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "PATH = '/Users/iremn/PythonClass/Spiced/Personal_Projects/BioReco/Pickles'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "file_list = os.listdir(PATH)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "cat_str = category[:4].lower()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "r = re.compile(cat_str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<filter at 0x10bbf41d0>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "filter(r.match, file_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "newlist = list(filter(r.match, file_list))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "newlist"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prototype User input"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "def entity_dict(tokenized_abst):\n",
    "    \"\"\"Build a dictionary of k:entity and v:ent_tag\"\"\"\n",
    "    tok_dict = {}\n",
    "    for ent in tokenized_abst.ents:\n",
    "        tok_dict[f'{ent.text}'] = ent.label_\n",
    "    return tok_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "def ent_count_dict(tokenized_abst):\n",
    "    \"\"\"Returns an entity count dictionary from tokenized abstract\"\"\"\n",
    "    ent_list = []\n",
    "    for ent in tokenized_abst.ents:\n",
    "        ent_list.append(ent)\n",
    "    #Convert them to string\n",
    "    ent_list = [str(i) for i in ent_list]\n",
    "    #Build an entity dictionary with labels\n",
    "    ent_dict = entity_dict(tokenized_abst)\n",
    "    #Get the keys to loop the entity list for counts\n",
    "    ent_keys = ent_dict.keys()\n",
    "    #Make a new dictionary of (k:ent, v: count)\n",
    "    ent_app = {}\n",
    "    for element in ent_keys:\n",
    "        ent_app[element] = ent_list.count(element)\n",
    "    return ent_app"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pymongo\n",
    "client = pymongo.MongoClient()\n",
    "db = client.BioReco\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "def query_mongo_article(doi):\n",
    "    \"\"\"Query Mongo DB with a specific doi\"\"\"\n",
    "    query_cat = list(db.BioReco_raw.find({'doi': f'{doi}'}))\n",
    "    size = len(query_cat)\n",
    "    print(f'There are {size} articles in {doi} category')\n",
    "    return query_cat"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {},
   "outputs": [],
   "source": [
    "def ET(query, category_df):\n",
    "    \"\"\"Takes the category and returns the article-entity df\"\"\"\n",
    "    dummy = np.zeros((1, len(category_df.columns)))\n",
    "    df = pd.DataFrame(dummy, index =[9999999], columns = category_df.columns)\n",
    "    #Fetch the absract and doi first\n",
    "    abstract = query[0]['abstract']\n",
    "    #Remove links\n",
    "    abstract = re.sub(r'^(http:\\/\\/www\\.|https:\\/\\/www\\.|http:\\/\\/|https:\\/\\/)?[a-z0-9]+([\\-\\.]{1}[a-z0-9]+)*\\.[a-z]{2,5}(:[0-9]{1,5})?(\\/.*)?$', '', abstract)\n",
    "    #Tokenize abstract\n",
    "    tok_abstr = ner_bio(abstract)\n",
    "    #Build entity-count dictionary\n",
    "    user_dict = ent_count_dict(tok_abstr)\n",
    "    print(user_dict)\n",
    "    for key in user_dict.keys():\n",
    "        df[key] = user_dict[key]\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {},
   "outputs": [],
   "source": [
    "def create_aa_matrix(user_input, df, keyword):\n",
    "    \"\"\"Takes the category df and makes and article-article matrix\"\"\"\n",
    "    #Take doi if keyword is False, extract the abstract and model.\n",
    "    #Create a user df with a single entry\n",
    "    if keyword == False:\n",
    "        doi = str(user_input)\n",
    "        doi_query = query_mongo_article(doi)\n",
    "        #If there are several versions of the queried article, take the last one\n",
    "        if len(doi_query) > 1:\n",
    "            doi_query = [doi_query[-1]]\n",
    "            df_user = ET(doi_query, df)\n",
    "            df_user['doi'] = doi_query[0]['doi']\n",
    "            df_user['version'] = doi_query[0]['version']\n",
    "        else:\n",
    "            df_user = ET(doi_query, df)\n",
    "            df_user['doi'] = doi_query[0]['doi']\n",
    "            df_user['version'] = doi_query[0]['version']\n",
    "    else:\n",
    "        user_keyword = str(user_input)\n",
    "        user_keyword = re.sub(r'^(http:\\/\\/www\\.|https:\\/\\/www\\.|http:\\/\\/|https:\\/\\/)?[a-z0-9]+([\\-\\.]{1}[a-z0-9]+)*\\.[a-z]{2,5}(:[0-9]{1,5})?(\\/.*)?$', '', user_keyword)\n",
    "        tok_kywd = ner_bio(user_keyword)\n",
    "        dummy = np.zeros((1, len(df.columns)))\n",
    "        df_user = pd.DataFrame(dummy, index =[9999999], columns = df.columns)\n",
    "        user_dict = ent_count_dict(tok_kywd)\n",
    "        for key in user_dict.keys():\n",
    "            df_user[key] = user_dict[key]\n",
    "    print(df_user)\n",
    "    #Fill the matrix with zeros for NaN\n",
    "    df = df.fillna(0)\n",
    "    #Create a temp df. Unique_ID may be index, change here accordingly\n",
    "    df_user_temp = df_user.drop(['unique_id', 'doi', 'version'], axis=1)\n",
    "    df_temp = df.drop(['unique_id', 'doi', 'version'], axis=1)\n",
    "    #Concat it with the user df\n",
    "    assert len(set(df_temp.columns).intersection((df_user_temp.columns))) >= 1\n",
    "    df_temp = pd.concat([df_temp, df_user_temp], axis = 0, ignore_index=False)\n",
    "    #Create an empty A-A matrix\n",
    "    AA = np.zeros((len(df_temp), len(df_temp)))\n",
    "    #Convert into df\n",
    "    AA = pd.DataFrame(AA, index=df_temp.index, columns=df_temp.index)\n",
    "    # #Calculate similarities\n",
    "    u = 9999999\n",
    "    for v in AA.columns:\n",
    "        AA.loc[u, v] = 1-distance.correlation(df_temp.loc[u], df_temp.loc[v])\n",
    "    active_user = 9999999\n",
    "    neighbors = AA.loc[active_user].sort_values(\n",
    "                                     ascending=False)[1:51]\n",
    "    neighbors = neighbors.to_frame()\n",
    "    #Refine neighbors\n",
    "    print(type(neighbors))\n",
    "    #Obtain overlapping number of entities\n",
    "    query_entities = list(df_user.loc[active_user][df_user.loc[active_user]!=0].index)\n",
    "    overlapping_count=[]\n",
    "    for i in list(neighbors.index):\n",
    "        neighbor_entities = list(df_temp.iloc[i][df_temp.iloc[i]!=0].index)\n",
    "        intersecting_ent = list(set(query_entities).intersection(neighbor_entities))\n",
    "        overlapping_count.append(len(intersecting_ent))\n",
    "    neighbors['overlapping'] = overlapping_count\n",
    "    #Make a score function as a product of similarity *  # of common entities\n",
    "    neighbors['score'] = neighbors[9999999]*neighbors['overlapping']\n",
    "    #Add doi of each\n",
    "    neighbors['doi'] = df.loc[neighbors.index]['doi']\n",
    "    #Filter with scores higher than 0\n",
    "    neighbors = neighbors[neighbors['score']> 0]\n",
    "    #Sort by score\n",
    "    neighbors = neighbors.sort_values(by=['score'], ascending=False)\n",
    "    #Drop duplicates, keep first\n",
    "    neighbors = neighbors.drop_duplicates(keep='first')\n",
    "    print(neighbors)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "import re\n",
    "ner_bio = spacy.load('en_ner_bionlp13cg_md')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 187,
   "metadata": {},
   "outputs": [],
   "source": [
    "user_doi = '10.1101/2020.02.05.935890'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 2 articles in 10.1101/2020.02.05.935890 category\n",
      "{'IDPs': 3, 'IDP': 2, 'Nuclear Magnetic Resonance (NMR) spectroscopy, Small-angle X-ray Scattering (SAXS': 1, 'Sic1': 3, 'higher-moments': 1, 'Sic1s': 1, 'Cdc4': 1}\n",
      "         DNA  chromatin  cells  Hi-C  nucleosome  partially-methylated-domain  \\\n",
      "9999999  0.0        0.0    0.0   0.0         0.0                          0.0   \n",
      "\n",
      "         non-partially-methylated-domain  cell  chromosomes  daughter cells  \\\n",
      "9999999                              0.0   0.0          0.0             0.0   \n",
      "\n",
      "         ...  diazirine moieties  3-hydroxyl  KK148  KK150  GABAARs  \\\n",
      "9999999  ...                 0.0         0.0    0.0    0.0      0.0   \n",
      "\n",
      "         electrons/pixel/sec  EER                        doi  unique_id  \\\n",
      "9999999                  0.0  0.0  10.1101/2020.02.05.935890        0.0   \n",
      "\n",
      "         version  \n",
      "9999999        2  \n",
      "\n",
      "[1 rows x 14934 columns]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/anaconda3/envs/scispacy/lib/python3.6/site-packages/scipy/spatial/distance.py:720: RuntimeWarning: invalid value encountered in double_scalars\n",
      "  dist = 1.0 - uv / np.sqrt(uu * vv)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "       9999999  overlapping     score                        doi\n",
      "3935  1.000000            7  7.000000  10.1101/2020.02.05.935890\n",
      "3035  0.649319            2  1.298638             10.1101/731133\n",
      "1605  0.573526            2  1.147051             10.1101/427476\n",
      "3878  0.543350            2  1.086699  10.1101/2020.01.25.919498\n",
      "3909  0.518735            2  1.037470  10.1101/2020.01.30.925636\n",
      "4215  0.518735            2  1.037470  10.1101/2020.03.27.012328\n",
      "1385  0.497767            2  0.995534             10.1101/377564\n",
      "3769  0.375247            2  0.750494  10.1101/2020.01.06.894576\n",
      "4400  0.364286            2  0.728571  10.1101/2020.04.22.055590\n",
      "2459  0.346399            2  0.692798             10.1101/621714\n",
      "3965  0.558090            1  0.558090  10.1101/2020.02.11.943969\n",
      "4194  0.268833            2  0.537665  10.1101/2020.03.23.004283\n",
      "1381  0.441087            1  0.441087             10.1101/377135\n",
      "1750  0.195787            2  0.391574             10.1101/376632\n",
      "4189  0.159221            2  0.318442  10.1101/2020.03.23.003269\n",
      "1029  0.223946            1  0.223946             10.1101/301911\n",
      "1654  0.201378            1  0.201378             10.1101/332205\n",
      "4233  0.136720            1  0.136720  10.1101/2020.03.30.016881\n",
      "2163  0.117364            1  0.117364             10.1101/557496\n",
      "714   0.112825            1  0.112825             10.1101/238170\n",
      "1819  0.076910            1  0.076910             10.1101/475277\n",
      "526   0.044647            1  0.044647             10.1101/202994\n"
     ]
    }
   ],
   "source": [
    "n = create_aa_matrix(user_doi, df, keyword=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[3935, 4050]"
      ]
     },
     "execution_count": 181,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 2 articles in 10.1101/2020.02.05.935890 category\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'2'"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query_mongo_article(user_doi)[-1]['version']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0      NaN\n",
       "1      NaN\n",
       "2      NaN\n",
       "3      NaN\n",
       "4      NaN\n",
       "        ..\n",
       "4427   NaN\n",
       "4428   NaN\n",
       "4429   NaN\n",
       "4430   NaN\n",
       "4431   NaN\n",
       "Name: IDPs, Length: 4432, dtype: float64"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df['IDPs']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Category Search "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "metadata": {},
   "outputs": [],
   "source": [
    "category_list = db.BioReco_raw.distinct(\"category\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 231,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 235,
   "metadata": {},
   "outputs": [],
   "source": [
    "PATH = \"/Users/iremn/PythonClass/Spiced/Personal_Projects/BioReco/Pickles\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 274,
   "metadata": {},
   "outputs": [],
   "source": [
    "def read_category_df(category):\n",
    "    \"\"\"Read the user input category and fetch files with duplicates\"\"\"\n",
    "    \n",
    "    #Define the path\n",
    "    PATH = \"../Pickles\"\n",
    "    files = os.listdir(PATH)\n",
    "\n",
    "    #Slice the category to get the unique names out of it\n",
    "    cat_str = category[1:6]\n",
    "    print(cat_str)\n",
    "    meta_file_list = []\n",
    "    category_file_list=[]\n",
    "    #Search the file list\n",
    "    for category in files:\n",
    "        x = re.search(ca, category)\n",
    "        if x is not None:\n",
    "            meta = re.search('meta', category)\n",
    "            if meta is not None:\n",
    "                meta_file_list.append(category)\n",
    "            else:\n",
    "                category_file_list.append(category)\n",
    "    print(meta_file_list, category_file_list)\n",
    "    df = pd.DataFrame()\n",
    "    for category in category_file_list:\n",
    "        df1 = pd.read_pickle(f'../Pickles/{category}')\n",
    "        df = pd.concat([df, df1], axis = 0)\n",
    "    df_meta = pd.DataFrame()\n",
    "    for category_meta in meta_file_list:\n",
    "        df1_meta = pd.read_pickle(f'../Pickles/{category_meta}')\n",
    "        df_meta = pd.concat([df_meta, df1_meta])\n",
    "    return df, df_meta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 275,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "euros\n",
      "['biochemistry_meta.pkl', 'Biochemistry _meta.pkl'] ['Biochemistry .pkl', 'biochemistry.pkl']\n"
     ]
    }
   ],
   "source": [
    "dfneuro, df_meta_neuro = read_category_df('neuroscience')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 273,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>version</th>\n",
       "      <th>doi</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>author_corresponding</th>\n",
       "      <th>author_corresponding_institution</th>\n",
       "      <th>date</th>\n",
       "      <th>type</th>\n",
       "      <th>category</th>\n",
       "      <th>published</th>\n",
       "      <th>server</th>\n",
       "      <th>unique_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/080440</td>\n",
       "      <td>The large terminase DNA packaging motor grips ...</td>\n",
       "      <td>Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...</td>\n",
       "      <td>Brian A. Kelch</td>\n",
       "      <td>University of Massachusetts Medical School</td>\n",
       "      <td>2016-10-17</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1093/nar/gkw1356</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa048f9b21ead543e417a7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>10.1101/080440</td>\n",
       "      <td>The large terminase DNA packaging motor grips ...</td>\n",
       "      <td>Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...</td>\n",
       "      <td>Brian A. Kelch</td>\n",
       "      <td>University of Massachusetts Medical School</td>\n",
       "      <td>2016-10-19</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1093/nar/gkw1356</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04949b21ead543e417cc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/081331</td>\n",
       "      <td>A novel mechanism of “metal gel-shift” by hist...</td>\n",
       "      <td>Shelake, R. M.; Ito, Y.; Masumoto, J.; Morita,...</td>\n",
       "      <td>Hidenori Hayashi</td>\n",
       "      <td>Proteo-Science Center, Ehime University, Matsu...</td>\n",
       "      <td>2016-10-20</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1371/journal.pone.0172182</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04949b21ead543e417e5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/083014</td>\n",
       "      <td>p53 dynamically directs TFIID assembly on targ...</td>\n",
       "      <td>Coleman, R. A.; Qiao, Z.; Singh, S. K.; Peng, ...</td>\n",
       "      <td>Robert A Coleman</td>\n",
       "      <td>Albert Einstein College of Medicine</td>\n",
       "      <td>2016-10-24</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1128/MCB.00085-17</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04999b21ead543e4182f</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2</td>\n",
       "      <td>10.1101/068502</td>\n",
       "      <td>Multiple binding modes of ibuprofen in human s...</td>\n",
       "      <td>Evoli, S.; Mobley, D.; Guzzi, R.; Rizzuti, B.</td>\n",
       "      <td>Bruno Rizzuti</td>\n",
       "      <td>University of Calabria</td>\n",
       "      <td>2016-10-25</td>\n",
       "      <td>new results</td>\n",
       "      <td>biochemistry</td>\n",
       "      <td>10.1039/C6CP05680F</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04999b21ead543e4184c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/074567</td>\n",
       "      <td>Multi-laboratory assessment of reproducibility...</td>\n",
       "      <td>Ben C Collins;Christie L Hunter;Yansheng Liu;B...</td>\n",
       "      <td>Ruedi Aebersold</td>\n",
       "      <td>ETH Zurich</td>\n",
       "      <td>2016-09-14</td>\n",
       "      <td>New Results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>10.1038/s41467-017-00249-5</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04739b21ead543e41557</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/078147</td>\n",
       "      <td>Proper modelling of ligand binding requires an...</td>\n",
       "      <td>Nicholas M Pearce;Frank von Delft;</td>\n",
       "      <td>Frank von Delft</td>\n",
       "      <td>University of Oxford</td>\n",
       "      <td>2016-09-28</td>\n",
       "      <td>New Results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>10.1107/S2059798317003412</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04819b21ead543e4164b</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/078337</td>\n",
       "      <td>SCFSlmb recognizes a conserved degron within t...</td>\n",
       "      <td>Kelsey M Gray;Kevin A Kaifer;David Baillat;Yin...</td>\n",
       "      <td>A. Gregory Matera</td>\n",
       "      <td>University of North Carolina</td>\n",
       "      <td>2016-09-30</td>\n",
       "      <td>New Results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>10.1091/mbc.E17-11-0627</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04819b21ead543e41667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/078733</td>\n",
       "      <td>Crystal structure and RNA-binding properties o...</td>\n",
       "      <td>Kimberly A Stanek;Jennifer Patterson West;Pete...</td>\n",
       "      <td>Cameron Mura</td>\n",
       "      <td>University of Virginia</td>\n",
       "      <td>2016-10-01</td>\n",
       "      <td>New Results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>10.1107/S2059798317000031</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04819b21ead543e4168b</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>1</td>\n",
       "      <td>10.1101/080010</td>\n",
       "      <td>The Chd1 chromatin remodeler shifts hexasomes ...</td>\n",
       "      <td>Robert F Levendosky;Anton Sabantsev;Sebastian ...</td>\n",
       "      <td>Gregory D Bowman</td>\n",
       "      <td>Johns Hopkins University</td>\n",
       "      <td>2016-10-09</td>\n",
       "      <td>New Results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>10.7554/eLife.21356</td>\n",
       "      <td>biorxiv</td>\n",
       "      <td>5eaa04869b21ead543e416f6</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3236 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    version             doi  \\\n",
       "0         1  10.1101/080440   \n",
       "1         2  10.1101/080440   \n",
       "2         1  10.1101/081331   \n",
       "3         1  10.1101/083014   \n",
       "4         2  10.1101/068502   \n",
       "..      ...             ...   \n",
       "102       1  10.1101/074567   \n",
       "103       1  10.1101/078147   \n",
       "104       1  10.1101/078337   \n",
       "105       1  10.1101/078733   \n",
       "106       1  10.1101/080010   \n",
       "\n",
       "                                                 title  \\\n",
       "0    The large terminase DNA packaging motor grips ...   \n",
       "1    The large terminase DNA packaging motor grips ...   \n",
       "2    A novel mechanism of “metal gel-shift” by hist...   \n",
       "3    p53 dynamically directs TFIID assembly on targ...   \n",
       "4    Multiple binding modes of ibuprofen in human s...   \n",
       "..                                                 ...   \n",
       "102  Multi-laboratory assessment of reproducibility...   \n",
       "103  Proper modelling of ligand binding requires an...   \n",
       "104  SCFSlmb recognizes a conserved degron within t...   \n",
       "105  Crystal structure and RNA-binding properties o...   \n",
       "106  The Chd1 chromatin remodeler shifts hexasomes ...   \n",
       "\n",
       "                                               authors author_corresponding  \\\n",
       "0    Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...       Brian A. Kelch   \n",
       "1    Hilbert, B. J.; Hayes, J. A.; Stone, N. P.; Xu...       Brian A. Kelch   \n",
       "2    Shelake, R. M.; Ito, Y.; Masumoto, J.; Morita,...     Hidenori Hayashi   \n",
       "3    Coleman, R. A.; Qiao, Z.; Singh, S. K.; Peng, ...     Robert A Coleman   \n",
       "4        Evoli, S.; Mobley, D.; Guzzi, R.; Rizzuti, B.        Bruno Rizzuti   \n",
       "..                                                 ...                  ...   \n",
       "102  Ben C Collins;Christie L Hunter;Yansheng Liu;B...      Ruedi Aebersold   \n",
       "103                 Nicholas M Pearce;Frank von Delft;      Frank von Delft   \n",
       "104  Kelsey M Gray;Kevin A Kaifer;David Baillat;Yin...    A. Gregory Matera   \n",
       "105  Kimberly A Stanek;Jennifer Patterson West;Pete...         Cameron Mura   \n",
       "106  Robert F Levendosky;Anton Sabantsev;Sebastian ...     Gregory D Bowman   \n",
       "\n",
       "                      author_corresponding_institution        date  \\\n",
       "0           University of Massachusetts Medical School  2016-10-17   \n",
       "1           University of Massachusetts Medical School  2016-10-19   \n",
       "2    Proteo-Science Center, Ehime University, Matsu...  2016-10-20   \n",
       "3                  Albert Einstein College of Medicine  2016-10-24   \n",
       "4                               University of Calabria  2016-10-25   \n",
       "..                                                 ...         ...   \n",
       "102                                         ETH Zurich  2016-09-14   \n",
       "103                               University of Oxford  2016-09-28   \n",
       "104                       University of North Carolina  2016-09-30   \n",
       "105                             University of Virginia  2016-10-01   \n",
       "106                           Johns Hopkins University  2016-10-09   \n",
       "\n",
       "            type       category                     published   server  \\\n",
       "0    new results   biochemistry           10.1093/nar/gkw1356  biorxiv   \n",
       "1    new results   biochemistry           10.1093/nar/gkw1356  biorxiv   \n",
       "2    new results   biochemistry  10.1371/journal.pone.0172182  biorxiv   \n",
       "3    new results   biochemistry          10.1128/MCB.00085-17  biorxiv   \n",
       "4    new results   biochemistry            10.1039/C6CP05680F  biorxiv   \n",
       "..           ...            ...                           ...      ...   \n",
       "102  New Results  Biochemistry     10.1038/s41467-017-00249-5  biorxiv   \n",
       "103  New Results  Biochemistry      10.1107/S2059798317003412  biorxiv   \n",
       "104  New Results  Biochemistry        10.1091/mbc.E17-11-0627  biorxiv   \n",
       "105  New Results  Biochemistry      10.1107/S2059798317000031  biorxiv   \n",
       "106  New Results  Biochemistry            10.7554/eLife.21356  biorxiv   \n",
       "\n",
       "                    unique_id  \n",
       "0    5eaa048f9b21ead543e417a7  \n",
       "1    5eaa04949b21ead543e417cc  \n",
       "2    5eaa04949b21ead543e417e5  \n",
       "3    5eaa04999b21ead543e4182f  \n",
       "4    5eaa04999b21ead543e4184c  \n",
       "..                        ...  \n",
       "102  5eaa04739b21ead543e41557  \n",
       "103  5eaa04819b21ead543e4164b  \n",
       "104  5eaa04819b21ead543e41667  \n",
       "105  5eaa04819b21ead543e4168b  \n",
       "106  5eaa04869b21ead543e416f6  \n",
       "\n",
       "[3236 rows x 12 columns]"
      ]
     },
     "execution_count": 273,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_meta_neuro"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "metadata": {},
   "outputs": [],
   "source": [
    "ca = \"Biochemistry\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "metadata": {},
   "outputs": [],
   "source": [
    "ca = ca[1:6]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 253,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'ioche'"
      ]
     },
     "execution_count": 253,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ca"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 265,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_file_list = []\n",
    "category_file_list=[]\n",
    "for category in files:\n",
    "    x = re.search(ca, category)\n",
    "    if x is not None:\n",
    "        meta = re.search('meta', category)\n",
    "        if meta is not None:\n",
    "            meta_file_list.append(category)\n",
    "        else:\n",
    "            category_file_list.append(category)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 266,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['biochemistry_meta.pkl', 'Biochemistry _meta.pkl']"
      ]
     },
     "execution_count": 266,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_file_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Biochemistry .pkl', 'biochemistry.pkl']"
      ]
     },
     "execution_count": 267,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "category_file_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 276,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>4624</th>\n",
       "      <th>4625</th>\n",
       "      <th>4626</th>\n",
       "      <th>4627</th>\n",
       "      <th>4628</th>\n",
       "      <th>4629</th>\n",
       "      <th>4630</th>\n",
       "      <th>4631</th>\n",
       "      <th>4632</th>\n",
       "      <th>4633</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4629</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4630</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4631</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4632</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4633</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4634 rows × 4634 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      0     1     2     3     4     5     6     7     8     9     ...  4624  \\\n",
       "0      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "1      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "2      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "3      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "4      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "...    ...   ...   ...   ...   ...   ...   ...   ...   ...   ...  ...   ...   \n",
       "4629   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "4630   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "4631   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "4632   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "4633   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  ...   0.0   \n",
       "\n",
       "      4625  4626  4627  4628  4629  4630  4631  4632  4633  \n",
       "0      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "1      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "2      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "3      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "4      0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "...    ...   ...   ...   ...   ...   ...   ...   ...   ...  \n",
       "4629   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "4630   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "4631   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "4632   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "4633   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0   0.0  \n",
       "\n",
       "[4634 rows x 4634 columns]"
      ]
     },
     "execution_count": 276,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "UU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 285,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/anaconda3/envs/scispacy/lib/python3.6/site-packages/pandas/core/generic.py:2505: PerformanceWarning: \n",
      "your performance may suffer as PyTables will pickle object types that it cannot\n",
      "map directly to c-types [inferred_type->mixed,key->block1_values] [items->Index(['index', 'doi', 'version'], dtype='object')]\n",
      "\n",
      "  encoding=encoding,\n"
     ]
    }
   ],
   "source": [
    "df.to_hdf('../Pickles/biophysics.h5', key='df', format='fixed')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>prostate cancer cell lines</th>\n",
       "      <th>LNCaP</th>\n",
       "      <th>PC3</th>\n",
       "      <th>\\\"second order\\</th>\n",
       "      <th>cellular line PC3</th>\n",
       "      <th>tumors.\\n\\nHighlightsO_LICancer</th>\n",
       "      <th>system.\\nC_LIO_LIProstate cancer cell lines</th>\n",
       "      <th>Gompertz dynamics\\nC_LIO_LIProstate cancer cell lines</th>\n",
       "      <th>matrix</th>\n",
       "      <th>...</th>\n",
       "      <th>tea</th>\n",
       "      <th>E. coli</th>\n",
       "      <th>single cell</th>\n",
       "      <th>Tat</th>\n",
       "      <th>inner membrane</th>\n",
       "      <th>TatA</th>\n",
       "      <th>TatB</th>\n",
       "      <th>TatBC</th>\n",
       "      <th>doi</th>\n",
       "      <th>version</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5eaa034c9b21ead543e3fb49</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/011189</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5eaa034c9b21ead543e3fb82</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/012930</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5eaa034c9b21ead543e3fb83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/012955</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5eaa03509b21ead543e3fc08</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/014449</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5eaa03559b21ead543e3fc23</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/014704</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>5eaa04819b21ead543e41662</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078352</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>5eaa04819b21ead543e41685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078626</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>5eaa04819b21ead543e41688</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078543</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>5eaa04819b21ead543e4169c</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1101/078998</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>5eaa04869b21ead543e416e0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>10.1101/079715</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>202 rows × 883 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        index  prostate cancer cell lines  LNCaP  PC3  \\\n",
       "0    5eaa034c9b21ead543e3fb49                         1.0    3.0  2.0   \n",
       "1    5eaa034c9b21ead543e3fb82                         NaN    NaN  NaN   \n",
       "2    5eaa034c9b21ead543e3fb83                         NaN    NaN  NaN   \n",
       "3    5eaa03509b21ead543e3fc08                         NaN    NaN  NaN   \n",
       "4    5eaa03559b21ead543e3fc23                         NaN    NaN  NaN   \n",
       "..                        ...                         ...    ...  ...   \n",
       "197  5eaa04819b21ead543e41662                         NaN    NaN  NaN   \n",
       "198  5eaa04819b21ead543e41685                         NaN    NaN  NaN   \n",
       "199  5eaa04819b21ead543e41688                         NaN    NaN  NaN   \n",
       "200  5eaa04819b21ead543e4169c                         NaN    NaN  NaN   \n",
       "201  5eaa04869b21ead543e416e0                         NaN    NaN  NaN   \n",
       "\n",
       "     \\\"second order\\  cellular line PC3  tumors.\\n\\nHighlightsO_LICancer  \\\n",
       "0                1.0                1.0                              1.0   \n",
       "1                NaN                NaN                              NaN   \n",
       "2                NaN                NaN                              NaN   \n",
       "3                NaN                NaN                              NaN   \n",
       "4                NaN                NaN                              NaN   \n",
       "..               ...                ...                              ...   \n",
       "197              NaN                NaN                              NaN   \n",
       "198              NaN                NaN                              NaN   \n",
       "199              NaN                NaN                              NaN   \n",
       "200              NaN                NaN                              NaN   \n",
       "201              NaN                NaN                              NaN   \n",
       "\n",
       "     system.\\nC_LIO_LIProstate cancer cell lines  \\\n",
       "0                                            1.0   \n",
       "1                                            NaN   \n",
       "2                                            NaN   \n",
       "3                                            NaN   \n",
       "4                                            NaN   \n",
       "..                                           ...   \n",
       "197                                          NaN   \n",
       "198                                          NaN   \n",
       "199                                          NaN   \n",
       "200                                          NaN   \n",
       "201                                          NaN   \n",
       "\n",
       "     Gompertz dynamics\\nC_LIO_LIProstate cancer cell lines  matrix  ...  tea  \\\n",
       "0                                                  1.0         NaN  ...  NaN   \n",
       "1                                                  NaN         1.0  ...  NaN   \n",
       "2                                                  NaN         NaN  ...  NaN   \n",
       "3                                                  NaN         NaN  ...  NaN   \n",
       "4                                                  NaN         NaN  ...  NaN   \n",
       "..                                                 ...         ...  ...  ...   \n",
       "197                                                NaN         NaN  ...  NaN   \n",
       "198                                                NaN         NaN  ...  NaN   \n",
       "199                                                NaN         NaN  ...  1.0   \n",
       "200                                                NaN         NaN  ...  NaN   \n",
       "201                                                NaN         NaN  ...  NaN   \n",
       "\n",
       "     E. coli  single cell  Tat  inner membrane  TatA  TatB  TatBC  \\\n",
       "0        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "1        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "2        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "3        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "4        NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "..       ...          ...  ...             ...   ...   ...    ...   \n",
       "197      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "198      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "199      NaN          NaN  NaN             NaN   NaN   NaN    NaN   \n",
       "200      1.0          1.0  NaN             NaN   NaN   NaN    NaN   \n",
       "201      2.0          NaN  4.0             6.0   8.0   1.0    2.0   \n",
       "\n",
       "                doi  version  \n",
       "0    10.1101/011189        2  \n",
       "1    10.1101/012930        1  \n",
       "2    10.1101/012955        1  \n",
       "3    10.1101/014449        1  \n",
       "4    10.1101/014704        1  \n",
       "..              ...      ...  \n",
       "197  10.1101/078352        1  \n",
       "198  10.1101/078626        1  \n",
       "199  10.1101/078543        1  \n",
       "200  10.1101/078998        1  \n",
       "201  10.1101/079715        1  \n",
       "\n",
       "[202 rows x 883 columns]"
      ]
     },
     "execution_count": 286,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_hdf('../Pickles/biophysics.h5')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Try reading the journal names etc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_short = pd.read_csv('../meta_short.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_short = meta_short[['title', 'published']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>published</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DNA Methylation Landscape Reflects the Spatial...</td>\n",
       "      <td>10.1016/j.bpj.2017.08.019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Computational study of the impacts of mechanic...</td>\n",
       "      <td>10.1371/journal.pcbi.1005533</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Collapse Precedes Folding in Denaturant-Depend...</td>\n",
       "      <td>10.1021/acs.jpcb.6b13100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Hi-C-constrained physical models of human chro...</td>\n",
       "      <td>10.1038/srep35985</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Flexibility and design: conformational heterog...</td>\n",
       "      <td>10.1016/j.str.2017.03.009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2504</th>\n",
       "      <td>Co-operation, competition and crowding: a disc...</td>\n",
       "      <td>10.1038/srep42134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2505</th>\n",
       "      <td>Composition fluctuations in lipid bilayers</td>\n",
       "      <td>10.1016/j.bpj.2017.10.009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2506</th>\n",
       "      <td>A pipeline approach to single-particle process...</td>\n",
       "      <td>10.1107/S2059798316019276</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2507</th>\n",
       "      <td>Kinematics of the Lever Arm Swing in Myosin VI</td>\n",
       "      <td>10.1073/pnas.1615708114</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2508</th>\n",
       "      <td>TatA complexes exhibit a marked change in orga...</td>\n",
       "      <td>10.1042/BCJ20160952</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2509 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     DNA Methylation Landscape Reflects the Spatial...   \n",
       "1     Computational study of the impacts of mechanic...   \n",
       "2     Collapse Precedes Folding in Denaturant-Depend...   \n",
       "3     Hi-C-constrained physical models of human chro...   \n",
       "4     Flexibility and design: conformational heterog...   \n",
       "...                                                 ...   \n",
       "2504  Co-operation, competition and crowding: a disc...   \n",
       "2505         Composition fluctuations in lipid bilayers   \n",
       "2506  A pipeline approach to single-particle process...   \n",
       "2507     Kinematics of the Lever Arm Swing in Myosin VI   \n",
       "2508  TatA complexes exhibit a marked change in orga...   \n",
       "\n",
       "                         published  \n",
       "0        10.1016/j.bpj.2017.08.019  \n",
       "1     10.1371/journal.pcbi.1005533  \n",
       "2         10.1021/acs.jpcb.6b13100  \n",
       "3                10.1038/srep35985  \n",
       "4        10.1016/j.str.2017.03.009  \n",
       "...                            ...  \n",
       "2504             10.1038/srep42134  \n",
       "2505     10.1016/j.bpj.2017.10.009  \n",
       "2506     10.1107/S2059798316019276  \n",
       "2507       10.1073/pnas.1615708114  \n",
       "2508           10.1042/BCJ20160952  \n",
       "\n",
       "[2509 rows x 2 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_short"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_doi_publ(doi):\n",
    "    \"\"\"\n",
    "    Adapted from the code in:\n",
    "    https://github.com/bibcure/doi2bib/blob/master/doi2bib/crossref.py\n",
    "\n",
    "    Parameters\n",
    "    ----------\n",
    "        doi: str\n",
    "    Returns\n",
    "    -------\n",
    "        found: bool\n",
    "        item: dict\n",
    "            Response from crossref\n",
    "    \"\"\"\n",
    "    bare_url = \"http://api.crossref.org/\"\n",
    "    url = \"{}works/{}\"\n",
    "    url = url.format(bare_url, doi)\n",
    "    r = requests.get(url)\n",
    "    found = False if r.status_code != 200 else True\n",
    "    item = r.json()\n",
    "\n",
    "    return found, item"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "query = meta_short['published'].apply(get_doi_publ)\n",
    "\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "journal_name = []\n",
    "citation_count = []\n",
    "for paper in query:\n",
    "    journal_name.append(paper[1]['message']['short-container-title'])\n",
    "    citation_count.append(paper[1]['message']['is-referenced-by-count'])\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_short['journal'] = journal_name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_short['citation_count'] = citation_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>published</th>\n",
       "      <th>journal</th>\n",
       "      <th>citation_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DNA Methylation Landscape Reflects the Spatial...</td>\n",
       "      <td>10.1016/j.bpj.2017.08.019</td>\n",
       "      <td>[Biophysical Journal]</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Computational study of the impacts of mechanic...</td>\n",
       "      <td>10.1371/journal.pcbi.1005533</td>\n",
       "      <td>[PLoS Comput Biol]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Collapse Precedes Folding in Denaturant-Depend...</td>\n",
       "      <td>10.1021/acs.jpcb.6b13100</td>\n",
       "      <td>[J. Phys. Chem. B]</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Hi-C-constrained physical models of human chro...</td>\n",
       "      <td>10.1038/srep35985</td>\n",
       "      <td>[Sci Rep]</td>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Flexibility and design: conformational heterog...</td>\n",
       "      <td>10.1016/j.str.2017.03.009</td>\n",
       "      <td>[Structure]</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2504</th>\n",
       "      <td>Co-operation, competition and crowding: a disc...</td>\n",
       "      <td>10.1038/srep42134</td>\n",
       "      <td>[Sci Rep]</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2505</th>\n",
       "      <td>Composition fluctuations in lipid bilayers</td>\n",
       "      <td>10.1016/j.bpj.2017.10.009</td>\n",
       "      <td>[Biophysical Journal]</td>\n",
       "      <td>23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2506</th>\n",
       "      <td>A pipeline approach to single-particle process...</td>\n",
       "      <td>10.1107/S2059798316019276</td>\n",
       "      <td>[Acta Cryst Sect D Struct Biol, Acta Crystallo...</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2507</th>\n",
       "      <td>Kinematics of the Lever Arm Swing in Myosin VI</td>\n",
       "      <td>10.1073/pnas.1615708114</td>\n",
       "      <td>[Proc Natl Acad Sci USA]</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2508</th>\n",
       "      <td>TatA complexes exhibit a marked change in orga...</td>\n",
       "      <td>10.1042/BCJ20160952</td>\n",
       "      <td>[]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2509 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     DNA Methylation Landscape Reflects the Spatial...   \n",
       "1     Computational study of the impacts of mechanic...   \n",
       "2     Collapse Precedes Folding in Denaturant-Depend...   \n",
       "3     Hi-C-constrained physical models of human chro...   \n",
       "4     Flexibility and design: conformational heterog...   \n",
       "...                                                 ...   \n",
       "2504  Co-operation, competition and crowding: a disc...   \n",
       "2505         Composition fluctuations in lipid bilayers   \n",
       "2506  A pipeline approach to single-particle process...   \n",
       "2507     Kinematics of the Lever Arm Swing in Myosin VI   \n",
       "2508  TatA complexes exhibit a marked change in orga...   \n",
       "\n",
       "                         published  \\\n",
       "0        10.1016/j.bpj.2017.08.019   \n",
       "1     10.1371/journal.pcbi.1005533   \n",
       "2         10.1021/acs.jpcb.6b13100   \n",
       "3                10.1038/srep35985   \n",
       "4        10.1016/j.str.2017.03.009   \n",
       "...                            ...   \n",
       "2504             10.1038/srep42134   \n",
       "2505     10.1016/j.bpj.2017.10.009   \n",
       "2506     10.1107/S2059798316019276   \n",
       "2507       10.1073/pnas.1615708114   \n",
       "2508           10.1042/BCJ20160952   \n",
       "\n",
       "                                                journal  citation_count  \n",
       "0                                 [Biophysical Journal]              11  \n",
       "1                                    [PLoS Comput Biol]               8  \n",
       "2                                    [J. Phys. Chem. B]              22  \n",
       "3                                             [Sci Rep]              37  \n",
       "4                                           [Structure]               9  \n",
       "...                                                 ...             ...  \n",
       "2504                                          [Sci Rep]              13  \n",
       "2505                              [Biophysical Journal]              23  \n",
       "2506  [Acta Cryst Sect D Struct Biol, Acta Crystallo...              98  \n",
       "2507                           [Proc Natl Acad Sci USA]              12  \n",
       "2508                                                 []               2  \n",
       "\n",
       "[2509 rows x 4 columns]"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_short"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "meta_ = meta_short.iloc[0:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "query = meta_['published'].apply(get_doi_publ)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Biophysical Journal']\n",
      "['PLoS Comput Biol']\n",
      "['J. Phys. Chem. B']\n",
      "['Sci Rep']\n",
      "['Structure']\n",
      "['Analyst']\n",
      "['Current Opinion in Structural Biology']\n",
      "['MBoC']\n",
      "['Biophysical Journal']\n",
      "['Nat Microbiol']\n"
     ]
    }
   ],
   "source": [
    "for paper in query:\n",
    "    print(paper[1]['message']['short-container-title'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "1    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "2    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "3    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "4    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "5    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "6    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "7    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "8    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "9    (True, {'status': 'ok', 'message-type': 'work'...\n",
       "Name: published, dtype: object"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_short['published'].apply(get_doi_publ).values[0][1]['message']['short-container-title']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
